<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Anish S. Shah, MD|MS">

<title>Research - Phenotyping AF</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="../site_libs/quarto-diagram/mermaid.min.js"></script>
<script src="../site_libs/quarto-diagram/mermaid-init.js"></script>
<link href="../site_libs/quarto-diagram/mermaid.css" rel="stylesheet">


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Research</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://asshah4.github.io" rel="" target=""><i class="bi bi-house-door" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../index.html" rel="" target="">
 <span class="menu-text">Overview</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="mailto:ashah282@uic.edu" rel="" target=""><i class="bi bi-envelope" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="http://github.com/asshah4" rel="" target=""><i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/shah_in_boots" rel="" target=""><i class="bi bi-twitter" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/anish-shah-326991177/" rel="" target=""><i class="bi bi-linkedin" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#proposal" id="toc-proposal" class="nav-link active" data-scroll-target="#proposal">Proposal</a></li>
  <li><a href="#background" id="toc-background" class="nav-link" data-scroll-target="#background">Background</a>
  <ul class="collapse">
  <li><a href="#phenotyping-approaches" id="toc-phenotyping-approaches" class="nav-link" data-scroll-target="#phenotyping-approaches">Phenotyping approaches</a></li>
  <li><a href="#genotyping-approaches" id="toc-genotyping-approaches" class="nav-link" data-scroll-target="#genotyping-approaches">Genotyping approaches</a></li>
  </ul></li>
  <li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data">Data</a>
  <ul class="collapse">
  <li><a href="#data-sources" id="toc-data-sources" class="nav-link" data-scroll-target="#data-sources">Data sources</a></li>
  <li><a href="#key-variables" id="toc-key-variables" class="nav-link" data-scroll-target="#key-variables">Key variables</a></li>
  </ul></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods">Methods</a>
  <ul class="collapse">
  <li><a href="#general-methods" id="toc-general-methods" class="nav-link" data-scroll-target="#general-methods">General methods</a></li>
  <li><a href="#detailed-approach" id="toc-detailed-approach" class="nav-link" data-scroll-target="#detailed-approach">Detailed approach</a>
  <ul class="collapse">
  <li><a href="#electrocardiogram-processing" id="toc-electrocardiogram-processing" class="nav-link" data-scroll-target="#electrocardiogram-processing">Electrocardiogram processing</a></li>
  <li><a href="#clinical-data-processing" id="toc-clinical-data-processing" class="nav-link" data-scroll-target="#clinical-data-processing">Clinical data processing</a></li>
  <li><a href="#clustering-analysis" id="toc-clustering-analysis" class="nav-link" data-scroll-target="#clustering-analysis">Clustering analysis</a></li>
  <li><a href="#genetic-variant-analysis" id="toc-genetic-variant-analysis" class="nav-link" data-scroll-target="#genetic-variant-analysis">Genetic variant analysis</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a></li>
  <li><a href="#timeline" id="toc-timeline" class="nav-link" data-scroll-target="#timeline">Timeline</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="phenotype-af.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Phenotyping AF</h1>
<p class="subtitle lead">Project Page</p>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Anish S. Shah, MD|MS </p>
          </div>
  </div>
    
  
    
  </div>
  

</header>

<section id="proposal" class="level1 page-columns page-full">
<h1>Proposal</h1>
<p>This proposal describes a year-long research prospecta covering the academic 2023-2024 year. The proposal builds on previous skills and background for a focused project that should serve as both preliminary data and support for a future K23 award. The focus of the proposal is on phenotyping of paroxysmal AF alone, however additional data will likely be generated to support the K23 application via preliminary data.</p>

<div class="no-row-height column-margin column-container"><div class="">
<p><strong>Aim 1: Identify electrocardiographic phenotypes of paroxysmal AF with structurally normal atria.</strong></p>
</div></div><p>This proposal specifies an approach to identify clinical phenotypes of <strong>paroxysmal AF</strong> that have a higher or increased risk of progression to a more <em>persistent</em> or <em>treatment-resistant</em> class based on burden of disease, electrical and structural remodeling, and the development of both atrial and ventricular cardiomyopathy.</p>
<p><u>Question</u>:</p>
<ol type="1">
<li>Can we identify which patients will progress more rapidly to persistent AF, or develop complications of AF?</li>
<li>Are there electrocardiographic or clinical parameters that predict electrical and structural remodeling?</li>
<li>Can the duration or chronicity of AF be predicted based off of ECG parameters?</li>
</ol>
<p><u>Hypothesis</u>:</p>
<p><strong>The longitudinal trajectory of pAF can be predicted by subphenotypes of paroxysmal AF based on electrical, clinical, and structural cardiac properties.</strong></p>
<p><u>Proposal</u>:</p>
<ol type="1">
<li>Identify clusters of paroxysmal AF based on cross-sectional electrical properties in both sinus rhythm and in atrial fibrillation.</li>
<li>Assess the trajectory of AF based on progression of the disease, including burden, severity, and complications (from CVA to both atrial and ventricular cardiomyopathy).</li>
<li>Generate a polygenic risk score for (A) the separate phenotypes of paroxysmal AF and (B) the risk of rapidity of progression of paroxysmal AF</li>
<li>Evaluate disease-specific ECG features for specific pathogenic/likely pathogenic mutants (e.g.&nbsp;TTN variants) within the AF population.</li>
</ol>
</section>
<section id="background" class="level1">
<h1>Background</h1>
<section id="phenotyping-approaches" class="level2">
<h2 class="anchored" data-anchor-id="phenotyping-approaches">Phenotyping approaches</h2>
<p>Paroxysmal AF contains a broad spectrum of disease, regardless of burden, frequency, hemodynamic or vascular complications, or etiology. Several attempts have been made at pursuing more salient models, however they combine both paroxysmal and persistent AF into a binary diagnosis. Two recent studies have documented 4 potential clusters may exist <span class="citation" data-cites="Pastori2020 Vitolo2021">(<a href="#ref-Pastori2020" role="doc-biblioref">Pastori et al. 2020</a>; <a href="#ref-Vitolo2021" role="doc-biblioref">Vitolo et al. 2021</a>)</span>:</p>
<ol type="1">
<li>Younger, lower comorbidities</li>
<li>High CV risk factors</li>
<li>High CV comorbidities</li>
<li>High rates of non-CV comorbidities (e.g.&nbsp;cancer)</li>
</ol>
<p><span class="citation" data-cites="Wineinger2019">Wineinger et al. (<a href="#ref-Wineinger2019" role="doc-biblioref">2019</a>)</span> showed that in terms of duration and episodes of AF, there may be a <em>staccato</em> (short, frequent episodes) versus <em>legato</em> (long, infrequent) patterns to AF, however unknown about underlying clinical relevance.</p>
</section>
<section id="genotyping-approaches" class="level2">
<h2 class="anchored" data-anchor-id="genotyping-approaches">Genotyping approaches</h2>
<p>There are an extensive number of studies on genetic contributions to AF. <span class="citation" data-cites="Ritchie2012">Ritchie et al. (<a href="#ref-Ritchie2012" role="doc-biblioref">2012</a>)</span> showed the association of the 4q25 region with familial AF. In a smaller study, <span class="citation" data-cites="Husser2009">Husser et al. (<a href="#ref-Husser2009" role="doc-biblioref">2009</a>)</span> showed that in certain populations of lone AF, the F-wave fibrillatory rate was determined by the KCNE1 gene. <span class="citation" data-cites="Kavousi2023">Kavousi and Ellinor (<a href="#ref-Kavousi2023" role="doc-biblioref">2023</a>)</span> notes that polygenic risk scores (PRS) may be equally important as monogenic risk scores, with similar effect sizes as seen in the UK Biobank (~3 fold increase in incident and prevalent AF risk). However, this data is limited to just European ancestry, was only case-control and with only ~1500 cases <span class="citation" data-cites="Choi2020">(<a href="#ref-Choi2020" role="doc-biblioref">Choi et al. 2020</a>)</span>.</p>
<div id="fig-morphology-and-variants" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../figures/verweij-2020-ecg-and-variants.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;1: The ECG features can be associated with specific genetic variants known to lead to cardiomyopathy and arrhythmia diagnoses, as expanded on by <span class="citation" data-cites="Verweij2020">Verweij et al. (<a href="#ref-Verweij2020" role="doc-biblioref">2020</a>)</span></figcaption>
</figure>
</div>
<p>The ECG is highly relevant as a corrolary to genetic variants, as seen in <a href="#fig-morphology-and-variants">Figure&nbsp;1</a>, and may provide insight into clinical phenotypes that may otherwise go undetected. For example, <span class="citation" data-cites="Wang2023">Wang et al. (<a href="#ref-Wang2023" role="doc-biblioref">2023</a>)</span> showed that in the UK Biobank, ECG-predicted risk of AF in a complimentary fashion to clinical risk while also helping to identify <em>novel</em> genetic variants. It remains unknown however if <em>certain genetic variants increase the rate of progression of paroxysmal AF</em> or how they may lead to <em>structural and electrical remodeling</em>.</p>
</section>
</section>
<section id="data" class="level1">
<h1>Data</h1>
<section id="data-sources" class="level2">
<h2 class="anchored" data-anchor-id="data-sources">Data sources</h2>
<p>The data that is currently available to answer this is derived from three sources:</p>
<ol type="a">
<li>UIC Multi-Ethnic AF Registry: ~1800 patients with various classes of AF that have contributed blood sampling for genetic analysis</li>
<li>Clinical Data Warehouse: collected by the CCTS, houses Cerner and EPIC charts from 2010 onwards, including clinical notes, procedure notes, diagnoses, vitals, medications</li>
<li>MUSE: ECGs from ~2000s to today with full digital/raw signal (10 seconds x 8 leads)</li>
</ol>
</section>
<section id="key-variables" class="level2">
<h2 class="anchored" data-anchor-id="key-variables">Key variables</h2>
<p>The key variables that will be used are a combination of clinical, electrical, imaging that will allow for creation of multiple dimensions for analysis.</p>
<div class="columns">
<div class="column" style="width:34%;">
<p><strong>Electrocardiography</strong></p>
<p><u>Sinus</u></p>
<ul>
<li>P-wave indices: voltage, AUC, duration, dispersion, terminal forces</li>
<li>Global parameters: PR interval, P/R amplitude ratio</li>
<li>Rate: high-frequency HRV, HR</li>
</ul>
<p><u>AF</u></p>
<ul>
<li>Conduction properties: ventricular response, shortest/longest RR intervals</li>
<li>Atrial parameters: dominant frequency (Fourier multi-transform), F-wave relative voltage + ∆ voltage over time</li>
</ul>
</div><div class="column" style="width:33%;">
<p><strong>Clinical</strong></p>
<ul>
<li>Age, Sex, Race</li>
<li>Weight + ∆ weight over time</li>
<li>Comorbidities (HTN, DM, HLD, CHF, CAD)</li>
<li>CHADS2VAS2C score</li>
<li>ADHF and h/o CHF</li>
<li>Anti-arrhythmic intensification (AAD, BB uptitration, cardioversion, ablation)</li>
<li>Atrial arrhythmias + recurrence + burden of AF over time</li>
<li>Blood pressure + ∆ over time</li>
<li>HR and responsiveness of HR over time</li>
</ul>
</div><div class="column" style="width:33%;">
<p><strong>Echocardiography</strong></p>
<ul>
<li>Volumes: LA, RA, LV, LVIDD</li>
<li>Pressures: RVSP, PA pressures</li>
<li>Diastology: E/e’ ratio, A wave duration/velocity, A wave reversal</li>
<li>Function: LVEF, RVEF, LA strain</li>
</ul>
</div>
</div>
</section>
</section>
<section id="methods" class="level1">
<h1>Methods</h1>
<section id="general-methods" class="level2">
<h2 class="anchored" data-anchor-id="general-methods">General methods</h2>
<p>The overall approach is inspired by the work of <span class="citation" data-cites="Nagamine2020">Nagamine et al. (<a href="#ref-Nagamine2020" role="doc-biblioref">2020</a>)</span> and <span class="citation" data-cites="Nagamine2022">Nagamine et al. (<a href="#ref-Nagamine2022" role="doc-biblioref">2022</a>)</span> to phenotype <em>worsening heart failure</em>. This is similar to how we will proceed with paroxysmal AF by identifying worsening or <em>progression</em> of the disease. There are major components of this work that will eventually come-together for analysis.</p>
<ol type="a">
<li>Electrocardiography-based rhythm and morphology evaluation: capitalizes on arrhythmia-focused research while utilizing signal processing and ML-based clustering approaches</li>
<li>Echocardiography and structural imaging assessments of cardiac function: low-level NLP in structured data parsing</li>
<li>Clinical context including co-morbidities, vitals, medications, procedures: epidemiology-focused contextualization of patient attributes/trajectories</li>
<li>Changes in parameters with time: survival analyses and recurrent event analyses (multi-state models)</li>
<li>Common and rare genetic variants: provides an approach to learn and expand on computational genomics to generate a polygenic risk score</li>
</ol>
<p>This project will have two major phases: 1) create clusters or phenotypes of paroxysmal AF based on both ECG and clinical features, 2) evaluate trajectory of disease and rapidity of increase in burden. The outcome is the <strong>trajectory of paroxysmal AF</strong>, which will be defined by the following concepts:</p>
<ul>
<li>Increase in burden of AF by CIED or cardiac telemetry device</li>
<li>Longest measured duration of AF</li>
<li>Need for intensification of medication therapy, e.g.&nbsp;initiation of class I or class III medications</li>
<li>Repeat ECG x 2-3 in AF consistently</li>
<li>Invasive rhythm control strategies, including electrical cardioversion and ablation</li>
<li>Reduction on LVEF or dilation of LV cavity</li>
<li>Increase in LA volumes</li>
</ul>
</section>
<section id="detailed-approach" class="level2">
<h2 class="anchored" data-anchor-id="detailed-approach">Detailed approach</h2>
<section id="electrocardiogram-processing" class="level3">
<h3 class="anchored" data-anchor-id="electrocardiogram-processing">Electrocardiogram processing</h3>
<ol type="1">
<li>Intake of all clinical data through CCTS as a superset of information on the patients within the AF Registry. This includes 10+ years of notes, procedures, medications, vitals, etc. Intake of all imaging data, including cardiac MRI and echocardiograms.</li>
<li>Intake of all ECG data, through the MUSE export to XML option, and converting the data into a WFDB-compatible format with data and header files.</li>
<li>Annotation and pre-processing of ECG data, including selection, removing baseline wander, high and low band-pass filtering, interpolation, removal of artifact/noise, and scaling/centering data.</li>
<li>Machine learning approach to segment morphology components and determine rhythm features (e.g.&nbsp;P-wave indices, F-wave dominant frequencies), as seen in <a href="#fig-ecg-ai">Figure&nbsp;2</a></li>
</ol>
<div id="fig-ecg-ai" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../figures/siontis-2021-ecg-ai-diagram.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;2: <span class="citation" data-cites="Siontis2021">Siontis et al. (<a href="#ref-Siontis2021" role="doc-biblioref">2021</a>)</span> demonstrates a pipeline for how ECG can be processed from computational perspective. This pipeline has been refined and is somewhat ubiquitous, and more importantly, has high accuracy for beat segmentation and arrhythmia diagnosis.</figcaption>
</figure>
</div>
</section>
<section id="clinical-data-processing" class="level3">
<h3 class="anchored" data-anchor-id="clinical-data-processing">Clinical data processing</h3>
<ol type="1">
<li>Generation of cross-sectional data using visit data, including ICD-9 and ICD-10 codes for diagnoses, encounters/discharges, CPT codes for procedures, and active medications, vital signs, and laboratory values.</li>
<li>Supplementation of cross-sectional data using study results, including echocardiography, device interrogations, procedure text (including ablations and cardioversions).</li>
<li>Creation of longitudinal data using changes in parameters over time, including heart rate &amp; blood pressure, medication intensification, echocardiography parameters, recurrence of arrhythmias, changes in disease burden.</li>
<li>Evaluation of unstructured text, as in <a href="#fig-patient-timeline">Figure&nbsp;3</a>, using a natural language processing (NLP) approach</li>
</ol>
<div id="fig-patient-timeline" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../figures/nagamine-2022-patient-timeline.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;3: Clinical features can be generated using both unstructured text, as well as structured imaging data (both electrocardiography and echocardiography based), as done by <span class="citation" data-cites="Nagamine2020">Nagamine et al. (<a href="#ref-Nagamine2020" role="doc-biblioref">2020</a>)</span></figcaption>
</figure>
</div>
</section>
<section id="clustering-analysis" class="level3">
<h3 class="anchored" data-anchor-id="clustering-analysis">Clustering analysis</h3>
<ol type="1">
<li>Initially, clustering will be performed similar to <a href="#fig-clusters">Figure&nbsp;4</a>, however this will primarily be focused on cross-sectional data and ECG-data. Future analyses will include the unstructured text data.</li>
<li>After general clusters are determined, can assess trajectory of AF. The trajectories can then <em>feedback</em> to generate phenotypical categories. Deep-learning approach will be used to identifying features associated with higher-risk phenotypes of paroxysmal AF.</li>
</ol>
<div id="fig-clusters" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../figures/nagamine-2022-heart-failure-clusters.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;4: Example of clustering approach to a disease phenotype by <span class="citation" data-cites="Nagamine2020">Nagamine et al. (<a href="#ref-Nagamine2020" role="doc-biblioref">2020</a>)</span>. Although related to heart failure, can likely be applied to paroxysmal AF and its progressive nature.</figcaption>
</figure>
</div>
</section>
<section id="genetic-variant-analysis" class="level3">
<h3 class="anchored" data-anchor-id="genetic-variant-analysis">Genetic variant analysis</h3>
<ol type="1">
<li>Genetic variant data will be incorporated by using evaluation of common common variants through polygenic risk score.</li>
<li>Monogenic analysis will be performed as described by <span class="citation" data-cites="vandeLeur2021">van de Leur et al. (<a href="#ref-vandeLeur2021" role="doc-biblioref">2021</a>)</span> using a combination of case/control for specific cardiomyopathy/arrhythmia genes and the corresponding ECG data. The number of samples can be surprisingly low, as seen in <a href="#fig-ecg-dz-methods">Figure&nbsp;5</a>, to be adequately powered to differentiate phenotypes (<a href="#fig-ecg-dz-ml">Figure&nbsp;6</a>).</li>
</ol>
<div class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-ecg-dz-methods" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../figures/van-de-Leur-2021-ecg-dnn-methods.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;5: As per <span class="citation" data-cites="vandeLeur2021">van de Leur et al. (<a href="#ref-vandeLeur2021" role="doc-biblioref">2021</a>)</span>, the power needed for evaluating a single gene is 20-50 individuals, with a 1:4 up to 1:20 case-control pattern.</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-ecg-dz-ml" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../figures/van-de-Leur-2021-ecg-disease-specific-ml.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;6: After creating median beats, components of the ECG can be compared between disease and non-disease states. This can be used to help classify risk for disease based on ECG-based</figcaption>
</figure>
</div>
</div>
</div>
</div>
</section>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<ol type="1">
<li>Clusters of ECG and clinical parameters that help identify phenotypes of paroxysmal AF, similar to <span class="citation" data-cites="Nagamine2022">Nagamine et al. (<a href="#ref-Nagamine2022" role="doc-biblioref">2022</a>)</span> (<a href="#fig-clusters">Figure&nbsp;4</a>).</li>
<li>Hazard/risk between clusters for worsening/progression of paroxysmal AF</li>
<li>Polygenic risk score to predict progression of paroxysmal AL</li>
<li>ECG features that correlate with pathogenic/likely pathogenic variants in cardiomyopathy/arrhythmia genes (e.g.&nbsp;TTN, 4q25)</li>
</ol>
</section>
<section id="timeline" class="level1">
<h1>Timeline</h1>
<div class="cell" data-fig-responsive="true" data-fig-width="6" data-fig-height="6" data-fig-label="fig-timeline">
<div class="cell-output-display">
<div>
<div>
<pre class="mermaid mermaid-js">gantt
    dateFormat MM/YYYY
    axisFormat %b-%Y
    excludes weekends
    section Intake
        MUSE ECG to XML                                 :done, muse, 05/2023, 07/2023
        XML ECG to WFDB                                 :done, wfdb, 04/2023, 90d
        Cluster hosting of data                 :done, acer, 06/2023, 60d
        Import CCTS data                                :done, 05/2023, 90d
    section Data tidying
        Sinus rhythm parameters                 :done, nsr, 07/2023, 30d
        AF rhythm parameters                        :paf, 07/2023, 180d
        Clinical visits and diagnoses       :visits, 07/2023, 120d
        Labs and lab changes                        :labs, after visits, 30d
        Clinical medication changes         :meds, after labs, 30d
        Rhythm/control strategies               :rhythm, after meds, 60d
        EPS data extraction                         :eps, 01/2024, 90d
        NLP for notes                                       :nlp, after visits, 180d
    section Analysis
        Cross-sectional ECG                         :clusters, after nsr, 30d
        ECG trajectories                                :delta, after paf, 30d
        Cross-sectional clincal                 :cross, after rhythm, 30d
        Clinical trajectories                       :after tidy, 60d
    section Production
        CCTS/CBCD IRB                                       :done, irb, 04/2023, 30d
        ECG/AF Clusters                                 :milestone, ecg, after delta, 7d
        Phenotype AF                                        :milestone, phenotype, after rhythm, 7d
</pre>
</div>
</div>
</div>
</div>
</section>
<section id="references" class="level1">




</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-Choi2020" class="csl-entry" role="listitem">
Choi, Seung Hoan, Sean J. Jurgens, Lu-Chen Weng, James P. Pirruccello, Carolina Roselli, Mark Chaffin, Christina J.-Y. Lee, et al. 2020. <span>“Monogenic and <span>Polygenic Contributions</span> to <span>Atrial Fibrillation Risk</span>.”</span> <em>Circulation Research</em> 126 (2): 200–209. <a href="https://doi.org/10.1161/CIRCRESAHA.119.315686">https://doi.org/10.1161/CIRCRESAHA.119.315686</a>.
</div>
<div id="ref-Husser2009" class="csl-entry" role="listitem">
Husser, Daniela, Martin Stridh, Leif Sörnmo, Dan M. Roden, Dawood Darbar, and Andreas Bollmann. 2009. <span>“A <span>Genotype-Dependent Intermediate ECG Phenotype</span> in <span>Patients With Persistent Lone Atrial Fibrillation</span>.”</span> <em>Circulation: Arrhythmia and Electrophysiology</em> 2 (1): 24–28. <a href="https://doi.org/10.1161/CIRCEP.108.799098">https://doi.org/10.1161/CIRCEP.108.799098</a>.
</div>
<div id="ref-Kavousi2023" class="csl-entry" role="listitem">
Kavousi, Maryam, and Patrick T. Ellinor. 2023. <span>“Polygenic Risk Scores for Prediction of Atrial Fibrillation.”</span> <em>Netherlands Heart Journal</em> 31 (1): 1–2. <a href="https://doi.org/10.1007/s12471-022-01755-y">https://doi.org/10.1007/s12471-022-01755-y</a>.
</div>
<div id="ref-vandeLeur2021" class="csl-entry" role="listitem">
Leur, Rutger R. van de, Karim Taha, Max N. Bos, Jeroen F. van der Heijden, Deepak Gupta, Maarten J. Cramer, Rutger J. Hassink, et al. 2021. <span>“Discovering and <span>Visualizing Disease-Specific Electrocardiogram Features Using Deep Learning</span>.”</span> <em>Circulation: Arrhythmia and Electrophysiology</em> 14 (2): e009056. <a href="https://doi.org/10.1161/CIRCEP.120.009056">https://doi.org/10.1161/CIRCEP.120.009056</a>.
</div>
<div id="ref-Nagamine2022" class="csl-entry" role="listitem">
Nagamine, Tasha, Brian Gillette, John Kahoun, Rolf Burghaus, Jörg Lippert, and Mayur Saxena. 2022. <span>“Data-Driven Identification of Heart Failure Disease States and Progression Pathways Using Electronic Health Records.”</span> <em>Scientific Reports</em> 12 (1, 1): 17871. <a href="https://doi.org/10.1038/s41598-022-22398-4">https://doi.org/10.1038/s41598-022-22398-4</a>.
</div>
<div id="ref-Nagamine2020" class="csl-entry" role="listitem">
Nagamine, Tasha, Brian Gillette, Alexey Pakhomov, John Kahoun, Hannah Mayer, Rolf Burghaus, Jörg Lippert, and Mayur Saxena. 2020. <span>“Multiscale Classification of Heart Failure Phenotypes by Unsupervised Clustering of Unstructured Electronic Medical Record Data.”</span> <em>Scientific Reports</em> 10 (1, 1): 21340. <a href="https://doi.org/10.1038/s41598-020-77286-6">https://doi.org/10.1038/s41598-020-77286-6</a>.
</div>
<div id="ref-Pastori2020" class="csl-entry" role="listitem">
Pastori, D, E Antonucci, A Milanese, F Violi, P Pignatelli, G Palareti, and A Farcomeni. 2020. <span>“Clinical Phenotypes of Atrial Fibrillation and Risk of Mortality: A Cluster Analysis.”</span> <em>European Heart Journal</em> 41 (November). <a href="https://doi.org/10.1093/ehjci/ehaa946.2893">https://doi.org/10.1093/ehjci/ehaa946.2893</a>.
</div>
<div id="ref-Ritchie2012" class="csl-entry" role="listitem">
Ritchie, Marylyn D., Shane Rowan, Gayle Kucera, Tanya Stubblefield, Marcia Blair, Shannon Carter, Dan M. Roden, and Dawood Darbar. 2012. <span>“Chromosome 4q25 Variants Are Genetic Modifiers of Rare Ion Channel Mutations Associated with Familial Atrial Fibrillation.”</span> <em>Journal of the American College of Cardiology</em> 60 (13): 1173–81. <a href="https://doi.org/10.1016/j.jacc.2012.04.030">https://doi.org/10.1016/j.jacc.2012.04.030</a>.
</div>
<div id="ref-Siontis2021" class="csl-entry" role="listitem">
Siontis, Konstantinos C., Peter A. Noseworthy, Zachi I. Attia, and Paul A. Friedman. 2021. <span>“Artificial Intelligence-Enhanced Electrocardiography in Cardiovascular Disease Management.”</span> <em>Nature Reviews Cardiology</em> 18 (7, 7): 465–78. <a href="https://doi.org/10.1038/s41569-020-00503-2">https://doi.org/10.1038/s41569-020-00503-2</a>.
</div>
<div id="ref-Verweij2020" class="csl-entry" role="listitem">
Verweij, Niek, Jan-Walter Benjamins, Michael P. Morley, Yordi J. Van De Vegte, Alexander Teumer, Teresa Trenkwalder, Wibke Reinhard, Thomas P. Cappola, and Pim Van Der Harst. 2020. <span>“The <span>Genetic Makeup</span> of the <span>Electrocardiogram</span>.”</span> <em>Cell Systems</em> 11 (3): 229–238.e5. <a href="https://doi.org/10.1016/j.cels.2020.08.005">https://doi.org/10.1016/j.cels.2020.08.005</a>.
</div>
<div id="ref-Vitolo2021" class="csl-entry" role="listitem">
Vitolo, Marco, Marco Proietti, Alena Shantsila, Giuseppe Boriani, and Gregory Y. H. Lip. 2021. <span>“Clinical <span>Phenotype Classification</span> of <span>Atrial Fibrillation Patients Using Cluster Analysis</span> and <span>Associations</span> with <span>Trial-Adjudicated Outcomes</span>.”</span> <em>Biomedicines</em> 9 (7): 843. <a href="https://doi.org/10.3390/biomedicines9070843">https://doi.org/10.3390/biomedicines9070843</a>.
</div>
<div id="ref-Wang2023" class="csl-entry" role="listitem">
Wang, Xin, Shaan Khurshid, Seung Hoan Choi, Samuel Friedman, Lu-Chen Weng, Christopher Reeder, James P. Pirruccello, et al. 2023. <span>“Genetic <span>Susceptibility</span> to <span>Atrial Fibrillation Identified</span> via <span>Deep Learning</span> of 12-<span>Lead Electrocardiograms</span>.”</span> <em>Circulation: Genomic and Precision Medicine</em> 0 (0): e003808. <a href="https://doi.org/10.1161/CIRCGEN.122.003808">https://doi.org/10.1161/CIRCGEN.122.003808</a>.
</div>
<div id="ref-Wineinger2019" class="csl-entry" role="listitem">
Wineinger, Nathan E., Paddy M. Barrett, Yunyue Zhang, Ikram Irfanullah, Evan D. Muse, Steven R. Steinhubl, and Eric J. Topol. 2019. <span>“Identification of Paroxysmal Atrial Fibrillation Subtypes in over 13,000 Individuals.”</span> <em>Heart Rhythm</em> 16 (1): 26–30. <a href="https://doi.org/10.1016/j.hrthm.2018.08.012">https://doi.org/10.1016/j.hrthm.2018.08.012</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">Copyright (c) 2023, Anish. S. Shah, MD, MS</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>



</body></html>