---
title: Phenotyping AF
subtitle: Project Page
author: Anish S. Shah, MD|MS
format: 
  html: default
  pdf:
    documentclass: article
    fig-pos: H
    number-sections: true
tbl-cap-location: bottom
fig-cap-location: bottom
---

# Proposal

This proposal describes a year-long research prospecta covering the academic 2023-2024 year. 
The proposal builds on previous skills and background for a focused project that should serve as both preliminary data and support for a future K23 award.
The focus of the proposal is on phenotyping of paroxysmal AF alone, however additional data will likely be generated to support the K23 application via preliminary data. 

::: {.column-margin}

__Aim 1: Identify electrocardiographic phenotypes of paroxysmal AF with structurally normal atria.__

:::

This proposal specifies an approach to identify clinical phenotypes of **paroxysmal AF** that have a higher or increased risk of progression to a more *persistent* or *treatment-resistant* class based on burden of disease, electrical and structural remodeling, and the development of both atrial and ventricular cardiomyopathy. 

[Question]{.underline}: 

1. Can we identify which patients will progress more rapidly to persistent AF, or develop complications of AF? 
1. Are there electrocardiographic or clinical parameters that predict electrical and structural remodeling?
1. Can the duration or chronicity of AF be predicted based off of ECG parameters?

[Hypothesis]{.underline}: 

**The longitudinal trajectory of pAF can be predicted by subphenotypes of paroxysmal AF based on electrical, clinical, and structural cardiac properties.**

[Proposal]{.underline}: 

1. Identify clusters of paroxysmal AF based on cross-sectional electrical properties in both sinus rhythm and in atrial fibrillation.
1. Assess the trajectory of AF based on progression of the disease, including burden, severity, and complications (from CVA to both atrial and ventricular cardiomyopathy).
1. Generate a polygenic risk score for (A) the separate phenotypes of paroxysmal AF and (B) the risk of rapidity of progression of paroxysmal AF
1. Evaluate disease-specific ECG features for specific pathogenic/likely pathogenic mutants (e.g. TTN variants) within the AF population.

\newpage

# Background

## Phenotyping approaches

Paroxysmal AF contains a broad spectrum of disease, regardless of burden, frequency, hemodynamic or vascular complications, or etiology.
Several attempts have been made at pursuing more salient models, however they combine both paroxysmal and persistent AF into a binary diagnosis.
Two recent studies have documented 4 potential clusters may exist [@Pastori2020; @Vitolo2021]:

1.  Younger, lower comorbidities
1.  High CV risk factors
1.  High CV comorbidities
1.  High rates of non-CV comorbidities (e.g. cancer)

@Wineinger2019 showed that in terms of duration and episodes of AF, there may be a *staccato* (short, frequent episodes) versus *legato* (long, infrequent) patterns to AF, however unknown about underlying clinical relevance.

## Genotyping approaches

There are an extensive number of studies on genetic contributions to AF.
@Ritchie2012 showed the association of the 4q25 region with familial AF.
In a smaller study, @Husser2009 showed that in certain populations of lone AF, the F-wave fibrillatory rate was determined by the KCNE1 gene. 
@Kavousi2023 notes that polygenic risk scores (PRS) may be equally important as monogenic risk scores, with similar effect sizes as seen in the UK Biobank (~3 fold increase in incident and prevalent AF risk).
However, this data is limited to just European ancestry, was only case-control and with only ~1500 cases [@Choi2020].

![The ECG features can be associated with specific genetic variants known to lead to cardiomyopathy and arrhythmia diagnoses, as expanded on by @Verweij2020](../figures/verweij-2020-ecg-and-variants.png){#fig-morphology-and-variants}

The ECG is highly relevant as a corrolary to genetic variants, as seen in @fig-morphology-and-variants, and may provide insight into clinical phenotypes that may otherwise go undetected.
For example, @Wang2023 showed that in the UK Biobank, ECG-predicted risk of AF in a complimentary fashion to clinical risk while also helping to identify *novel* genetic variants. 
It remains unknown however if *certain genetic variants increase the rate of progression of paroxysmal AF* or how they may lead to *structural and electrical remodeling*.

\newpage

# Data

## Data sources

The data that is currently available to answer this is derived from three sources:

a. UIC Multi-Ethnic AF Registry: ~1800 patients with various classes of AF that have contributed blood sampling for genetic analysis
b. Clinical Data Warehouse: collected by the CCTS, houses Cerner and EPIC charts from 2010 onwards, including clinical notes, procedure notes, diagnoses, vitals, medications
c. MUSE: ECGs from ~2000s to today with full digital/raw signal (10 seconds x 8 leads)

## Key variables

The key variables that will be used are a combination of clinical, electrical, imaging that will allow for creation of multiple dimensions for analysis.

:::: {.columns}
::: {.column width="34%"}

**Electrocardiography**

[Sinus]{.underline}

- P-wave indices: voltage, AUC, duration, dispersion, terminal forces
- Global parameters: PR interval, P/R amplitude ratio
- Rate: high-frequency HRV, HR
 
[AF]{.underline}

- Conduction properties: ventricular response, shortest/longest RR intervals 
- Atrial parameters: dominant frequency (Fourier multi-transform), F-wave relative voltage + ∆ voltage over time

:::
::: {.column width="33%"}

**Clinical**

- Age, Sex, Race
- Weight + ∆ weight over time
- Comorbidities (HTN, DM, HLD, CHF, CAD)
- CHADS2VAS2C score
- ADHF and h/o CHF
- Anti-arrhythmic intensification (AAD, BB uptitration, cardioversion, ablation)
- Atrial arrhythmias + recurrence + burden of AF over time
- Blood pressure + ∆ over time
- HR and responsiveness of HR over time

:::
::: {.column width="33%"}

**Echocardiography**

- Volumes: LA, RA, LV, LVIDD
- Pressures: RVSP, PA pressures
- Diastology: E/e' ratio, A wave duration/velocity, A wave reversal
- Function: LVEF, RVEF, LA strain

:::
::::

\newpage

# Methods

## General methods

The overall approach is inspired by the work of @Nagamine2020 and @Nagamine2022 to phenotype *worsening heart failure*. 
This is similar to how we will proceed with paroxysmal AF by identifying worsening or *progression* of the disease.
There are major components of this work that will eventually come-together for analysis.

a. Electrocardiography-based rhythm and morphology evaluation: capitalizes on arrhythmia-focused research while utilizing signal processing and ML-based clustering approaches
a. Echocardiography and structural imaging assessments of cardiac function: low-level NLP in structured data parsing
a. Clinical context including co-morbidities, vitals, medications, procedures: epidemiology-focused contextualization of patient attributes/trajectories
a. Changes in parameters with time: survival analyses and recurrent event analyses (multi-state models)
a. Common and rare genetic variants: provides an approach to learn and expand on computational genomics to generate a polygenic risk score 

This project will have two major phases: 1) create clusters or phenotypes of paroxysmal AF based on both ECG and clinical features, 2) evaluate trajectory of disease and rapidity of increase in burden. 
The outcome is the **trajectory of paroxysmal AF**, which will be defined by the following concepts:

- Increase in burden of AF by CIED or cardiac telemetry device
- Longest measured duration of AF
- Need for intensification of medication therapy, e.g. initiation of class I or class III medications
- Repeat ECG x 2-3 in AF consistently
- Invasive rhythm control strategies, including electrical cardioversion and ablation 
- Reduction on LVEF or dilation of LV cavity
- Increase in LA volumes

\newpage

## Detailed approach

\newpage

### Electrocardiogram processing

1. Intake of all clinical data through CCTS as a superset of information on the patients within the AF Registry. This includes 10+ years of notes, procedures, medications, vitals, etc. Intake of all imaging data, including cardiac MRI and echocardiograms.
1. Intake of all ECG data, through the MUSE export to XML option, and converting the data into a WFDB-compatible format with data and header files.
1. Annotation and pre-processing of ECG data, including selection, removing baseline wander, high and low band-pass filtering, interpolation, removal of artifact/noise, and scaling/centering data.
1. Machine learning approach to segment morphology components and determine rhythm features (e.g. P-wave indices, F-wave dominant frequencies), as seen in @fig-ecg-ai

![@Siontis2021 demonstrates a pipeline for how ECG can be processed from computational perspective. This pipeline has been refined and is somewhat ubiquitous, and more importantly, has high accuracy for beat segmentation and arrhythmia diagnosis.](../figures/siontis-2021-ecg-ai-diagram.png){#fig-ecg-ai}

\newpage

### Clinical data processing

1. Generation of cross-sectional data using visit data, including ICD-9 and ICD-10 codes for diagnoses, encounters/discharges, CPT codes for procedures, and active medications, vital signs, and laboratory values.
1. Supplementation of cross-sectional data using study results, including echocardiography, device interrogations, procedure text (including ablations and cardioversions).
1. Creation of longitudinal data using changes in parameters over time, including heart rate & blood pressure, medication intensification, echocardiography parameters, recurrence of arrhythmias, changes in disease burden. 
1. Evaluation of unstructured text, as in @fig-patient-timeline, using a natural language processing (NLP) approach

![Clinical features can be generated using both unstructured text, as well as structured imaging data (both electrocardiography and echocardiography based), as done by @Nagamine2020](../figures/nagamine-2022-patient-timeline.png){#fig-patient-timeline}

\newpage

### Clustering analysis

1. Initially, clustering will be performed similar to @fig-clusters, however this will primarily be focused on cross-sectional data and ECG-data. Future analyses will include the unstructured text data. 
1. After general clusters are determined, can assess trajectory of AF. The trajectories can then *feedback* to generate phenotypical categories. Deep-learning approach will be used to identifying features associated with higher-risk phenotypes of paroxysmal AF.

![Example of clustering approach to a disease phenotype by @Nagamine2020. Although related to heart failure, can likely be applied to paroxysmal AF and its progressive nature.](../figures/nagamine-2022-heart-failure-clusters.png){#fig-clusters}

\newpage

### Genetic variant analysis

1. Genetic variant data will be incorporated by using evaluation of common common variants through polygenic risk score. 
1. Monogenic analysis will be performed as described by @vandeLeur2021 using a combination of case/control for specific cardiomyopathy/arrhythmia genes and the corresponding ECG data. The number of samples can be surprisingly low, as seen in @fig-ecg-dz-methods, to be adequately powered to differentiate phenotypes (@fig-ecg-dz-ml).

::: {layout-ncol=2}

![As per @vandeLeur2021, the power needed for evaluating a single gene is 20-50 individuals, with a 1:4 up to 1:20 case-control pattern.](../figures/van-de-Leur-2021-ecg-dnn-methods.png){#fig-ecg-dz-methods}

![After creating median beats, components of the ECG can be compared between disease and non-disease states. This can be used to help classify risk for disease based on ECG-based](../figures/van-de-Leur-2021-ecg-disease-specific-ml.png){#fig-ecg-dz-ml}

:::

\newpage

# Results

1. Clusters of ECG and clinical parameters that help identify phenotypes of paroxysmal AF, similar to @Nagamine2022 (@fig-clusters).
1. Hazard/risk between clusters for worsening/progression of paroxysmal AF 
1. Polygenic risk score to predict progression of paroxysmal AL
1. ECG features that correlate with pathogenic/likely pathogenic variants in cardiomyopathy/arrhythmia genes (e.g. TTN, 4q25)

\newpage

# Timeline

```{mermaid}
%%| fig-responsive: true
%%| fig-width: 6
%%| fig-height:	6 
%%| fig-label: fig-timeline
gantt
	dateFormat MM/YYYY
	axisFormat %b-%Y
	excludes weekends
	section Intake
		MUSE ECG to XML									:done, muse, 05/2023, 07/2023
		XML ECG to WFDB 								:done, wfdb, 04/2023, 90d
		Cluster hosting of data					:done, acer, 06/2023, 60d
		Import CCTS data								:ccts, 05/2023, 90d
	section Data tidying
		Sinus rhythm parameters 				:nsr, 07/2023, 30d
		AF rhythm parameters 		 				:paf, 07/2023, 60d
		Clinical visits and diagnoses		:visits, 07/2023, 120d
		Labs and lab changes						:labs, after visits, 30d
		Clinical medication changes			:meds, after labs, 30d
		Rhythm/control strategies				:rhythm, after meds, 60d
		EPS data extraction							:eps, 01/2024, 90d
		NLP for notes										:nlp, after visits, 180d
	section Analysis
		Cross-sectional ECG							:clusters, after nsr, 30d
		ECG trajectories								:delta, after paf, 30d
		Cross-sectional clincal					:cross, after rhythm, 30d
		Clinical trajectories						:after tidy, 60d
	section Production
		CCTS/CBCD IRB										:done, irb, 04/2023, 30d
		ECG/AF Clusters 								:milestone, ecg, after delta, 7d
		Phenotype AF										:milestone, phenotype, after rhythm, 7d
```

\newpage

# References
