<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.553">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Anish Sanjay Shah, MD/MS Cardiology Fellow University of Illinois at Chicago">

<title>Research - Research-In-Progress</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Research</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://asshah4.github.io"> <i class="bi bi-house-door" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../index.html"> 
<span class="menu-text">Overview</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="mailto:ashah282@uic.edu"> <i class="bi bi-envelope" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="http://github.com/asshah4"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/shah_in_boots"> <i class="bi bi-twitter" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/anish-shah-326991177/"> <i class="bi bi-linkedin" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#april-5-2024" id="toc-april-5-2024" class="nav-link active" data-scroll-target="#april-5-2024">April 5, 2024</a>
  <ul class="collapse">
  <li><a href="#updates" id="toc-updates" class="nav-link" data-scroll-target="#updates">Updates</a></li>
  </ul></li>
  <li><a href="#april-1-2024" id="toc-april-1-2024" class="nav-link" data-scroll-target="#april-1-2024">April 1, 2024</a>
  <ul class="collapse">
  <li><a href="#updates-1" id="toc-updates-1" class="nav-link" data-scroll-target="#updates-1">Updates</a></li>
  <li><a href="#ecg-prediction-of-structural-and-ion-channel-mutants-in-af" id="toc-ecg-prediction-of-structural-and-ion-channel-mutants-in-af" class="nav-link" data-scroll-target="#ecg-prediction-of-structural-and-ion-channel-mutants-in-af">ECG Prediction of Structural and Ion Channel Mutants in AF</a></li>
  </ul></li>
  <li><a href="#march-29-2024" id="toc-march-29-2024" class="nav-link" data-scroll-target="#march-29-2024">March 29, 2024</a>
  <ul class="collapse">
  <li><a href="#update" id="toc-update" class="nav-link" data-scroll-target="#update">Update</a></li>
  <li><a href="#ecgttn" id="toc-ecgttn" class="nav-link" data-scroll-target="#ecgttn">ECG/TTN</a></li>
  </ul></li>
  <li><a href="#march-25-2024" id="toc-march-25-2024" class="nav-link" data-scroll-target="#march-25-2024">March 25, 2024</a>
  <ul class="collapse">
  <li><a href="#updates-2" id="toc-updates-2" class="nav-link" data-scroll-target="#updates-2">Updates</a></li>
  </ul></li>
  <li><a href="#february-26-2024" id="toc-february-26-2024" class="nav-link" data-scroll-target="#february-26-2024">February 26, 2024</a>
  <ul class="collapse">
  <li><a href="#updates-3" id="toc-updates-3" class="nav-link" data-scroll-target="#updates-3">Updates</a></li>
  <li><a href="#afeqtsdoh-paper-proposal" id="toc-afeqtsdoh-paper-proposal" class="nav-link" data-scroll-target="#afeqtsdoh-paper-proposal">AFEQT/SDOH paper proposal</a></li>
  </ul></li>
  <li><a href="#february-12-2024" id="toc-february-12-2024" class="nav-link" data-scroll-target="#february-12-2024">February 12, 2024</a>
  <ul class="collapse">
  <li><a href="#whole-exome-sequencing" id="toc-whole-exome-sequencing" class="nav-link" data-scroll-target="#whole-exome-sequencing">Whole Exome Sequencing</a></li>
  <li><a href="#afeqtsdoh" id="toc-afeqtsdoh" class="nav-link" data-scroll-target="#afeqtsdoh">AFEQT/SDOH</a></li>
  </ul></li>
  <li><a href="#january-29-2024" id="toc-january-29-2024" class="nav-link" data-scroll-target="#january-29-2024">January 29, 2024</a>
  <ul class="collapse">
  <li><a href="#updates-4" id="toc-updates-4" class="nav-link" data-scroll-target="#updates-4">Updates</a></li>
  </ul></li>
  <li><a href="#january-21-2024" id="toc-january-21-2024" class="nav-link" data-scroll-target="#january-21-2024">January 21, 2024</a>
  <ul class="collapse">
  <li><a href="#ecgttn-project" id="toc-ecgttn-project" class="nav-link" data-scroll-target="#ecgttn-project">ECG/<strong>TTN</strong> Project</a></li>
  <li><a href="#ecgttn-project-1" id="toc-ecgttn-project-1" class="nav-link" data-scroll-target="#ecgttn-project-1"><strong>ECG</strong>/TTN Project</a></li>
  </ul></li>
  <li><a href="#january-19-2024" id="toc-january-19-2024" class="nav-link" data-scroll-target="#january-19-2024">January 19, 2024</a>
  <ul class="collapse">
  <li><a href="#aflfh-revisions" id="toc-aflfh-revisions" class="nav-link" data-scroll-target="#aflfh-revisions">AFL/FH Revisions</a></li>
  <li><a href="#statistical-review" id="toc-statistical-review" class="nav-link" data-scroll-target="#statistical-review">Statistical Review</a></li>
  <li><a href="#statistical-response" id="toc-statistical-response" class="nav-link" data-scroll-target="#statistical-response">Statistical Response</a></li>
  </ul></li>
  <li><a href="#january-8-2024" id="toc-january-8-2024" class="nav-link" data-scroll-target="#january-8-2024">January 8, 2024</a>
  <ul class="collapse">
  <li><a href="#afeqtsdoh-background" id="toc-afeqtsdoh-background" class="nav-link" data-scroll-target="#afeqtsdoh-background">AFEQT/SDOH background</a></li>
  </ul></li>
  <li><a href="#december-18-2023" id="toc-december-18-2023" class="nav-link" data-scroll-target="#december-18-2023">December 18, 2023</a>
  <ul class="collapse">
  <li><a href="#afeqtsdoh-updates" id="toc-afeqtsdoh-updates" class="nav-link" data-scroll-target="#afeqtsdoh-updates">AFEQT/SDOH updates</a></li>
  </ul></li>
  <li><a href="#december-11-2023" id="toc-december-11-2023" class="nav-link" data-scroll-target="#december-11-2023">December 11, 2023</a>
  <ul class="collapse">
  <li><a href="#ecg-ttn-variant-project" id="toc-ecg-ttn-variant-project" class="nav-link" data-scroll-target="#ecg-ttn-variant-project">ECG &amp; TTN Variant Project</a></li>
  <li><a href="#ecg-data" id="toc-ecg-data" class="nav-link" data-scroll-target="#ecg-data">ECG data</a></li>
  <li><a href="#model" id="toc-model" class="nav-link" data-scroll-target="#model">Model</a></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a></li>
  <li><a href="#next-steps" id="toc-next-steps" class="nav-link" data-scroll-target="#next-steps">Next steps</a></li>
  </ul></li>
  <li><a href="#december-4-2023" id="toc-december-4-2023" class="nav-link" data-scroll-target="#december-4-2023">December 4, 2023</a>
  <ul class="collapse">
  <li><a href="#hrs-abstract-proposals" id="toc-hrs-abstract-proposals" class="nav-link" data-scroll-target="#hrs-abstract-proposals">HRS abstract proposals</a></li>
  </ul></li>
  <li><a href="#november-20-2023" id="toc-november-20-2023" class="nav-link" data-scroll-target="#november-20-2023">November 20, 2023</a>
  <ul class="collapse">
  <li><a href="#aflfh-updates" id="toc-aflfh-updates" class="nav-link" data-scroll-target="#aflfh-updates">AFL/FH updates</a></li>
  <li><a href="#af-registry-and-ccts-data" id="toc-af-registry-and-ccts-data" class="nav-link" data-scroll-target="#af-registry-and-ccts-data">AF Registry and CCTS Data</a></li>
  <li><a href="#afeqtsdoh-1" id="toc-afeqtsdoh-1" class="nav-link" data-scroll-target="#afeqtsdoh-1">AFEQT/SDOH</a></li>
  <li><a href="#ecgttn-1" id="toc-ecgttn-1" class="nav-link" data-scroll-target="#ecgttn-1">ECG/TTN</a></li>
  </ul></li>
  <li><a href="#november-13-2023" id="toc-november-13-2023" class="nav-link" data-scroll-target="#november-13-2023">November 13, 2023</a>
  <ul class="collapse">
  <li><a href="#updates-5" id="toc-updates-5" class="nav-link" data-scroll-target="#updates-5">Updates</a></li>
  </ul></li>
  <li><a href="#october-26-2023" id="toc-october-26-2023" class="nav-link" data-scroll-target="#october-26-2023">October 26, 2023</a>
  <ul class="collapse">
  <li><a href="#aflfh-simple-revisions" id="toc-aflfh-simple-revisions" class="nav-link" data-scroll-target="#aflfh-simple-revisions">AFL/FH Simple Revisions</a></li>
  <li><a href="#aflfh-complex-revisions" id="toc-aflfh-complex-revisions" class="nav-link" data-scroll-target="#aflfh-complex-revisions">AFL/FH Complex Revisions</a></li>
  </ul></li>
  <li><a href="#october-23-2023" id="toc-october-23-2023" class="nav-link" data-scroll-target="#october-23-2023">October 23, 2023</a>
  <ul class="collapse">
  <li><a href="#updates-6" id="toc-updates-6" class="nav-link" data-scroll-target="#updates-6">Updates</a></li>
  </ul></li>
  <li><a href="#september-18-2023" id="toc-september-18-2023" class="nav-link" data-scroll-target="#september-18-2023">September 18, 2023</a>
  <ul class="collapse">
  <li><a href="#emr-data" id="toc-emr-data" class="nav-link" data-scroll-target="#emr-data">EMR Data</a></li>
  <li><a href="#afeqtsdoh-2" id="toc-afeqtsdoh-2" class="nav-link" data-scroll-target="#afeqtsdoh-2">AFEQT/SDOH</a></li>
  <li><a href="#afgenetics" id="toc-afgenetics" class="nav-link" data-scroll-target="#afgenetics">AF/Genetics</a></li>
  </ul></li>
  <li><a href="#september-11-2023" id="toc-september-11-2023" class="nav-link" data-scroll-target="#september-11-2023">September 11, 2023</a>
  <ul class="collapse">
  <li><a href="#afeqt" id="toc-afeqt" class="nav-link" data-scroll-target="#afeqt">AFEQT</a></li>
  <li><a href="#prs-for-afemerge" id="toc-prs-for-afemerge" class="nav-link" data-scroll-target="#prs-for-afemerge">PRS for AF/eMERGE</a></li>
  </ul></li>
  <li><a href="#september-8-2023" id="toc-september-8-2023" class="nav-link" data-scroll-target="#september-8-2023">September 8, 2023</a>
  <ul class="collapse">
  <li><a href="#afgenetics-1" id="toc-afgenetics-1" class="nav-link" data-scroll-target="#afgenetics-1">AF/Genetics</a></li>
  <li><a href="#ecgpaf" id="toc-ecgpaf" class="nav-link" data-scroll-target="#ecgpaf">ECG/pAF</a></li>
  <li><a href="#afsdoh" id="toc-afsdoh" class="nav-link" data-scroll-target="#afsdoh">AF/SDOH</a></li>
  </ul></li>
  <li><a href="#august-21-2023" id="toc-august-21-2023" class="nav-link" data-scroll-target="#august-21-2023">August 21, 2023</a>
  <ul class="collapse">
  <li><a href="#phenotyping-af" id="toc-phenotyping-af" class="nav-link" data-scroll-target="#phenotyping-af">Phenotyping AF</a></li>
  <li><a href="#ecg-and-genetic-variants" id="toc-ecg-and-genetic-variants" class="nav-link" data-scroll-target="#ecg-and-genetic-variants">ECG and Genetic Variants</a></li>
  <li><a href="#sdohafhsr" id="toc-sdohafhsr" class="nav-link" data-scroll-target="#sdohafhsr">SDOH/AF/HSR</a></li>
  </ul></li>
  <li><a href="#august-14-2023" id="toc-august-14-2023" class="nav-link" data-scroll-target="#august-14-2023">August 14, 2023</a>
  <ul class="collapse">
  <li><a href="#aflfh" id="toc-aflfh" class="nav-link" data-scroll-target="#aflfh">AFL/FH</a></li>
  <li><a href="#afsdoh-1" id="toc-afsdoh-1" class="nav-link" data-scroll-target="#afsdoh-1">AF/SDOH</a></li>
  <li><a href="#af-phenotyping" id="toc-af-phenotyping" class="nav-link" data-scroll-target="#af-phenotyping">AF Phenotyping</a></li>
  </ul></li>
  <li><a href="#july-31-2023" id="toc-july-31-2023" class="nav-link" data-scroll-target="#july-31-2023">July 31, 2023</a>
  <ul class="collapse">
  <li><a href="#aflfh-reviewer-concerns" id="toc-aflfh-reviewer-concerns" class="nav-link" data-scroll-target="#aflfh-reviewer-concerns">AFL/FH Reviewer Concerns</a></li>
  </ul></li>
  <li><a href="#july-24-2023" id="toc-july-24-2023" class="nav-link" data-scroll-target="#july-24-2023">July 24, 2023</a>
  <ul class="collapse">
  <li><a href="#active-projects" id="toc-active-projects" class="nav-link" data-scroll-target="#active-projects">Active projects</a></li>
  <li><a href="#phenotyping-af-1" id="toc-phenotyping-af-1" class="nav-link" data-scroll-target="#phenotyping-af-1">Phenotyping AF</a></li>
  <li><a href="#data-overview" id="toc-data-overview" class="nav-link" data-scroll-target="#data-overview">Data overview</a></li>
  <li><a href="#af-specific-data" id="toc-af-specific-data" class="nav-link" data-scroll-target="#af-specific-data">AF-specific data</a></li>
  <li><a href="#project-status" id="toc-project-status" class="nav-link" data-scroll-target="#project-status">Project status</a></li>
  </ul></li>
  <li><a href="#june-12-2023" id="toc-june-12-2023" class="nav-link" data-scroll-target="#june-12-2023">June 12, 2023</a>
  <ul class="collapse">
  <li><a href="#backbone-for-aims" id="toc-backbone-for-aims" class="nav-link" data-scroll-target="#backbone-for-aims">Backbone for aims</a></li>
  <li><a href="#trajectory-of-paf" id="toc-trajectory-of-paf" class="nav-link" data-scroll-target="#trajectory-of-paf">Trajectory of pAF</a></li>
  <li><a href="#predictors-of-paf-progression" id="toc-predictors-of-paf-progression" class="nav-link" data-scroll-target="#predictors-of-paf-progression">Predictors of pAF progression</a></li>
  <li><a href="#approach-for-evaluation" id="toc-approach-for-evaluation" class="nav-link" data-scroll-target="#approach-for-evaluation">Approach for evaluation</a></li>
  <li><a href="#next-steps-1" id="toc-next-steps-1" class="nav-link" data-scroll-target="#next-steps-1">Next steps</a></li>
  </ul></li>
  <li><a href="#june-5-2023" id="toc-june-5-2023" class="nav-link" data-scroll-target="#june-5-2023">June 5, 2023</a>
  <ul class="collapse">
  <li><a href="#aims-progress" id="toc-aims-progress" class="nav-link" data-scroll-target="#aims-progress">Aims Progress</a></li>
  <li><a href="#af-phenotyping-1" id="toc-af-phenotyping-1" class="nav-link" data-scroll-target="#af-phenotyping-1">AF Phenotyping</a></li>
  </ul></li>
  <li><a href="#may-22-2023" id="toc-may-22-2023" class="nav-link" data-scroll-target="#may-22-2023">May 22, 2023</a>
  <ul class="collapse">
  <li><a href="#ecg-analysis" id="toc-ecg-analysis" class="nav-link" data-scroll-target="#ecg-analysis">ECG analysis</a></li>
  </ul></li>
  <li><a href="#may-12-2023" id="toc-may-12-2023" class="nav-link" data-scroll-target="#may-12-2023">May 12, 2023</a>
  <ul class="collapse">
  <li><a href="#updates-7" id="toc-updates-7" class="nav-link" data-scroll-target="#updates-7">Updates</a></li>
  </ul></li>
  <li><a href="#may-8-2023" id="toc-may-8-2023" class="nav-link" data-scroll-target="#may-8-2023">May 8, 2023</a>
  <ul class="collapse">
  <li><a href="#goals" id="toc-goals" class="nav-link" data-scroll-target="#goals">Goals</a></li>
  <li><a href="#hrx-abstract" id="toc-hrx-abstract" class="nav-link" data-scroll-target="#hrx-abstract">HRx abstract</a></li>
  </ul></li>
  <li><a href="#april-17-2023" id="toc-april-17-2023" class="nav-link" data-scroll-target="#april-17-2023">April 17, 2023</a>
  <ul class="collapse">
  <li><a href="#goals-1" id="toc-goals-1" class="nav-link" data-scroll-target="#goals-1">Goals</a></li>
  </ul></li>
  <li><a href="#april-10-2023" id="toc-april-10-2023" class="nav-link" data-scroll-target="#april-10-2023">April 10, 2023</a>
  <ul class="collapse">
  <li><a href="#af-ontology" id="toc-af-ontology" class="nav-link" data-scroll-target="#af-ontology">AF Ontology</a></li>
  <li><a href="#phenotype-based-wes-cohort" id="toc-phenotype-based-wes-cohort" class="nav-link" data-scroll-target="#phenotype-based-wes-cohort">Phenotype-based WES cohort</a></li>
  </ul></li>
  <li><a href="#march-31-2023" id="toc-march-31-2023" class="nav-link" data-scroll-target="#march-31-2023">March 31, 2023</a>
  <ul class="collapse">
  <li><a href="#af-phenotyping-2" id="toc-af-phenotyping-2" class="nav-link" data-scroll-target="#af-phenotyping-2">AF Phenotyping</a></li>
  <li><a href="#eps-signal-processing" id="toc-eps-signal-processing" class="nav-link" data-scroll-target="#eps-signal-processing">EPS signal processing</a></li>
  </ul></li>
  <li><a href="#march-20-2023" id="toc-march-20-2023" class="nav-link" data-scroll-target="#march-20-2023">March 20, 2023</a>
  <ul class="collapse">
  <li><a href="#updates-8" id="toc-updates-8" class="nav-link" data-scroll-target="#updates-8">Updates</a></li>
  <li><a href="#help-wanted" id="toc-help-wanted" class="nav-link" data-scroll-target="#help-wanted">Help Wanted</a></li>
  </ul></li>
  <li><a href="#february-27-2023" id="toc-february-27-2023" class="nav-link" data-scroll-target="#february-27-2023">February 27, 2023</a>
  <ul class="collapse">
  <li><a href="#updates-9" id="toc-updates-9" class="nav-link" data-scroll-target="#updates-9">Updates</a></li>
  <li><a href="#non-linearity-in-ans-dysfunction" id="toc-non-linearity-in-ans-dysfunction" class="nav-link" data-scroll-target="#non-linearity-in-ans-dysfunction">Non-linearity in ANS dysfunction</a></li>
  <li><a href="#genetic-analysis" id="toc-genetic-analysis" class="nav-link" data-scroll-target="#genetic-analysis">Genetic analysis</a></li>
  <li><a href="#specific-aims" id="toc-specific-aims" class="nav-link" data-scroll-target="#specific-aims">Specific aims</a></li>
  </ul></li>
  <li><a href="#february-20-2023" id="toc-february-20-2023" class="nav-link" data-scroll-target="#february-20-2023">February 20, 2023</a>
  <ul class="collapse">
  <li><a href="#aflfh-1" id="toc-aflfh-1" class="nav-link" data-scroll-target="#aflfh-1">AFL/FH</a></li>
  <li><a href="#cbcd" id="toc-cbcd" class="nav-link" data-scroll-target="#cbcd">CBCD</a></li>
  </ul></li>
  <li><a href="#february-6-2023" id="toc-february-6-2023" class="nav-link" data-scroll-target="#february-6-2023">February 6, 2023</a>
  <ul class="collapse">
  <li><a href="#updates-10" id="toc-updates-10" class="nav-link" data-scroll-target="#updates-10">Updates</a></li>
  </ul></li>
  <li><a href="#january-9-2023" id="toc-january-9-2023" class="nav-link" data-scroll-target="#january-9-2023">January 9, 2023</a>
  <ul class="collapse">
  <li><a href="#updates-11" id="toc-updates-11" class="nav-link" data-scroll-target="#updates-11">Updates</a></li>
  <li><a href="#aflgenetics" id="toc-aflgenetics" class="nav-link" data-scroll-target="#aflgenetics"><span class="blue">AFL/genetics</span></a></li>
  <li><a href="#multiple-pedigrees" id="toc-multiple-pedigrees" class="nav-link" data-scroll-target="#multiple-pedigrees">Multiple Pedigrees</a></li>
  <li><a href="#updates-12" id="toc-updates-12" class="nav-link" data-scroll-target="#updates-12">Updates</a></li>
  <li><a href="#k23-aims" id="toc-k23-aims" class="nav-link" data-scroll-target="#k23-aims">K23 Aims</a></li>
  <li><a href="#k23-approach" id="toc-k23-approach" class="nav-link" data-scroll-target="#k23-approach">K23 Approach</a></li>
  <li><a href="#murine-model-of-vagolysis" id="toc-murine-model-of-vagolysis" class="nav-link" data-scroll-target="#murine-model-of-vagolysis">Murine Model of Vagolysis</a></li>
  </ul></li>
  <li><a href="#december-12-2022" id="toc-december-12-2022" class="nav-link" data-scroll-target="#december-12-2022">December 12, 2022</a>
  <ul class="collapse">
  <li><a href="#af-catheter-ablation-and-recurrence" id="toc-af-catheter-ablation-and-recurrence" class="nav-link" data-scroll-target="#af-catheter-ablation-and-recurrence">AF Catheter Ablation and Recurrence</a></li>
  <li><a href="#atrial-flutter-and-family-history" id="toc-atrial-flutter-and-family-history" class="nav-link" data-scroll-target="#atrial-flutter-and-family-history">Atrial Flutter and Family History</a></li>
  <li><a href="#ecgegm-analysis" id="toc-ecgegm-analysis" class="nav-link" data-scroll-target="#ecgegm-analysis">ECG/EGM analysis</a></li>
  </ul></li>
  <li><a href="#november-21-2022" id="toc-november-21-2022" class="nav-link" data-scroll-target="#november-21-2022">November 21, 2022</a>
  <ul class="collapse">
  <li><a href="#updates-13" id="toc-updates-13" class="nav-link" data-scroll-target="#updates-13">Updates</a></li>
  </ul></li>
  <li><a href="#november-14-2022" id="toc-november-14-2022" class="nav-link" data-scroll-target="#november-14-2022">November 14, 2022</a>
  <ul class="collapse">
  <li><a href="#early-onset-atrial-flutter" id="toc-early-onset-atrial-flutter" class="nav-link" data-scroll-target="#early-onset-atrial-flutter">Early Onset Atrial Flutter</a></li>
  </ul></li>
  <li><a href="#october-10-2022" id="toc-october-10-2022" class="nav-link" data-scroll-target="#october-10-2022">October 10, 2022</a></li>
  <li><a href="#october-2-2022" id="toc-october-2-2022" class="nav-link" data-scroll-target="#october-2-2022">October 2, 2022</a></li>
  <li><a href="#september-26-2022" id="toc-september-26-2022" class="nav-link" data-scroll-target="#september-26-2022">September 26, 2022</a>
  <ul class="collapse">
  <li><a href="#af-registry" id="toc-af-registry" class="nav-link" data-scroll-target="#af-registry">AF Registry</a></li>
  </ul></li>
  <li><a href="#september-12-2022" id="toc-september-12-2022" class="nav-link" data-scroll-target="#september-12-2022">September 12, 2022</a>
  <ul class="collapse">
  <li><a href="#af-registry-1" id="toc-af-registry-1" class="nav-link" data-scroll-target="#af-registry-1">AF Registry</a></li>
  <li><a href="#genetics" id="toc-genetics" class="nav-link" data-scroll-target="#genetics">Genetics</a></li>
  </ul></li>
  <li><a href="#august-22-2022" id="toc-august-22-2022" class="nav-link" data-scroll-target="#august-22-2022">August 22, 2022</a>
  <ul class="collapse">
  <li><a href="#af-registry-2" id="toc-af-registry-2" class="nav-link" data-scroll-target="#af-registry-2">AF Registry</a></li>
  <li><a href="#carto-maps" id="toc-carto-maps" class="nav-link" data-scroll-target="#carto-maps">CARTO Maps</a></li>
  </ul></li>
  <li><a href="#august-15-2022" id="toc-august-15-2022" class="nav-link" data-scroll-target="#august-15-2022">August 15, 2022</a></li>
  <li><a href="#august-1-2022" id="toc-august-1-2022" class="nav-link" data-scroll-target="#august-1-2022">August 1, 2022</a>
  <ul class="collapse">
  <li><a href="#af-registry-3" id="toc-af-registry-3" class="nav-link" data-scroll-target="#af-registry-3">AF Registry</a></li>
  <li><a href="#murine-ep-studies" id="toc-murine-ep-studies" class="nav-link" data-scroll-target="#murine-ep-studies">Murine EP Studies</a></li>
  <li><a href="#human-ep-studies" id="toc-human-ep-studies" class="nav-link" data-scroll-target="#human-ep-studies">Human EP Studies</a></li>
  </ul></li>
  <li><a href="#july-25-2022" id="toc-july-25-2022" class="nav-link" data-scroll-target="#july-25-2022">July 25, 2022</a>
  <ul class="collapse">
  <li><a href="#forest-plots" id="toc-forest-plots" class="nav-link" data-scroll-target="#forest-plots">Forest Plots</a></li>
  </ul></li>
  <li><a href="#july-18-2022" id="toc-july-18-2022" class="nav-link" data-scroll-target="#july-18-2022">July 18, 2022</a>
  <ul class="collapse">
  <li><a href="#af-registry-4" id="toc-af-registry-4" class="nav-link" data-scroll-target="#af-registry-4">AF Registry</a></li>
  </ul></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="rip.html"><i class="bi bi-file-slides"></i>RevealJS</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Research-In-Progress</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Anish Sanjay Shah, MD/MS<br><span style="font-size: 70%;">Cardiology Fellow<br>University of Illinois at Chicago</span> </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<div class="cell">
<div class="cell-output cell-output-stderr">
<pre><code>! Using an auto-discovered, cached token.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>  To suppress this message, modify your code or options to clearly consent to
  the use of a cached token.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>  See gargle's "Non-interactive auth" vignette for more details:</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>  &lt;https://gargle.r-lib.org/articles/non-interactive-auth.html&gt;</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>ℹ The googledrive package is using a cached token for 'mrshahman@gmail.com'.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Auto-refreshing stale OAuth token.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ The input `path` resolved to exactly 1 file.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>! Using an auto-discovered, cached token.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>  To suppress this message, modify your code or options to clearly consent to
  the use of a cached token.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>  See gargle's "Non-interactive auth" vignette for more details:</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>  &lt;https://gargle.r-lib.org/articles/non-interactive-auth.html&gt;</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>ℹ The googlesheets4 package is using a cached token for 'mrshahman@gmail.com'.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Auto-refreshing stale OAuth token.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ Reading from "research_log".</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ Range ''project_sheet''.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ The input `path` resolved to exactly 1 file.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ Reading from "research_log".</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ Range ''calendar_sheet''.</code></pre>
</div>
<div id="tbl-progress" class="cell quarto-float anchored">
<figure class="quarto-float quarto-float-tbl figure">
<div aria-describedby="tbl-progress-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<div>
<div id="ouokkhqozw" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#ouokkhqozw table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#ouokkhqozw thead, #ouokkhqozw tbody, #ouokkhqozw tfoot, #ouokkhqozw tr, #ouokkhqozw td, #ouokkhqozw th {
  border-style: none;
}

#ouokkhqozw p {
  margin: 0;
  padding: 0;
}

#ouokkhqozw .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: none;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: none;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#ouokkhqozw .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#ouokkhqozw .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#ouokkhqozw .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#ouokkhqozw .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ouokkhqozw .gt_bottom_border {
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ouokkhqozw .gt_col_headings {
  border-top-style: none;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ouokkhqozw .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#ouokkhqozw .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#ouokkhqozw .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#ouokkhqozw .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#ouokkhqozw .gt_column_spanner {
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#ouokkhqozw .gt_spanner_row {
  border-bottom-style: hidden;
}

#ouokkhqozw .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: none;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#ouokkhqozw .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: none;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#ouokkhqozw .gt_from_md > :first-child {
  margin-top: 0;
}

#ouokkhqozw .gt_from_md > :last-child {
  margin-bottom: 0;
}

#ouokkhqozw .gt_row {
  padding-top: 2px;
  padding-bottom: 2px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: none;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#ouokkhqozw .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#ouokkhqozw .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#ouokkhqozw .gt_row_group_first td {
  border-top-width: 2px;
}

#ouokkhqozw .gt_row_group_first th {
  border-top-width: 2px;
}

#ouokkhqozw .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ouokkhqozw .gt_first_summary_row {
  border-top-style: none;
  border-top-color: #D3D3D3;
}

#ouokkhqozw .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#ouokkhqozw .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ouokkhqozw .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ouokkhqozw .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: none;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#ouokkhqozw .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: none;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#ouokkhqozw .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#ouokkhqozw .gt_table_body {
  border-top-style: none;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ouokkhqozw .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ouokkhqozw .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ouokkhqozw .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ouokkhqozw .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ouokkhqozw .gt_left {
  text-align: left;
}

#ouokkhqozw .gt_center {
  text-align: center;
}

#ouokkhqozw .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#ouokkhqozw .gt_font_normal {
  font-weight: normal;
}

#ouokkhqozw .gt_font_bold {
  font-weight: bold;
}

#ouokkhqozw .gt_font_italic {
  font-style: italic;
}

#ouokkhqozw .gt_super {
  font-size: 65%;
}

#ouokkhqozw .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#ouokkhqozw .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#ouokkhqozw .gt_indent_1 {
  text-indent: 5px;
}

#ouokkhqozw .gt_indent_2 {
  text-indent: 10px;
}

#ouokkhqozw .gt_indent_3 {
  text-indent: 15px;
}

#ouokkhqozw .gt_indent_4 {
  text-indent: 20px;
}

#ouokkhqozw .gt_indent_5 {
  text-indent: 25px;
}
</style>

<table class="gt_table do-not-create-environment cell table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 40%">
<col style="width: 45%">
<col style="width: 15%">
</colgroup>
<thead>
<tr class="header gt_col_headings">
<th class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"></th>
<th id="Task" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" style="text-align: center; font-size: large; font-weight: bold; text-transform: capitalize;" scope="col">Task</th>
<th id="Deadline" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" style="text-align: center; font-size: large; font-weight: bold; text-transform: capitalize;" scope="col">Deadline</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd gt_group_heading_row">
<td colspan="3" id="Epidemiology (1.4%)" class="gt_group_heading" data-quarto-table-cell-role="th" style="background-color: #2096F2; color: #FFFFFF; border-left-width: 1px; border-left-style: solid; border-left-color: #2096F2FF; border-right-width: 1px; border-right-style: solid; border-right-color: #2096F2FF; border-top-width: 1px; border-top-style: solid; border-top-color: #2096F2FF; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #2096F2FF" scope="colgroup">Epidemiology (1.4%)</td>
</tr>
<tr class="even gt_row_group_first">
<td id="stub_1_1" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">AFEQT and SDOH</td>
<td class="gt_row gt_left" headers="Epidemiology (1.4%) stub_1_1 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Additional cross-sectional analyses; mean regressions over time</td>
<td class="gt_row gt_right" headers="Epidemiology (1.4%) stub_1_1 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-04-05</td>
</tr>
<tr class="odd">
<td id="stub_1_2" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; color: #FFFFFF; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">AFEQT and SDOH</td>
<td class="gt_row gt_left" headers="Epidemiology (1.4%) stub_1_2 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Manuscript draft</td>
<td class="gt_row gt_right" headers="Epidemiology (1.4%) stub_1_2 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-04-10</td>
</tr>
<tr class="even">
<td id="stub_1_3" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">GEH and DM</td>
<td class="gt_row gt_left" headers="Epidemiology (1.4%) stub_1_3 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Draft of paper</td>
<td class="gt_row gt_right" headers="Epidemiology (1.4%) stub_1_3 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-04-01</td>
</tr>
<tr class="odd gt_group_heading_row">
<td colspan="3" id="Arrhythmia (29%)" class="gt_group_heading" data-quarto-table-cell-role="th" style="background-color: #F8A725; color: #FFFFFF; border-left-width: 1px; border-left-style: solid; border-left-color: #F8A725FF; border-right-width: 1px; border-right-style: solid; border-right-color: #F8A725FF; border-top-width: 1px; border-top-style: solid; border-top-color: #F8A725FF; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #F8A725FF" scope="colgroup">Arrhythmia (29%)</td>
</tr>
<tr class="even gt_row_group_first">
<td id="stub_1_4" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">AFGen Fellowship</td>
<td class="gt_row gt_left" headers="Arrhythmia (29%) stub_1_4 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Bimonthly meetings with AFGen Consortium</td>
<td class="gt_row gt_right" headers="Arrhythmia (29%) stub_1_4 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-06-30</td>
</tr>
<tr class="odd">
<td id="stub_1_5" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">Atrial Flutter and Family History</td>
<td class="gt_row gt_left" headers="Arrhythmia (29%) stub_1_5 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">JAHA reviewer response to additional revisions</td>
<td class="gt_row gt_right" headers="Arrhythmia (29%) stub_1_5 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-03-29</td>
</tr>
<tr class="even">
<td id="stub_1_6" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">K23</td>
<td class="gt_row gt_left" headers="Arrhythmia (29%) stub_1_6 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">K23 specific aims draft</td>
<td class="gt_row gt_right" headers="Arrhythmia (29%) stub_1_6 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-03-30</td>
</tr>
<tr class="odd">
<td id="stub_1_7" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">Phenotype AF</td>
<td class="gt_row gt_left" headers="Arrhythmia (29%) stub_1_7 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Pipeline for VCF annotation extraction</td>
<td class="gt_row gt_right" headers="Arrhythmia (29%) stub_1_7 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-05-01</td>
</tr>
<tr class="even">
<td id="stub_1_8" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; color: #FFFFFF; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">Phenotype AF</td>
<td class="gt_row gt_left" headers="Arrhythmia (29%) stub_1_8 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Training neural networks to optimize model predictions</td>
<td class="gt_row gt_right" headers="Arrhythmia (29%) stub_1_8 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-05-01</td>
</tr>
<tr class="odd gt_group_heading_row">
<td colspan="3" id="Computational (16%)" class="gt_group_heading" data-quarto-table-cell-role="th" style="background-color: #00796B; color: #FFFFFF; border-left-width: 1px; border-left-style: solid; border-left-color: #00796BFF; border-right-width: 1px; border-right-style: solid; border-right-color: #00796BFF; border-top-width: 1px; border-top-style: solid; border-top-color: #00796BFF; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #00796BFF" scope="colgroup">Computational (16%)</td>
</tr>
<tr class="even gt_row_group_first">
<td id="stub_1_9" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">Causal Modeling</td>
<td class="gt_row gt_left" headers="Computational (16%) stub_1_9 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Version 1.0 release</td>
<td class="gt_row gt_right" headers="Computational (16%) stub_1_9 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-04-01</td>
</tr>
<tr class="odd">
<td id="stub_1_10" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">Harmonic Regressions</td>
<td class="gt_row gt_left" headers="Computational (16%) stub_1_10 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Implementation of cosinor, population-based cosinor, and multicomponent cosinor metrics</td>
<td class="gt_row gt_right" headers="Computational (16%) stub_1_10 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-04-01</td>
</tr>
<tr class="even gt_group_heading_row">
<td colspan="3" id="Clinical (49%)" class="gt_group_heading" data-quarto-table-cell-role="th" style="background-color: #E91E63; color: #FFFFFF; border-left-width: 1px; border-left-style: solid; border-left-color: #E91E63FF; border-right-width: 1px; border-right-style: solid; border-right-color: #E91E63FF; border-top-width: 1px; border-top-style: solid; border-top-color: #E91E63FF; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #E91E63FF" scope="colgroup">Clinical (49%)</td>
</tr>
<tr class="odd gt_row_group_first">
<td id="stub_1_11" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">Clinic</td>
<td class="gt_row gt_left" headers="Clinical (49%) stub_1_11 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">JBVA FIT Clinic</td>
<td class="gt_row gt_right" headers="Clinical (49%) stub_1_11 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-06-30</td>
</tr>
<tr class="even">
<td id="stub_1_12" class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" scope="row" style="text-align: center; font-style: italic; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF;">EP Lab</td>
<td class="gt_row gt_left" headers="Clinical (49%) stub_1_12 Task" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF">Working with EP faculty (Avitall/Johnson/Ringwala)</td>
<td class="gt_row gt_right" headers="Clinical (49%) stub_1_12 Deadline" style="border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: #5F7D8BFF; font-size: x-small">2024-06-30</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-tbl" id="tbl-progress-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Progress updates
</figcaption>
</figure>
</div>
</div>
<section id="april-5-2024" class="level1">
<h1>April 5, 2024</h1>
<section id="updates" class="level2 smallest">
<h2 class="smallest anchored" data-anchor-id="updates">Updates</h2>
<ul>
<li>ECG/Genetic Variants
<ul>
<li>Re-analyze/label sarcomeric/ion channel/“wild-type” AF WES dataset</li>
<li>AHA submission for abstract</li>
<li>Pending validation cohort</li>
</ul></li>
<li>AFEQT/SDOH
<ul>
<li>Table 1: Baseline characteristics of full cohort (longitudinal <span class="math inline">\(\rightarrow\)</span> supplement)</li>
<li>Table 2: AFEQT change by SDOH (NDI and ethnicity)</li>
<li>Table 3: AFEQT change predictors (SDOH)</li>
<li>Table 4: Rhythm/rate control strategy predictors</li>
<li>Figure 1: Study flow-chart</li>
<li>Figure 2: AFEQT predictive means over time (by cluster)</li>
<li>Pending data collection</li>
</ul></li>
<li>AF-CTRS Score
<ul>
<li>Meeting with MB and JS</li>
</ul></li>
</ul>
</section>
</section>
<section id="april-1-2024" class="level1">
<h1>April 1, 2024</h1>
<section id="updates-1" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="updates-1">Updates</h2>
<ul>
<li>AFEQT/SDOH
<ul>
<li>Additional analysis on cross-sectional cohort</li>
<li>Conditional/clustered analyses for predictive means</li>
</ul></li>
<li>Atrial Fibrillation Composite Treatment Response Score (AF-CTRS)
<ul>
<li>AF-CTRS to be used to predict efficacy of treatment (regardless of burden % or AFEQT score)</li>
<li>Output is algorithm that ties “response parameters” to a dose-responsive score <span class="math inline">\(\rightarrow\)</span> validate in AF Registry</li>
<li>Algorithm: ECG findings, AFEQT score, clinical parameters, MCT/ILR (require complex MI)</li>
</ul></li>
</ul>
</section>
<section id="ecg-prediction-of-structural-and-ion-channel-mutants-in-af" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="ecg-prediction-of-structural-and-ion-channel-mutants-in-af">ECG Prediction of Structural and Ion Channel Mutants in AF</h2>
<p>This is an expansion of the <strong>ECG/TTN</strong> project, which found <u>ECG could feasibly provide above-chance accuracy in identify TTN LOF variants</u>. Now, expand to additional <em>wild-types</em> and <em>mutants</em>…</p>
<ul>
<li>Sarcomeric/structural variants (e.g.&nbsp;TTN, PITX2, LMNA, NUP155, GJA1/5)</li>
<li>Ionic channel variants (e.g.&nbsp;KCNE1-5, KCNQ1, SCN5A)</li>
<li>No known genetic contributors (excluding any patient with non-benign VUS in any above genes <span class="math inline">\(\pm\)</span> SNPs associated with polygenic risk for AF)</li>
</ul>
<p>Models, trained on ECG in sinus rhythm in those with known AF, <u>will need validation</u>. However, should publication happen <u>before or after</u> validation (e.g.&nbsp;with TOPMed/UK Biobank)</p>
</section>
</section>
<section id="march-29-2024" class="level1">
<h1>March 29, 2024</h1>
<section id="update" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="update">Update</h2>
<ul>
<li>AFL/FH:
<ul>
<li>Revisions complete, pending table placement considerations</li>
</ul></li>
<li>ECG/TTN:
<ul>
<li>Results presented to AFGen fellows</li>
<li>Revised project proposal in conjunction with AFGen (for UK Biobank)</li>
<li>Potential K23 aim</li>
</ul></li>
<li>AFEQT/SDOH:
<ul>
<li>Additional analysis on cross-sectional cohort</li>
<li>Conditional/clustered analyses for predictive means</li>
</ul></li>
</ul>
</section>
<section id="ecgttn" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="ecgttn">ECG/TTN</h2>
<p>Can ECG discriminate between <u>wild-type AF and mutant-type</u> with known, causal/associated genes?</p>
<ul>
<li>Increase scope to include additional monogenic genes</li>
<li>Train to target individual cases based on genetic variant</li>
<li>Train to classify those with <em>no</em> genetic mutations</li>
<li>Validate in UK Biobank</li>
</ul>
</section>
</section>
<section id="march-25-2024" class="level1">
<h1>March 25, 2024</h1>
<section id="updates-2" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="updates-2">Updates</h2>
<ul>
<li>ECG/TTN
<ul>
<li>Draft analysis complete, presenting to AFGen fellows on 3/27</li>
<li>Brainstorm scope of paper</li>
</ul></li>
<li>AFEQT/SDOH
<ul>
<li>Positive feedback from co-author on results draft</li>
<li>Potential expansion of “baseline” cohort (AFEQT)</li>
<li><dl>
<dt>Paper writing (new draft in progress), expect 2 weeks for first draft <span class="math inline">\(\pm\)</span> new analyses</dt>
<dd>
AFL/FMHX
</dd>
</dl></li>
<li>Revisions in progress, responses likely Thursday/Friday</li>
</ul></li>
</ul>
</section>
</section>
<section id="february-26-2024" class="level1">
<h1>February 26, 2024</h1>
<section id="updates-3" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="updates-3">Updates</h2>
<ul>
<li>AFEQT/SDOH
<ul>
<li>data cleaning started on JBVA data, largest hurdle is getting NDI data from addresses</li>
<li>have rough draft of analysis plan, with plans to get early feedback from Alvaro (standing meeting next week)</li>
</ul></li>
<li>ECG/TTN
<ul>
<li>Filtering pipeline effective for TTN variant, will review with Choi (have 8-10 P/LP variants that are potentially LOF)</li>
<li>Presentation on 3/27 with AFGen group</li>
</ul></li>
<li>K23
<ul>
<li>Feedback from Benjamin: after refining genetics aim, can consider implementation in UK Biobank</li>
<li>Feedback from Ranjan: need to consider the sponsorship/mentorship committee at Utah (e.g.&nbsp;Martin Tristani-Firouzi, Rob McLeod, Jared Bunch, etc)</li>
</ul></li>
</ul>
</section>
<section id="afeqtsdoh-paper-proposal" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="afeqtsdoh-paper-proposal">AFEQT/SDOH paper proposal</h2>
<ol type="1">
<li>Table = cohort description by baseline AFEQT scores, with sections on clinical covariates, social determinants, etc</li>
<li>Table = Cohort description with statistical testing by AFEQT score changes</li>
<li>Figure = AFEQT score breakdown in the population, with subscales (bar graph), showing score changes</li>
<li>Table = regression model for SDOH factors and AFEQT change</li>
<li>Table = mediation analysis for SDOH factors, treatment strategies, and AFEQT scores</li>
<li>Figure = forest plot of interaction of the <strong>highest yield</strong> SDOH marker and AFEQT score changes</li>
</ol>
</section>
</section>
<section id="february-12-2024" class="level1">
<h1>February 12, 2024</h1>
<section id="whole-exome-sequencing" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="whole-exome-sequencing">Whole Exome Sequencing</h2>
<p>In light of project-focus on deleterious TTN variants and a/w ECG features, needed a statistically appropriate method for variant selection.</p>
<ul>
<li>Using <em>variant effect prediction</em> software, including loss-of-function predction, Polyphen, SIFT, and phenotypic data, <strong>created software pipeline to analysis WES</strong></li>
<li>Each WES VCF, roughly 50-100 Mb in size, can be annotated to 500-800 Mb of annotations based on dozens of classifier models (called <strong>consequences</strong>)</li>
<li>These <strong>consequence</strong> additions are part of Choi’s recommendation for having agreement in &gt;80% of predictions to serve as deleterious variant “ground truth”</li>
<li>Need to manually adjudicate the TTN variants, thus far is 1) reproducible with different prediction cut-offs, 2) identifies some <em>miRNA</em> sequenes that may be consequential, 3) has high degree of confidence in the LOF and INDEL variants</li>
</ul>
<div class="callout callout-style-default callout-tip callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Tip
</div>
</div>
<div class="callout-body-container callout-body">
<p>Variant pipeline requires combination of cluster + local software, and is stable for running new variants PRN.</p>
</div>
</div>
</section>
<section id="afeqtsdoh" class="level2">
<h2 class="anchored" data-anchor-id="afeqtsdoh">AFEQT/SDOH</h2>
<ol type="1">
<li>VA data cleaning, expected 10-12 hours of cleaning</li>
<li>Results and tables to be created throughout this week, pending VA data</li>
<li>Need to decide on key figures and tables for the paper</li>
<li>Expect manuscript draft (dependent on results) <em>week of 2/23</em></li>
</ol>
</section>
</section>
<section id="january-29-2024" class="level1">
<h1>January 29, 2024</h1>
<section id="updates-4" class="level2">
<h2 class="anchored" data-anchor-id="updates-4">Updates</h2>
<ul>
<li>AFEQT/SDOH: CDW data pull pending, may be able to get all ICD codes <span class="math inline">\(\pm\)</span> medicaions</li>
<li>AFL/FH: Completing revisions to paper, sharing in next 2-3 days</li>
<li>ECG/TTN: focusing on variant pipeline currently</li>
</ul>
</section>
</section>
<section id="january-21-2024" class="level1">
<h1>January 21, 2024</h1>
<section id="ecgttn-project" class="level2">
<h2 class="anchored" data-anchor-id="ecgttn-project">ECG/<strong>TTN</strong> Project</h2>
<ul>
<li>Variant annotation pipeline drafted, with classification based on series of annotation tools used in parallel</li>
<li>AFGen fellowship: suggested minimum threshold of &gt;80% of the ~30 annotation system to be in agreement for tentative “ground truth” for the “severe” or LOF variants</li>
<li>Algorithm for mis-sense variants being developed</li>
</ul>
</section>
<section id="ecgttn-project-1" class="level2">
<h2 class="anchored" data-anchor-id="ecgttn-project-1"><strong>ECG</strong>/TTN Project</h2>
<ul>
<li>Have developed an <strong>alpha</strong> version of end-to-end pipeline for sinus beat analysis</li>
<li>Utilizing time varying machine learning layers to add power for <em>repeat</em> samples and <em>time-to</em> samples</li>
<li>UK Biobank may be a fair option for validation if race/ethnicity factors isn’t a major concern</li>
</ul>
</section>
</section>
<section id="january-19-2024" class="level1">
<h1>January 19, 2024</h1>
<section id="aflfh-revisions" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="aflfh-revisions">AFL/FH Revisions</h2>
<ul>
<li>Reviewer #1: No comments as had no initial comments or suggestions on initial submission</li>
<li>Reviewer #2: Complimentary of revisions and thought the concerns had been <em>satisfactorily addressed</em></li>
<li>Reviewer #3: Thankful for <em>thoughtful and detailed responses</em> to questions</li>
<li>EDI Reviewer: Thought comments had been <em>appropriately addressed</em></li>
</ul>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p><em>A statistical reviewer has entered the fray</em></p>
</div>
</div>
</section>
<section id="statistical-review" class="level2 smallest">
<h2 class="smallest anchored" data-anchor-id="statistical-review">Statistical Review</h2>
<ul>
<li>“The study is described in the abstract as a prospective, observational cohort but it seems to be a cross-sectional analysis. Can the authors please clarify the design.”</li>
<li>“The abstract also notes that the primary independent variable is family history of atrial fibrillation in first degree relatives, and that secondary independent variables include self-reported race-ethnicity and sex. The statistical analysis section indicates that the secondary exposure includes race-ethnicity, sex, and potential modifiable risk factors. The title suggests that interest lies in the association between family history and early onset atrial flutter and that race/ethnicity might be a potential confounder as the title suggests exploring the association in racial/ethnic subgroups. Nothing is mentioned about sex. The focus of the paper needs to be made very clear throughout.”</li>
<li>“The statistical analysis section is confusing.”</li>
<li>“The section indicates that multivariate (should be multivariable) logistic regression analysis was used with sequential adjustment for sex, race, smoking, etc. Sex and race should not be listed as potential confounders (per point above – please clarify the role of these variables in the analysis).”</li>
<li>“The analysis in Tables 2 are unadjusted but very problematic given the 10-year age difference in individuals with early- versus late-onset atrial flutter. These should be, at a minimum, adjusted for age.”</li>
<li>“The statistical analysis section indicates the use of multivariable logistic regression, mixed effects models and interaction models – but the analysis presented are almost all unadjusted.”</li>
<li>“Table 3 stratifies by family history. Why stratify by the primary independent variable? The sample size is also too small in the family history group.”</li>
<li>“The statistical analysis approach needs to be better organized to address the key questions.”</li>
<li>“In the abstract, please clarify that the odds ratios that are reported are all unadjusted, if this is the case.”</li>
<li>“Throughout, the authors refer to effect size when they mean effect – it might even be clearer to refer to odds ratios rather than effects here.”</li>
<li>“The language in the clinical implications section needs to be toned down as it infers causality – e.g., individuals with a family history and individuals who self-identify as Black had higher odds.”</li>
<li>“Lastly, I am not sure the Figure is needed.”</li>
</ul>
</section>
<section id="statistical-response" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="statistical-response">Statistical Response</h2>
<ul class="task-list">
<li><label><input type="checkbox">Study design: Although patients were enrolled in a prospective, observational manner, the analysis is cross-sectional (no <em>semiparametric</em> elements)</label></li>
<li><label><input type="checkbox">Variable roles: Independent exposure is family history, while race/ethnicity is a potential confounder in a highly diverse cohort. Sex + other risk factors are secondary exposures.</label></li>
<li><label><input type="checkbox">Statistical analysis section: Word-choice and revision are clarified. Specific models are labeled explicitly in results and tables.</label></li>
<li><label><input type="checkbox">Table 3: Concern about analyzing by family history strata. Table 1 can instead by changed to strata by exposure (instead of outcome). Can include subgroup analysis as alternative table for all variables.</label></li>
<li><label><input type="checkbox">Odds Ratios: Language changed to reflect intent (e.g.&nbsp;<strong>effect</strong> and <strong>OR</strong> instead of <strong>effect size</strong>).</label></li>
<li><label><input type="checkbox">Clinical Implications: Language toned down to reflect observational nature of study.</label></li>
<li><label><input type="checkbox">Figure: Justification on why family history and pedigrees are relevant even though this is observational study. Consideration for supplement.</label></li>
</ul>
</section>
</section>
<section id="january-8-2024" class="level1">
<h1>January 8, 2024</h1>
<section id="afeqtsdoh-background" class="level2">
<h2 class="anchored" data-anchor-id="afeqtsdoh-background">AFEQT/SDOH background</h2>
<ul>
<li>Currently there is a push for social determinant research to be integrated into atrial fibrillation research <span class="citation" data-cites="Benjamin2023">(<a href="#ref-Benjamin2023" role="doc-biblioref">Benjamin et al. 2023</a>)</span></li>
<li>Social determinants in AF research has been predominantly oriented at diagnosis and clinical outcomes <span class="citation" data-cites="Essien2021">(<a href="#ref-Essien2021" role="doc-biblioref">Essien et al. 2021</a>)</span></li>
<li>Neither race, ethnicity, or socioeconomic differences have significant research progress in AF diagnosis or management [Norby2021]</li>
</ul>
<p><em>We have a rare opportunity to evaluate how <strong>symptom burden</strong> is affected by SDOH factors in a multi-ethnic population.</em></p>
</section>
</section>
<section id="december-18-2023" class="level1">
<h1>December 18, 2023</h1>
<section id="afeqtsdoh-updates" class="level2">
<h2 class="anchored" data-anchor-id="afeqtsdoh-updates">AFEQT/SDOH updates</h2>
<ul class="task-list">
<li><label><input type="checkbox" checked="">Data collection has been completed for UIC patients</label></li>
<li><label><input type="checkbox">Initial analyses to be performed this week (target: 12/22/23)</label></li>
<li><label><input type="checkbox">Will need to collect JBVA data = will need Jerry’s help… will send data collection template</label></li>
<li><label><input type="checkbox">Continue to collect additional AFEQTs from Hispanic/Latinx individuals = need to speak with Annette about focusing on Spanish-speakers</label></li>
</ul>
</section>
</section>
<section id="december-11-2023" class="level1">
<h1>December 11, 2023</h1>
<section id="ecg-ttn-variant-project" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="ecg-ttn-variant-project">ECG &amp; TTN Variant Project</h2>
<p><u>Background</u>: VUS/LP/P TTN variants are associated with increased incidence of EOAF. The mechanism is not fully understood.</p>
<p><u>Aims</u>: Identify if VUS/LP/P TTN variants may cause structural atrial changes (an atrial myopathy) that is identifiable or ECG, or other related conduction changes. We <span class="red">aim to identify if VUS/LP/P TTN variants can be classified on ECG using signal processing techniques</span></p>
<p><u>Approach</u>: Utilize sinus rhythm ECGs from patients with WES/CGS with paroxysmal AF. Develop beat-by-beat algorithm to identify features that may be associated with VUS/LP/P TTN variants.</p>
</section>
<section id="ecg-data" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="ecg-data">ECG data</h2>
<ul>
<li><code>n = 298</code> patients were available and had ECG in sinus rhythm at some point from 2010 to 2023</li>
<li>This amounted to <code>n = 7953</code> unique ECG that were thought to be in sinus rhythm</li>
<li>Using a wavelet-decomposition approach, identified morphology of P wave, QRS complex, and T wave and created single-beat structures</li>
<li>Using these <code>n = 51493</code> beats, there were <code>n = 6202</code> <span class="red">cases</span> (which represents <code>n = 36</code> individuals with potential pathogenic TTN variants (based on SIFT/PolyPhen scores)</li>
</ul>
</section>
<section id="model" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="model">Model</h2>
<ul>
<li>Sequential convoluted neural network approach</li>
<li>Masking layers to control for <code>0</code> values (representing non-depolarized intervals and padded sequences)</li>
<li>Use combination of <code>relu</code> (rectified linear activation) and increasing <code>dense</code> layers (with signal drop out)</li>
<li>Categorize using a <code>softmax</code> approach for classification between case/control</li>
<li>Initial training/testing to <strong>first-beat-only</strong> approach to avoid sample bias</li>
<li>Trained for 10 <code>epochs</code> based on model accuracy and minimizing <code>loss</code> on each repetition</li>
</ul>
</section>
<section id="results" class="level2">
<h2 class="anchored" data-anchor-id="results">Results</h2>
<p><span class="red">Training</span></p>
<ul>
<li>Training accuracy was <code>88%</code> with only <code>2%</code> loss after total of 10 steps</li>
</ul>
<p><span class="red">Testing</span></p>
<ul>
<li>Testing accuracy was <code>85%</code> with <code>13%</code> loss</li>
<li>Example: single beat (control) may have a confidence of <code>75%</code> in being a control</li>
</ul>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../figures/keras-accuracy-121023.png" class="img-fluid figure-img"></p>
<figcaption>First 10 epochs of training</figcaption>
</figure>
</div>
</section>
<section id="next-steps" class="level2">
<h2 class="anchored" data-anchor-id="next-steps">Next steps</h2>
<ol type="1">
<li>Confirm the TTN variant cases, trialing different definitions of pathological beats</li>
<li>Extend analysis to full data-set</li>
<li>Modify approaches in terms of machine learning “layers”</li>
</ol>
</section>
</section>
<section id="december-4-2023" class="level1">
<h1>December 4, 2023</h1>
<section id="hrs-abstract-proposals" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="hrs-abstract-proposals">HRS abstract proposals</h2>
<ul>
<li><span class="red">TTN variant prediction</span>: Would like to submit abstract, currently training an ECG-based model to classify variants (pathogenic vs.&nbsp;controls). Training data of approximately 50-60k sinus beats, with 500-700 “pathogenic TTN cases”. <em>Data augmented by breaking ECGs into individual beats</em></li>
<li><span class="red">AFL &amp; Family History</span>: Abstract to be submitted, with equivalent co-authors as manuscript submission</li>
<li><span class="red">TTN variants and EP outcomes</span>: Gursuk/James and others wanted to pursue EP-based outcomes. Have ~9 cases of PVI performed on TTN variants that are non-benign (e.g.&nbsp;VUS or LP/P).</li>
</ul>
</section>
</section>
<section id="november-20-2023" class="level1">
<h1>November 20, 2023</h1>
<section id="aflfh-updates" class="level2">
<h2 class="anchored" data-anchor-id="aflfh-updates">AFL/FH updates</h2>
<ul class="task-list">
<li><label><input type="checkbox">Feedback? Should we request for an extension?</label></li>
<li><label><input type="checkbox">Circulate amongst co-authors</label></li>
</ul>
<p><em>Why do we believe that Black v. Non-Black individuals had higher rates of AFL despite balanced risk factors?</em></p>
</section>
<section id="af-registry-and-ccts-data" class="level2">
<h2 class="anchored" data-anchor-id="af-registry-and-ccts-data">AF Registry and CCTS Data</h2>
<ol type="1">
<li>Have all EMR data from 2010 to 2023 now, which increased file sizes by 30% (now over 70 Gb), and increased <code>n = 140k</code> to <code>n = 210k</code></li>
<li>Have all ECG for same patient population (approximately 40 Gb in XML format)</li>
<li>Created <code>parquet</code> format to utilize Apache Arrow for I/O</li>
</ol>
<ul class="task-list">
<li><label><input type="checkbox">Development of filtering pipeline for MRNs, diagnoses, medications, procedures, etc.</label></li>
</ul>
</section>
<section id="afeqtsdoh-1" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="afeqtsdoh-1">AFEQT/SDOH</h2>
<p>Which key variables should be included for evaluation? Some of these have been studied, but not in context of AFEQT/symptom burden</p>
<ul>
<li>ECG factors at baseline compared to at time of repeat evaluation (heart rate, rhythm)</li>
<li>Maintenance of sinus rhythm (percent burden of AF, frequency of ECG in sinus rhythm)</li>
<li>Medication changes or intensification (addition of anti-arrhythmic)</li>
<li>Referral to cardioversion or ablation</li>
<li>Administration of anticoagulation therapy</li>
</ul>
</section>
<section id="ecgttn-1" class="level2">
<h2 class="anchored" data-anchor-id="ecgttn-1">ECG/TTN</h2>
<p>Timeline for abstract submission to HRS (12/15):</p>
<ul>
<li>Complete initial model training on pathogenic TTN cases</li>
<li>Exclude other TTN variants from analysis to have <em>cleaner</em> controls</li>
<li>Identify appropriate metrics (e.g.&nbsp;AUC, net classification, etc)</li>
<li>Identify if segments of ECG are more highly weighted (variance importance)</li>
</ul>
</section>
</section>
<section id="november-13-2023" class="level1">
<h1>November 13, 2023</h1>
<section id="updates-5" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="updates-5">Updates</h2>
<ul class="task-list">
<li><label><input type="checkbox">AFL/FH paper revisions before resubmission (<strong>deadline 12/05</strong>), would like feedback in next 3-4 days, then <em>circulate to co-authors</em></label></li>
<li><label><input type="checkbox">Clean CCTS/AF Registry data over next 2 weeks, with anticipation of AFEQT being complete in that time-frame</label></li>
<li><label><input type="checkbox">Submit rank list for EP fellowship on Wednesday 11/15</label></li>
<li><label><input type="checkbox">Meetings with Emilia Benjamin and Seung H. Choi in next 2-3 weeks to discuss AF phenotyping project</label></li>
<li><label><input type="checkbox">PRS calculations with SNPs from NW</label></li>
</ul>
</section>
</section>
<section id="october-26-2023" class="level1">
<h1>October 26, 2023</h1>
<section id="aflfh-simple-revisions" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="aflfh-simple-revisions">AFL/FH Simple Revisions</h2>
<ul>
<li>Conform with AHA guidelines on reporting health differences by races</li>
<li>Including <strong>Clinical Perspective</strong> section</li>
<li>More detailed methods on patient selection and recruitment</li>
<li>Directionality when discussing associations (e.g.&nbsp;weight versus <em>increased weight</em>)</li>
<li>Add definitions for social drinking</li>
<li>Clarify anti-hypertensive agents and selection</li>
<li>Clarify results and statistical methods to match discussion</li>
<li>Requesting how and why we defined EOAFL as age &lt; 65</li>
</ul>
</section>
<section id="aflfh-complex-revisions" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="aflfh-complex-revisions">AFL/FH Complex Revisions</h2>
<ol type="1">
<li>Concern is the definition of family history to include AF, HF, and CVA, as is <strong>inconsistent</strong> with the idea of EOAFL as a separate entity. <em>Unsure how to respond</em></li>
<li>Can occult AF be analyzed as a comparative group? <em>Unsure about an additional dataset. However would strengthen the paper if we had a matched cohort as sensitivity analysis.</em></li>
<li>In those without stated FH, the FDR were not contacted - reviewers are concerned that frequently patients are unaware of FH, and recommended contacting every study participant. <em>Two paths: 1) Ignore this and state we had a rigorous approach to FH questionnaires; 2) Pursue the ~300 patients with EOAFL and contact FDR to limit confounding.</em></li>
<li>Provide a hypothesis and reasoning for the choice of variables. Particularly that of race. <em>Would refer to genetics and poor representation in other studies.</em></li>
<li>Concern for why we emphasized difference between Black race and White/Latinx race. <em>Would compare Black to non-Black instead</em></li>
<li>Hypothesis on inverse relationsihp between HTN/BB/CCB and EOAFL risk</li>
</ol>
</section>
</section>
<section id="october-23-2023" class="level1">
<h1>October 23, 2023</h1>
<section id="updates-6" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="updates-6">Updates</h2>
<p>AFEQT/SDOH: Current focus is on obtaining updated AFEQT, led by <em>JS</em>, with goal to complete AFEQT in next several weeks. In parallel, using EMR-based records to evaluate clinical outcomes. It will take roughly 30 hours of statistical work to incorporate the clinical and AFEQT data (roughly 1-2 weeks), once the AFEQTs are done.</p>
<p>AFL/FH: Our good news was that JAHA is willing to review our paper again with revisions. A majority of these will likely be completed within the next 4-5 days (deadline = 10/31) but have to decide on if new/additional FH data collection is needed. The most challenging critique was the potential bias in FH reporting during initial screening. To overcome this, the most thorough option would be to obtain FH data from every patient to avoid systemic errors/bias. However, a limited approach of verifying FH in just the EOAFL may be reasonable. Would like everyone’s thoughts here.</p>
<p>Phenotype AF: Two students have joined this effort, <em>SS</em> and <em>SD</em>, who will be working on initially on the DSP and ML aspects. Current priority is to finish training the models using the TTN data. Unfortunately, this has been delayed with Boards and interviews, but hope to speed this back up in the next 1-2 weeks.</p>
<p>AF/PRS: For our evaluation of the PRS in AF in ethnic cohorts, we are meeting with the eMERGE group (NW). We will need to understand if the PRS can be validated in our UIC cohort, and if there is an existing mechanism to integrate a GIRA with our own EHR.</p>
</section>
</section>
<section id="september-18-2023" class="level1">
<h1>September 18, 2023</h1>
<section id="emr-data" class="level2">
<h2 class="anchored" data-anchor-id="emr-data">EMR Data</h2>
<p>This has been somewhat streamlined to <span class="green">extract data</span> from the CCTS data pull, with full cohort results returned from CCTS.</p>
<ol type="1">
<li>Need to extract AF registry patients and add SDOH, medication intensification, and procedure reports.</li>
<li>Working with Tofovic/Nischal to extract echocardiogram data</li>
<li>Extracted roughly 8k ECG, digitized, and segmented to individual beats, which provides roughly 40k samples for testing amongst the ~300 WES patients</li>
<li>Working with Chen to complete power calculations based on our population and current literature</li>
</ol>
</section>
<section id="afeqtsdoh-2" class="level2">
<h2 class="anchored" data-anchor-id="afeqtsdoh-2">AFEQT/SDOH</h2>
<ul>
<li>Spoke with Jerry/Muriel about re-organizing the AFEQT data for manuscript</li>
<li>Will need to complete roughly <code>n=600</code> follow-up AFEQT</li>
<li>Power calculation suggest would need <code>n=200</code> Hispanic/Latinx</li>
<li>Timeline TBD (expect ~ 4-6 weeks)</li>
<li>Subsequently will need to extract patient outcomes (intensification, burden) from CCTS pull</li>
<li>Adding Angela Hussain to data collection as student researcher</li>
<li>Check in with Annette to help with Spanish-speaking requirements</li>
</ul>
</section>
<section id="afgenetics" class="level2">
<h2 class="anchored" data-anchor-id="afgenetics">AF/Genetics</h2>
<p>Met with AFGen group (Patrick Elinor, Emilia Benjamin), encouraged computational approach in genetics. Currently working to filter and annotate our WES dataset, but expanding to novel gene regions (e.g.&nbsp;neurocardiac receptors) that are not included in general cardiomyopathy/arrhythmia panels.</p>
<ol type="1">
<li>Variant evaluation/assessment of different reference genomes</li>
<li>Variant filtering based on SIFT/PolyPhen</li>
<li>Annotation based on multiple sources (USCS, NCBI, etc)</li>
<li>Inclusion of broader, related genes (e.g.&nbsp;<strong>ADRB2</strong> not included in recent evaluation)</li>
</ol>
</section>
</section>
<section id="september-11-2023" class="level1">
<h1>September 11, 2023</h1>
<section id="afeqt" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="afeqt">AFEQT</h2>
<p>Current manuscript has <code>n=453</code> individuals split amongst races, with some strength of assocation b/w SES/SDOH and ∆AFEQT.</p>
<ul>
<li>Have <code>n=760</code> baseline AFEQT data points, but need roughly <code>n=600</code> additional follow-up AFEQT scores</li>
<li>Have <code>n=1506</code> currently in AF Registry, thus <code>743</code> more patients that can be added</li>
<li>Would allow us to be powered for mediation analysis, and stratified analyses</li>
<li>Able to incorporate <span class="orange">intensification</span> and <span class="orange">recurrence</span> rates now, as well as census tract (vs.&nbsp;zip) and language barriers now</li>
</ul>
<hr>
<p><u>Next</u>:</p>
<ol type="1">
<li>Complete the additional <code>600</code> AFEQT follow-ups</li>
<li>Extract data from CCTS to “flush out” registry data</li>
<li>More specific/appropriate analyses for SDOH including <span class="blue">mediation</span></li>
</ol>
</section>
<section id="prs-for-afemerge" class="level2">
<h2 class="anchored" data-anchor-id="prs-for-afemerge">PRS for AF/eMERGE</h2>
<ol type="1">
<li>We are meeting with NW and the eMERGE group to validate PRS in AA/HL population here at UIC this week.</li>
<li>Met with Hua Yua Chen to go over PRS power calculations</li>
<li>Generating multiple sample size/power calculations based on C-statistics seen in other AF/PRS papers → however may not apply equivalently in AA/HL population as primarily derived from Caucasian populations</li>
</ol>
</section>
</section>
<section id="september-8-2023" class="level1">
<h1>September 8, 2023</h1>
<section id="afgenetics-1" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="afgenetics-1">AF/Genetics</h2>
<p>Completed <span class="grey">computational genetics coursework</span> on variant analysis. Although some work has already been done on this, going through the basics will allow us to have in-house <span class="green">variant annotation pipeline</span></p>
<ol type="1">
<li>Uploaded our ~2G of patients to cluster</li>
<li>Generated functions to <span class="green">split VCF</span> by individual case</li>
<li>Identified variant locations</li>
</ol>
<p><u>Next</u>: Evaluate variants using <strong>PolyPhen</strong> and <strong>SIFT</strong> databases</p>
</section>
<section id="ecgpaf" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="ecgpaf">ECG/pAF</h2>
<p>Recently was asked to review/adjudicate ECG for the PhysioNet challenge on AF prediction algorithms. In line with my interest at phenotying AF based on ECG markers.</p>
<ol type="1">
<li>Segmentation of individual beats has been automated</li>
<li>Augmenting the data to individual beats (similar to <span class="citation" data-cites="vandeLeur2021">van de Leur et al. (<a href="#ref-vandeLeur2021" role="doc-biblioref">2021</a>)</span>)</li>
</ol>
<p><u>Next</u>: Training on labeled <span class="orange">AF patients with TTN variants</span></p>
<hr>
<div id="fig-annotated-ecg" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-annotated-ecg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/shiva-ecg-annotated.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-annotated-ecg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Annotation of individual beats
</figcaption>
</figure>
</div>
</section>
<section id="afsdoh" class="level2">
<h2 class="anchored" data-anchor-id="afsdoh">AF/SDOH</h2>
<ul>
<li>eMERGE network meeting this week</li>
<li>Processing UIC/CCTS data</li>
<li>Uploaded January to June ECG data from MUSE</li>
<li>Will still need updates from CCTS for 2023 remaining clinical data</li>
<li>Have to solidify definitions for <span class="red">therapy intensification</span> and <span class="red">AF outcomes</span></li>
</ul>
</section>
</section>
<section id="august-21-2023" class="level1">
<h1>August 21, 2023</h1>
<section id="phenotyping-af" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="phenotyping-af">Phenotyping AF</h2>
<p><em>Identifying phenotypes of AF that are at risk of progression (e.g.&nbsp;developing higher burden, persistent AF, TICM, etc)</em></p>
<ol type="1">
<li>Extracted clinical covariates for ~6k patients, stratified by time/onset (this is ~7 million diagnoses)</li>
<li>Extracted medications by time/duration of prescription
<ol type="a">
<li>Filtered down to all cardiovascular medications</li>
<li>Classified medications that are relevant to AF (AC/AP, AAD)</li>
<li>2.5 million prescriptions were analyzed</li>
</ol></li>
<li>Evaluated ECG related to this population and initiated segmentation/feature extraction</li>
<li><strong>Goals</strong>: Pull procedure reports to assess burden, and process text-based echocardiography data</li>
</ol>
</section>
<section id="ecg-and-genetic-variants" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="ecg-and-genetic-variants">ECG and Genetic Variants</h2>
<ol type="1">
<li>Completed the software needed to annotate ECGs on single leads</li>
<li>Will have to evaluate efficacy amongst 12-lead systems (e.g.&nbsp;5000 x 12 array)</li>
<li>Using <span class="green">semantic segmentation</span> approach to classify waveforms</li>
<li>Have started <span class="green">cluster-based</span> pipeline development for re-calling VCF to identify variants based on…
<ol type="a">
<li>Multiple different population panels (different allele frequencies)</li>
<li>Filter on read depth and quality</li>
<li><em>Will need to learn about other filtering approaches</em></li>
</ol></li>
</ol>
</section>
<section id="sdohafhsr" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="sdohafhsr">SDOH/AF/HSR</h2>
<ul>
<li>Current knowledge gap is interaction between race/ethnicity and socioeconomic status in management of AF <span class="citation" data-cites="Essien2021">(<a href="#ref-Essien2021" role="doc-biblioref">Essien et al. 2021</a>)</span></li>
<li>Similar to AF Registry + Cluster, can use the definition of <strong>treatment intensification</strong> as part of health care differences</li>
<li>Parent project to AF/language</li>
<li>Unlike other cohorts (e.g.&nbsp;ARIC had <code>n=3200</code> but only <code>n=500</code> were Black), have more diverse population pool</li>
</ul>
</section>
</section>
<section id="august-14-2023" class="level1">
<h1>August 14, 2023</h1>
<section id="aflfh" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="aflfh">AFL/FH</h2>
<p>Completed revisions to paper with tentative plans for re-submission.</p>
<ol type="1">
<li>Called <code>n=31</code> families to confirm AFL/AF diagnoses and clarify CHF and CVA histories</li>
<li>Re-analyzed pedigrees with multiple definitions of +FH in FDRs as a <em>sensitivity analysis</em></li>
<li>Major findings remain same (and <strong>robust</strong> to adjustment):
<ul>
<li>+FH doubles risk for “premature” or earlier onset of AFL</li>
<li>EOAFL as corrolary increases likelihood of AF and AFL in FDRs</li>
<li>Black race a/w EOAFL, partially mediated through OSA/ETOH (most prominent in this subgroup)</li>
</ul></li>
<li>Resubmission to… <strong>JAHA</strong>, <strong>AHJ</strong>, <strong>IJC</strong>, <strong>BMJ Heart</strong></li>
</ol>
</section>
<section id="afsdoh-1" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="afsdoh-1">AF/SDOH</h2>
<ul>
<li><code>n=4537</code> patients (still missing updated CCTS pull of about 2k)</li>
<li>92% have NDI via census tract</li>
<li>52% have social histories completed</li>
<li>81% are English speaking, 10% Spanish</li>
<li>Medications/Labs/Vitals/Visits/Comorbidities being evaluated currently</li>
</ul>
</section>
<section id="af-phenotyping" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="af-phenotyping">AF Phenotyping</h2>
<div id="fig-pwave-detection" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pwave-detection-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/saclova-2022-p-wave-detection.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pwave-detection-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: P wave detection
</figcaption>
</figure>
</div>
<p>Working on ECG beat segmentation and annotation using machine learning approach across multiple leads. Will be applicable across ECG-based projects. Necessary step prior to creating median beats.</p>
</section>
</section>
<section id="july-31-2023" class="level1">
<h1>July 31, 2023</h1>
<section id="aflfh-reviewer-concerns" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="aflfh-reviewer-concerns">AFL/FH Reviewer Concerns</h2>
<ol type="1">
<li>How many FDR were directly contacted?</li>
<li>What constitutes an “atrial arrhythmia” in family members? Often asymptomatic. Needs clear definition</li>
<li>Why LOAFL versus EOAFL, instead of EOAFL versus controls?</li>
<li>Why is FH/Race significant only in a single racial group?</li>
<li>Overstatement of importance of FH over <em>4q25</em> papers in isolated AFL</li>
<li>How are CVA events labeled as cardioembolic?</li>
<li>Why include CHF as part of arrhythmia risk factors in FDR?</li>
<li>Define excess ETOH usage</li>
<li>Pedigree needs to be labeled more specifically/informatively</li>
</ol>
</section>
</section>
<section id="july-24-2023" class="level1">
<h1>July 24, 2023</h1>
<section id="active-projects" class="level2">
<h2 class="anchored" data-anchor-id="active-projects">Active projects</h2>
<ol type="1">
<li>AFL/FH: Revisions to be addressed in next draft. Will identify additional journals to target.</li>
<li>HRV/CVD: EHJ</li>
<li>ECG/PAF: Active project involving computational approach at identifying sub-phenotypes of paroxysmal AF based on ECG-driven parameters <em>with subsequent work on clinical patterns and trajectory</em></li>
</ol>
</section>
<section id="phenotyping-af-1" class="level2">
<h2 class="anchored" data-anchor-id="phenotyping-af-1">Phenotyping AF</h2>
<p>Project is a “big data” project using over clinical data from 2010 to 2023, supplementing the AF Registry with <em>data mining</em> and <em>machine learning</em> approaches.</p>
<p>Composed of three elements:</p>
<ol type="1">
<li>ECG data in form 12-leads of 10 second signals, sampled at 500 Hz from 2010 to now</li>
<li>Clinical structured documentation including vitals, medications, labs, echo/study reports from 2010 to now</li>
<li>Clinical unstructured data from MD documentation</li>
</ol>
</section>
<section id="data-overview" class="level2">
<h2 class="anchored" data-anchor-id="data-overview">Data overview</h2>
<p>ECG data:</p>
<ul>
<li>500k ECGs collected from 2010 to 2013</li>
<li>70 Gb of raw XML data (converts to ~50 Gb of WFDB)</li>
</ul>
<p>Raw clinical data:</p>
<ul>
<li>43 Gb raw data (70% is from clinical notes alone)</li>
<li>145k unique patients with CVD diagnoses (cardiomyopathy, arrhythmia, vascular dz, etc)</li>
</ul>
</section>
<section id="af-specific-data" class="level2">
<h2 class="anchored" data-anchor-id="af-specific-data">AF-specific data</h2>
<p>ECG data:</p>
<ul>
<li>33k ECG with a diagnosis of an atrial arrhythmia</li>
<li>6k unique patients with a diagnosis of AF confirmed by ECG</li>
<li>Of those 6k patients, 95k ECG taken in total (both sinus + AF)</li>
</ul>
<p>Clinical data:</p>
<ul>
<li>Of the 6k patients c- AF on ECG, 3.9k have clinical data available from CCTS</li>
<li>Aggregate file size for this 3.9k is &lt; 1 Gb in total</li>
<li>600k ICD diagnoses, 4.5 million lab results, 500k visits, 1.5 million vital sign measurements, 2.9 million medications</li>
</ul>
</section>
<section id="project-status" class="level2">
<h2 class="anchored" data-anchor-id="project-status">Project status</h2>
<p>Currently have working pipeline for extracting relevant diagnoses, medications, labs, etc, for additional projects. Can email ashah282@uic.edu for questions or project proposals.</p>
<p>Can see current proposal <a href="https://asshah4.github.io/research/t32/phenotype-af.html">here</a></p>
</section>
</section>
<section id="june-12-2023" class="level1 page-columns page-full">
<h1>June 12, 2023</h1>
<section id="backbone-for-aims" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="backbone-for-aims">Backbone for aims</h2>
<ol type="1">
<li>Sympathetically-mediated AF occurs predominantly earlier in paroxysmal AF, in hearts with minimal scar burden.</li>
<li>To cause episodes of AF would require abnormalities in conduction from both intracardiac and extracardiac causes</li>
<li>Abnormalities from both intracardiac and extracardiac causes are either acquired or genetic</li>
<li>These factors may also lead to different rates of progression of AF</li>
</ol>
<p><u>Question</u>: Do certain genetic variants contribute to the progression of paroxysmal AF to persistent AF?</p>
</section>
<section id="trajectory-of-paf" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="trajectory-of-paf">Trajectory of pAF</h2>
<ul>
<li><span class="citation" data-cites="Kerr2005">Kerr et al. (<a href="#ref-Kerr2005" role="doc-biblioref">2005</a>)</span> from a Canadian Registry followed <code>n = 757</code> for ~8 years, 5 year recurrence was &gt;60%
<ul>
<li>Positive predictors = age, +AS, +MR, LAE, reduced LVEF</li>
<li>Negative predictor = rapid heart rate during AF</li>
</ul></li>
<li>Post catheter ablation, 5 years recurrence was 60% (1st ablation), and 20% after (&gt;2 ablations) (<span class="citation" data-cites="Takigawa2014">Takigawa et al. (<a href="#ref-Takigawa2014" role="doc-biblioref">2014</a>)</span>)
<ul>
<li>Predictors = LAVI and patient age</li>
</ul></li>
<li><span class="citation" data-cites="Holmqvist2015">Holmqvist et al. (<a href="#ref-Holmqvist2015" role="doc-biblioref">2015</a>)</span> noted progression based on age + increased heart rate in sinus (CHA2DS2VASc had minimal discriminatory power)</li>
<li><span class="citation" data-cites="Andrade2023">Andrade et al. (<a href="#ref-Andrade2023" role="doc-biblioref">2023</a>)</span> evaluated 3 year follow in ablation versus AAD, found <code>HR = 0.51</code></li>
<li><em>Acutely, post-cardiac surgery AF can be predicted by decreased HRV (<span class="citation" data-cites="Vesela2023">Veselá et al. (<a href="#ref-Vesela2023" role="doc-biblioref">2023</a>)</span>)</em></li>
</ul>
</section>
<section id="predictors-of-paf-progression" class="level2 smaller page-columns page-full">
<h2 class="smaller anchored" data-anchor-id="predictors-of-paf-progression">Predictors of pAF progression</h2>
<p><strong>CUrrently progression is poorly defined…</strong></p>
<ul>
<li>Clinical: need for rhythm control intervention (ECV, PVI), hospitalization for symptoms or heart failure, % burden increase by an inflection point</li>
<li>ECG: Decrease HF-HRV, increased P wave duration/dispersion, decreased P wave amplitude (during AF, decreased F-wave amplitude)</li>
</ul>
<p><strong>Persistent</strong> = <code>n</code> continous ECG in AF, diagnosis by cardiology, &gt;7 day duration of episodes (or less)</p>

<div class="no-row-height column-margin column-container"><div class="margin-aside">
<p>How should we define progression, rate of progression, and persistence?</p>
</div></div></section>
<section id="approach-for-evaluation" class="level2">
<h2 class="anchored" data-anchor-id="approach-for-evaluation">Approach for evaluation</h2>
<ol type="1">
<li>Establish clinical progression of disease with multiple <em>versions</em> of the definition</li>
<li>Obtain ECG data (segmentation to evaluate sinus beats)</li>
<li>Assess genetic variants ↔︎ ECG characteristics to compare with outcomes</li>
</ol>
<hr>
<div id="fig-ecg-genetic-manhattan" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ecg-genetic-manhattan-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/verweij-2020-manhattan.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ecg-genetic-manhattan-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Manhattan plot <span class="citation" data-cites="Verweij2020">Verweij et al. (<a href="#ref-Verweij2020" role="doc-biblioref">2020</a>)</span>
</figcaption>
</figure>
</div>
<hr>
<div id="fig-ecg-genetic-clusters" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ecg-genetic-clusters-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/verweij-2020-clusters.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ecg-genetic-clusters-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Cluster of genetic variants by ECG component <span class="citation" data-cites="Verweij2020">Verweij et al. (<a href="#ref-Verweij2020" role="doc-biblioref">2020</a>)</span>
</figcaption>
</figure>
</div>
<hr>
<div id="fig-ecg-variants-and-components" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ecg-variants-and-components-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/verweij-2020-ecg-and-variants.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ecg-variants-and-components-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5: Individual variants with ECG components <span class="citation" data-cites="Verweij2020">Verweij et al. (<a href="#ref-Verweij2020" role="doc-biblioref">2020</a>)</span>
</figcaption>
</figure>
</div>
</section>
<section id="next-steps-1" class="level2">
<h2 class="anchored" data-anchor-id="next-steps-1">Next steps</h2>
<ul class="task-list">
<li><label><input type="checkbox">Obtain ECG data (currently have from 2010 to 2017)</label></li>
<li><label><input type="checkbox">Segmentation approach for sinus beats</label></li>
<li><label><input type="checkbox">F-wave identification for AF beats</label></li>
<li><label><input type="checkbox" checked="">Contact Broad Institute about SNP-array data from previous GWAS</label></li>
<li><label><input type="checkbox">NLP approach to assessing paroxysmal versus persistent</label></li>
</ul>
</section>
</section>
<section id="june-5-2023" class="level1">
<h1>June 5, 2023</h1>
<section id="aims-progress" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="aims-progress">Aims Progress</h2>
<p>Goal: <strong>Identify characteristics that define sympathetically-mediated paroxysmal AF (as compared to scar-mediated pAF) in trajectory of paroxysmal → persistent</strong></p>
<ul>
<li>Pending mentorship meeting (dates requested for Alvaro, Amit, Dawood)</li>
<li>Met with Lampert - be much more specific in ANS factors in pAF
<ol type="a">
<li>Evaluate ECG during AF: coarse AF v. fine AF → triggering event (ADHF, surgery, etc…)</li>
<li>Evaluate ECG during sinus: P wave morphology/area/dispersion and AF burden</li>
<li><em>Evaluate candidate genes (ANS neurohormonal pathways) in Emory Cohort (which has high vagolysis burden) as initial study; validate these genes in AF registry with burden/rate of AF progression</em></li>
</ol></li>
</ul>
</section>
<section id="af-phenotyping-1" class="level2">
<h2 class="anchored" data-anchor-id="af-phenotyping-1">AF Phenotyping</h2>
<ul>
<li>Pending CCTS to upload data (hundreds of GB in size)</li>
<li>Complete setting up cluster computational pipeline for NLP of unstructured text</li>
<li>Specific goal is to identify timepoints: onset of AF, transition to persistent AF</li>
<li>Initiate analysis of ECG data
<ol type="1">
<li>Convert to annotation format (WFDB)</li>
<li>Identify median P waves per ECG</li>
</ol></li>
</ul>
</section>
</section>
<section id="may-22-2023" class="level1 page-columns page-full">
<h1>May 22, 2023</h1>
<section id="ecg-analysis" class="level2 smaller page-columns page-full">
<h2 class="smaller anchored" data-anchor-id="ecg-analysis">ECG analysis</h2>
<p><strong>K23 AIM #1</strong> is to identify paroxysmal AF into low-scar burden (and high vagolytic triggers) in AF registry.</p>
<ul class="task-list">
<li><label><input type="checkbox" checked="">Have imported <em>superset</em> of ECGs from MUSE (~500k ECGs) in XML/binary format (including raw signal, diagnoses, timing, etc), approximately ~1 TB</label></li>
<li><label><input type="checkbox" checked="">Convert XML format into digital signal… software developed (same as for <em>HRX</em> abstract), approximately ~250 GB</label></li>
<li><label><input type="checkbox">Analysis of ECG markers: high frequency HRV, P wave dispersion, P wave terminal forces, and GEH (Wilson’s vector gradient), and if possible to identify “coarseness” of AF</label></li>
<li><label><input type="checkbox">Split data into PRE/POST AF diagnosis, and identify control group from other ECGs in superset</label></li>
</ul>

<div class="no-row-height column-margin column-container"><div class="margin-aside">
<p>For EGM and ECG, working with Bhaumik to identify morphology of signals in machine learning project</p>
</div></div></section>
</section>
<section id="may-12-2023" class="level1">
<h1>May 12, 2023</h1>
<section id="updates-7" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="updates-7">Updates</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ol type="1">
<li>ECG data is being pulled from MUSE in XML format</li>
<li>CCTS data pull to be completed today</li>
<li>EGM signal processing - work in progress with goal of automated beat generation of multi-channel leads (R/C++ package)</li>
</ol>
</div><div class="column" style="width:50%;">
<ul class="task-list">
<li><label><input type="checkbox">Need storage space for ECG data (raw and processed), cluster allows only 50Gb for read/write storage (batch processing): &gt;100k ECG</label></li>
<li><label><input type="checkbox">CCTS data will need raw storage space: 10 years of patient charts + notes</label></li>
</ul>
</div>
</div>
</section>
</section>
<section id="may-8-2023" class="level1">
<h1>May 8, 2023</h1>
<section id="goals" class="level2">
<h2 class="anchored" data-anchor-id="goals">Goals</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ol type="1">
<li>Phenotyping of AF, as current grouping is inadequate. Focus on risk category changing from paroxysmal to persistent (baseline concept of multi-wavelet reentry)</li>
<li>ECG- and EGM-based analysis of AF as additive components</li>
<li>CARTO/Rhythmia/EnSite cardiac mapping software to determine LA scar burden</li>
</ol>
</div><div class="column" style="width:50%;">
<ul class="task-list">
<li><label><input type="checkbox" checked="">NLP coursework, cluster set-up preparation</label></li>
<li><label><input type="checkbox" checked="">HRX abstract preparation on EGM and <a href="https://physionet.org/content/wfdb/10.7.0/">Physionet/WFDB</a></label></li>
<li><label><input type="checkbox">Identify next paper to start preparing</label></li>
<li><label><input type="checkbox">eMERGE/AF IRB <em>waiting on approval from eMERGE</em></label></li>
<li><label><input type="checkbox">EPS study IRB <em>in progress</em></label></li>
</ul>
</div>
</div>
</section>
<section id="hrx-abstract" class="level2">
<h2 class="anchored" data-anchor-id="hrx-abstract">HRx abstract</h2>
<div id="fig-his-bundle-potentials" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-his-bundle-potentials-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/shiva-avnrt-egm.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-his-bundle-potentials-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6: Automated EGM annotation of single lead, with high-fidelity signal resolution. 75% file size reduction and 500% read/write speed-up.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="april-17-2023" class="level1">
<h1>April 17, 2023</h1>
<section id="goals-1" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="goals-1">Goals</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ol type="1">
<li>Phenotyping of AF, as current grouping is inadequate. Focus on risk category changing from paroxysmal to persistent</li>
<li>ECG- and EGM-based analysis of AF as additive components</li>
<li>CARTO/Rhythmia/EnSite cardiac mapping software to determine LA scar burden</li>
</ol>
</div><div class="column" style="width:50%;">
<ul class="task-list">
<li><label><input type="checkbox" checked="">AFL/FH manuscript submission</label></li>
<li><label><input type="checkbox" checked="">HRV/CVD manuscript revisions</label></li>
<li><label><input type="checkbox">Identify next paper to start preparing</label></li>
<li><label><input type="checkbox">eMERGE/AF IRB</label></li>
<li><label><input type="checkbox">EPS study IRB</label></li>
<li><label><input type="checkbox">NLP coursework, cluster set-up preparation</label></li>
<li><label><input type="checkbox">HRX abstract preparation on EGM and <a href="https://physionet.org/content/wfdb/10.7.0/">Physionet/WFDB</a></label></li>
</ul>
</div>
</div>
</section>
</section>
<section id="april-10-2023" class="level1">
<h1>April 10, 2023</h1>
<section id="af-ontology" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="af-ontology">AF Ontology</h2>
<p>Generation of key features that are related to arrhythmias:</p>
<ol type="1">
<li>Clinical history and trajectory</li>
<li>Echocardiographic findings</li>
<li>ECG-based features</li>
<li>Family and social history</li>
<li>Potential genetic markers</li>
</ol>
<hr>
<div id="fig-feature-extraction" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-feature-extraction-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/nagamine-2022-patient-timeline.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-feature-extraction-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;7: Example from HF evaluation and progression
</figcaption>
</figure>
</div>
<hr>
<div id="fig-feature-gen" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-feature-gen-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/nagamine-2022-feature-generation.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-feature-gen-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;8: Utilizing patient features to create a matrix-based cluster
</figcaption>
</figure>
</div>
<hr>
<div id="fig-adhf-cluster" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-adhf-cluster-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/nagamine-2022-heart-failure-clusters.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-adhf-cluster-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;9: Support-vector based clustering, multidimensional K-clustering in feature-space
</figcaption>
</figure>
</div>
</section>
<section id="phenotype-based-wes-cohort" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="phenotype-based-wes-cohort">Phenotype-based WES cohort</h2>
<div class="columns">
<div class="column" style="width:60%;">
<table class="table">
<thead>
<tr class="header">
<th>Age</th>
<th>Comorbidities</th>
<th>Proportion</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>-</td>
<td>none</td>
<td>10-15%</td>
</tr>
<tr class="even">
<td>≤ 55</td>
<td>*</td>
<td>20%</td>
</tr>
<tr class="odd">
<td>≤ 65</td>
<td>≤ 1</td>
<td>20%</td>
</tr>
<tr class="even">
<td>≤ 65</td>
<td>≤ 2</td>
<td>10%</td>
</tr>
<tr class="odd">
<td>≤ 65</td>
<td>≤ 3</td>
<td>10%</td>
</tr>
<tr class="even">
<td>≥ 65</td>
<td>≤ 1</td>
<td>10-15%</td>
</tr>
</tbody>
</table>
</div><div class="column" style="width:40%;">
<p>Alternatively can aim for two major groups within paroxysmal cohort definition:</p>
<ol type="1">
<li>≤ 65y + structural heart disease (70%)</li>
<li>≤ 65y - structural heart disease (30%)</li>
</ol>
<p><em>Aim for 20-30% to have had PVI to be able to integrate intracardiac findings</em>.</p>
</div>
</div>
<hr>
<div id="tbl-hrv-grps-cvd" class="quarto-float anchored">
<figure class="quarto-float quarto-float-tbl figure">
<div aria-describedby="tbl-hrv-grps-cvd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/tbl-hrv-groups-cvd.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-tbl" id="tbl-hrv-grps-cvd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: Autonomic inflexibility and CV mortality in HF and LF bands
</figcaption>
</figure>
</div>
<hr>
<div id="tbl-hrv-concordance" class="quarto-float anchored">
<figure class="quarto-float quarto-float-tbl figure">
<div aria-describedby="tbl-hrv-concordance-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/tbl-hrv-concordance.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-tbl" id="tbl-hrv-concordance-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;3: Improved model concordance with addition of autonomic inflexiblity
</figcaption>
</figure>
</div>
</section>
</section>
<section id="march-31-2023" class="level1">
<h1>March 31, 2023</h1>
<section id="af-phenotyping-2" class="level2">
<h2 class="anchored" data-anchor-id="af-phenotyping-2">AF Phenotyping</h2>
<p>NLP combined with DSP can help in identifying sub-phenotypes of AF…</p>
<ol type="1">
<li>Polygenic risk score assessment based on phenotypes and sub-phenotypes</li>
<li>Current phenotype approaches limited to structured text</li>
<li>Unstructured data can be extracted from clinical notes using NLP tools: BERT, Sci/RoBERTa, meta mapper, etc</li>
</ol>
<p><span class="green">AWS HPC + CCTS data + AF registry + EP lab + biomarkers</span></p>
<hr>
<div id="fig-emerge-example" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-emerge-example-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/emerge-example-creating-phenotypes.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-emerge-example-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10: Complexity of phenotyping is based on how structured or unstructured the data may be
</figcaption>
</figure>
</div>
<hr>
<div id="fig-emerge-true-cases" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-emerge-true-cases-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/emerge-venn-diagram-true-cases.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-emerge-true-cases-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;11: Identifying a true phenotype is complex and most often incorrect unless corroborated through other mechanisms
</figcaption>
</figure>
</div>
<hr>
<div id="fig-phenotype-kb-afib" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-phenotype-kb-afib-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/phenotype-kb-afib.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-phenotype-kb-afib-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;12: Phenotypke KB current standard for diagnoses of AF
</figcaption>
</figure>
</div>
</section>
<section id="eps-signal-processing" class="level2">
<h2 class="anchored" data-anchor-id="eps-signal-processing">EPS signal processing</h2>
<p>Utilize combination of surface ECG and intracardiac EGM to…</p>
<ol type="1">
<li>Identify intracardiac features, e.g.&nbsp;multichannel morphologies</li>
<li>Evaluate atrial abnormalities, such as conduction left → right activation</li>
<li>Feed features into LSTM or convolutional neural network to understand different AF phenotypes</li>
</ol>
<hr>
<div id="fig-shiva-ecg" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-shiva-ecg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/shiva-annotated-ecg-green.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-shiva-ecg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13: Annotated ECG to identify P wave, QRS, and T wave fiducial points
</figcaption>
</figure>
</div>
<ul>
<li>Summation of P wave force vectors across surface ECG (signal-averaged)</li>
<li>Identify terminal P wave force abnormalities</li>
<li>Evaluate changes in P wave morphology over time</li>
</ul>
</section>
</section>
<section id="march-20-2023" class="level1">
<h1>March 20, 2023</h1>
<section id="updates-8" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="updates-8">Updates</h2>
<ul>
<li>AFL/FH:
<ul>
<li>manuscript was sent for initial revisions (03/03/23)</li>
<li>feedback received on 03/17/23 from <em>Jordan</em></li>
<li>second draft to be returned on 3/20/23</li>
</ul></li>
<li><span class="green">Aim 1</span>: “Big data” approach using claims-based data, IRB/CCTS approval obtained
<ol type="1">
<li>NLP/ML area of focus will be initially on AF (only performed x 1)</li>
<li>Approach for recurrent events, Weibull distribution expected</li>
</ol></li>
<li><span class="blue">HRV/CVD</span> :
<ul>
<li>priority for this week</li>
<li>HF/LF HRV values showing &gt;10 fold HR for bottom versus top quartile (with almost &gt;95% sensitivity)</li>
</ul></li>
<li>Genetic analysis:
<ul>
<li>ACER will start AWS workspace this week</li>
</ul></li>
</ul>
</section>
<section id="help-wanted" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="help-wanted">Help Wanted</h2>
<ul>
<li>Ablation database: working with Wissner to revise ablation database
<ul>
<li>manual data reconciliation</li>
<li>OCR/OMR-based PDF conversion approach</li>
<li>ablation outcomes x genetics</li>
</ul></li>
<li>AF phenotyping: key part of <span class="green">Aim 1</span>
<ul>
<li>Computational Biorepository for Cardiovascular Disease: CCTS data pull active, all clinical notes, data points, etc since 2010 on CVD</li>
<li>AF ontology needed using a NLP/LLM (e.g.&nbsp;BERT)</li>
</ul></li>
<li>NLP/ML for EHR data:
<ul>
<li>CBCD: overlap of DCM/AF registries, CCTS data pull of all clinical notes since 2010 on CVD</li>
<li>Key part of <strong>Aim 1</strong> is to identify AF phenotypes in those with paroxysmal AF</li>
<li>AF Ontology: working with Andrew Boyd on AF-NLP framework (SciB)</li>
</ul></li>
</ul>
</section>
</section>
<section id="february-27-2023" class="level1">
<h1>February 27, 2023</h1>
<section id="updates-9" class="level2">
<h2 class="anchored" data-anchor-id="updates-9">Updates</h2>
<ul>
<li>AFL/FH: will need age of diagnosis of event of family history prior to final analyses</li>
<li>CAR: analysis pending by Konda, however unclear if we can strongly accept null-hypothesis</li>
</ul>
</section>
<section id="non-linearity-in-ans-dysfunction" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="non-linearity-in-ans-dysfunction">Non-linearity in ANS dysfunction</h2>
<ol type="1">
<li>Proportional hazard assumptions
<ol type="a">
<li>Defined using time-variance, time-interaction, Martingale residuals</li>
<li>Satisfied PH assumptions</li>
</ol></li>
<li>Non-linearity of HRV and CV mortality
<ol type="a">
<li>Spline analysis with up to 5 knots</li>
<li>Parametric threshold analysis (survival modeling)</li>
<li>Non-parametric (binary outcome) threshold analysis</li>
<li>Identified <strong>non-linearity</strong> of response</li>
</ol></li>
<li>Re-analyzed data using cut-point (as shown in <a href="#fig-hf-hrv-reactivity-quartiles" class="quarto-xref">Figure&nbsp;14</a>)
<ol type="a">
<li>Almost 100% accuracy in classifier of low-risk cohort</li>
<li>Almost 10-fold hazard in identify high-risk cohort</li>
</ol></li>
</ol>
<hr>
<div id="fig-hf-hrv-reactivity-quartiles" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hf-hrv-reactivity-quartiles-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/hf-hrv-reactivity-km-quartile-curves.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hf-hrv-reactivity-quartiles-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;14: Increase of ~10 fold in CV mortality in 1/4 patients identified by abnormal resting and reactive vagal tone. Robust classifier of resilience.
</figcaption>
</figure>
</div>
</section>
<section id="genetic-analysis" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="genetic-analysis">Genetic analysis</h2>
<ol type="1">
<li>WES-generated VCF files on n=28 patients from EO-AFL + FH subgroup</li>
<li>Due to size limitations (<strong>no access to HPC</strong>) converted VCF data …
<ol type="a">
<li>VCF to Apache Arrow</li>
<li>Arrow conversion to <code>feather</code> and to <code>parquet</code> for header, annotation, and columnar genotype data</li>
<li>Analyze as in-memory array</li>
<li>Re-vert to ASCII-based format for GZip formated VCF</li>
</ol></li>
<li>Filtering of data
<ol type="a">
<li>Limited to coding regions</li>
<li>Filtered for read depth &gt; 20</li>
<li>Compared to dbSNP builds using <code>SIFT</code> and <code>PolyPhen</code></li>
<li><strong>Pending further annotations based on arrhythmia-panels</strong></li>
</ol></li>
</ol>
</section>
<section id="specific-aims" class="level2">
<h2 class="anchored" data-anchor-id="specific-aims">Specific aims</h2>
<p>Have revised <a href="http://asshah4.github.io/research/k23/overview.html">aims</a>, and have drafted the x1 page <strong>specific aims</strong> to be shared with mentors.</p>
<p>Mentors: D Darbar, AJ Shah, A Alonso, M McCauley, <em>A Boyd</em></p>
</section>
</section>
<section id="february-20-2023" class="level1 page-columns page-full">
<h1>February 20, 2023</h1>
<section id="aflfh-1" class="level2">
<h2 class="anchored" data-anchor-id="aflfh-1">AFL/FH</h2>
<ul>
<li>Genetic basis for FH may be more promising than association with FH broadly, but would require additional WES to be sent</li>
<li>Would need to repeat VCF analysis for an arrhythmia panel (instead of cardiomyopathy panel)</li>
</ul>
</section>
<section id="cbcd" class="level2 smaller page-columns page-full">
<h2 class="smaller anchored" data-anchor-id="cbcd">CBCD</h2>
<p>Computational biorepository for cardiovascular disease</p>
<ul>
<li>Will need a large computational biorepository
<ul>
<li>Claims data from both CPT/ICD codes</li>
<li>Clinical documentation (raw text from clinical notes)</li>
<li>Medication history</li>
<li>Study data e.g.&nbsp;XML of ECG, echo reports, coronary angiograms, device interrogations, etc</li>
</ul></li>
<li>CCTS to pull data, DRA to be submitted today</li>
</ul>

<div class="no-row-height column-margin column-container"><div class="margin-aside">
<p>Also working on VA research database access, CART-CL reporting data, MVP, ARIC, and UK Biobank as additional large-scale data</p>
</div></div></section>
</section>
<section id="february-6-2023" class="level1 page-columns page-full">
<h1>February 6, 2023</h1>
<section id="updates-10" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="updates-10">Updates</h2>
<ul>
<li>AFL/FH: <strong>WES</strong> needed prior to completing revisions</li>
<li>ARIC: potential option to pursue genetic profiling in ARIC per Alvaro, need formal proposal</li>
<li>K23: drafting specific aims in 1-pager format</li>
<li>VCF: data able to be analyzed/cleaned, however requires more CPU to perform</li>
</ul>

<div class="no-row-height column-margin column-container"><div class="margin-aside">
<p>Currently on inpatient service with limited time</p>
</div></div><hr>
<div id="fig-vcfr-summary" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-vcfr-summary-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="../figures/vcf-sample-chr-1-from-mike.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-vcfr-summary-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;15: Chromosome 1 summary
</figcaption>
</figure>
</div>
</section>
</section>
<section id="january-9-2023" class="level1">
<h1>January 9, 2023</h1>
<section id="updates-11" class="level2">
<h2 class="anchored" data-anchor-id="updates-11">Updates</h2>
<div class="columns">
<div class="column" style="width:50%;">
<p>AFL/FH:</p>
<ul>
<li>Pedigrees/genetics completed (thanks to <span class="blue">Ana</span>, <span class="blue">Shashank</span>, <span class="blue">Mike</span>)</li>
<li>Outline/draft to be re-written</li>
</ul>
</div><div class="column" style="width:50%;">
<ol type="1">
<li><span class="blue">HRV/CV Mortality</span> paper rejected, need to re-think strategy with senior authors</li>
<li>K23 aims to include 1) EP lab as <span class="orange">translational</span> component, 2) <span class="blue">arrhythmia risk prediction</span> as <span class="green">computational</span> component</li>
</ol>
</div>
</div>
</section>
<section id="aflgenetics" class="level2">
<h2 class="anchored" data-anchor-id="aflgenetics"><span class="blue">AFL/genetics</span></h2>
<ul>
<li>~87 individuals with +FH</li>
<li>~81 individuals with WES</li>
<li>~18 individuals with VUS in total</li>
<li><strong>1 individual</strong> with VUS + FH</li>
</ul>
</section>
<section id="multiple-pedigrees" class="level2">
<h2 class="anchored" data-anchor-id="multiple-pedigrees">Multiple Pedigrees</h2>
<p><img src="../figures/multiple_pedigree_plot.png" class="img-fluid" alt="AFL/surrogate disease burden"> ## TTN Mutant</p>
<p><img src="../figures/pedigree_uic0530.png" class="img-fluid" alt="AFL/mutant"> # December 19, 2022</p>
</section>
<section id="updates-12" class="level2">
<h2 class="anchored" data-anchor-id="updates-12">Updates</h2>
<div class="columns">
<div class="column" style="width:50%;">
<p>AFL/FH:</p>
<ul>
<li>Pedigrees TBD</li>
<li>Draft completed, pending feedback</li>
<li>Supplemental tables needed?</li>
</ul>
</div><div class="column" style="width:50%;">
<p>AF/Recurrence:</p>
<ul>
<li>Scott added additional patients from UIC (September 2020 to now)</li>
<li>Data collection pending</li>
</ul>
</div>
</div>
</section>
<section id="k23-aims" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="k23-aims">K23 Aims</h2>
<p>Objective: <em>Clinical EP researcher using computational neurocardiology techniques to study arrhythmia mechanisms</em></p>
<ol type="1">
<li>Evaluating vagolysis and its effect on triggered arrhythmia mechanisms, both ventricular (e.g.&nbsp;SCD) and atrial (triggered AF)</li>
<li>Using computational approach to phenotype triggered onset arrhythmias, e.g.&nbsp;atrial fibrillation</li>
</ol>
<p>Mentorship Team: <strong>Amit J. Shah</strong>, <strong>Dawood Darbar</strong>, Rachel Lampert, Viola Vaccarino, Mark McCauley, Andrew Boyd, Alvaro Alonso</p>
</section>
<section id="k23-approach" class="level2">
<h2 class="anchored" data-anchor-id="k23-approach">K23 Approach</h2>
<p><strong>Vagal tone and triggered arrhythmias</strong></p>
<ul>
<li>Manuscript on stress-induced vagolysis and CV mortality <strong>under review</strong></li>
<li>ANS dysfunction during/with ischemia <strong>work-in-progress</strong></li>
<li>[Create murine/translational model for vagolysis and triggered arrhythmias]{.arrhythmia[2]}</li>
</ul>
<p><strong>Arrhythmia phenotyping</strong></p>
<ul>
<li>AFL/FH manuscript <strong>work-in-progress</strong></li>
<li>AF/race project <strong>work-in-progress</strong></li>
<li>[Classification of arrhythmia phenotypes using large data set (MVP/VA research data, UIC, ARIC)]{.computational[3]}</li>
</ul>
</section>
<section id="murine-model-of-vagolysis" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="murine-model-of-vagolysis">Murine Model of Vagolysis</h2>
<ul>
<li>Atria are heavily innervated by ANS ganglia
<ul>
<li>Sympathovagal balance locally mediated through adrenergic lysis of cholinergic activity</li>
<li>Neuropeptide Y (NPY) causes cholinergic inhibition through Y2R</li>
</ul></li>
<li>Pro-arrhythmic murine model (compared to healthy controls)
<ul>
<li>Ex-vivo vagal-sparing Langendorf preparation</li>
<li>Catecholamine infusion and NPY Y2R antagonists to modulate arrhythmic state</li>
</ul></li>
<li>Measure atrial conductive properties as outcome
<ul>
<li>Baseline, with catecholamine infusion, VNS stimulation, and Y2R antagonism</li>
</ul></li>
</ul>
</section>
</section>
<section id="december-12-2022" class="level1">
<h1>December 12, 2022</h1>
<section id="af-catheter-ablation-and-recurrence" class="level2">
<h2 class="anchored" data-anchor-id="af-catheter-ablation-and-recurrence">AF Catheter Ablation and Recurrence</h2>
<p>Data/power increase:</p>
<ul>
<li>Inclusion of VA data</li>
<li>Additional data pull from EPIC by Scott Uphouse (pending)</li>
<li>Inclusion of UIC billing code data?</li>
</ul>
</section>
<section id="atrial-flutter-and-family-history" class="level2">
<h2 class="anchored" data-anchor-id="atrial-flutter-and-family-history">Atrial Flutter and Family History</h2>
<div class="columns">
<div class="column" style="width:50%;">
<p>Pedigrees:</p>
<ul>
<li>How should these be incorporated into the manuscript?</li>
</ul>
</div><div class="column" style="width:50%;">
<p>Genetics:</p>
<ul>
<li>305 patients that have undergone whole exome sequencing, but unclear how to match these to the AFL data</li>
<li>Should we genotype the rest of the EO-AFL patients?</li>
</ul>
</div>
</div>
</section>
<section id="ecgegm-analysis" class="level2">
<h2 class="anchored" data-anchor-id="ecgegm-analysis">ECG/EGM analysis</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(shiva)</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>x <span class="ot">&lt;-</span> <span class="fu">read_lspro</span>(<span class="at">file =</span> <span class="fu">file.path</span>(<span class="fu">find_project_folder</span>(), <span class="st">"research"</span>, <span class="st">"t32"</span>, <span class="st">"sample-egm.txt"</span>))</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a><span class="fu">ggm</span>(x, <span class="at">channels =</span> <span class="fu">c</span>(<span class="st">"I"</span>, <span class="st">"CS"</span>, <span class="st">"HIS M"</span>, <span class="st">"HIS D"</span>, <span class="st">"RV"</span>)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="rip_files/figure-html/unnamed-chunk-3-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="november-21-2022" class="level1">
<h1>November 21, 2022</h1>
<section id="updates-13" class="level2">
<h2 class="anchored" data-anchor-id="updates-13">Updates</h2>
<ul>
<li>AHA IPA: Completed draft, pending revisions</li>
<li>AFL Paper: Draft to be done by Wednesday</li>
<li>AF Ablation + Biorepository: Pending IRB</li>
<li>Stress and CVD Mortality: Submitted to <em>Circulation</em>, revisions Pending</li>
</ul>
</section>
</section>
<section id="november-14-2022" class="level1">
<h1>November 14, 2022</h1>
<section id="early-onset-atrial-flutter" class="level2">
<h2 class="anchored" data-anchor-id="early-onset-atrial-flutter">Early Onset Atrial Flutter</h2>
<ol type="1">
<li>Draft of paper expanded to include results</li>
<li>Prominent findings of family history and association with arrhythmia</li>
<li>Key tables and figures established</li>
</ol>
<p>Next steps:</p>
<ul>
<li>Find agreement in key results and figures</li>
<li>Identify main finding</li>
<li>Complete discussion section</li>
</ul>
</section>
</section>
<section id="october-10-2022" class="level1">
<h1>October 10, 2022</h1>
<hr>
<p><strong>AF Registry</strong></p>
<p>Status:</p>
<ul>
<li>Current <a href="https://github.com/asshah4/aflubber/R/af_ablation_recurrence.html">overview</a></li>
<li>Not yet powered</li>
<li>Will need to revise out approach</li>
</ul>
</section>
<section id="october-2-2022" class="level1">
<h1>October 2, 2022</h1>
<hr>
<p><strong>AF Registry</strong></p>
<ul>
<li>Current number: 219</li>
<li>Recurrence rate: 46%</li>
<li>Death = 12</li>
<li>MACE = 70 (including repeats)</li>
</ul>
<hr>
<hr>
<div class="cell">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>x <span class="ot">&lt;-</span> cardiac_atlas<span class="sc">$</span>right_atrium_rao</span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_void</span>() <span class="sc">+</span></span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_sf</span>(<span class="at">data =</span> x<span class="sc">$</span>chambers, <span class="fu">aes</span>(<span class="at">fill =</span> structure)) <span class="sc">+</span></span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_sf</span>(<span class="at">data =</span> x<span class="sc">$</span>projections, <span class="fu">aes</span>(<span class="at">fill =</span> structure)) <span class="sc">+</span> </span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_sf</span>(<span class="at">data =</span> x<span class="sc">$</span>valves, <span class="fu">aes</span>(<span class="at">fill =</span> structure)) <span class="sc">+</span> </span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_sf</span>(<span class="at">data =</span> x<span class="sc">$</span>septum, <span class="fu">aes</span>(<span class="at">fill =</span> structure)) <span class="sc">+</span> </span>
<span id="cb20-9"><a href="#cb20-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_sf</span>(<span class="at">data =</span> x<span class="sc">$</span>vessels, <span class="at">fill =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb20-10"><a href="#cb20-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_brewer</span>(<span class="at">palette =</span> <span class="st">"YlGnBu"</span>) <span class="sc">+</span></span>
<span id="cb20-11"><a href="#cb20-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in RColorBrewer::brewer.pal(n, pal): n too large, allowed maximum for palette YlGnBu is 9
Returning the palette you asked for with that many colors</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="rip_files/figure-html/unnamed-chunk-5-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p>GIS-based MAP of right atrium for mapping and localization purposes for publications.</p>
</section>
<section id="september-26-2022" class="level1">
<h1>September 26, 2022</h1>
<hr>
<section id="af-registry" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="af-registry">AF Registry</h2>
<div class="columns">
<div class="column" style="width:40%;">
<p><strong>Analysis</strong></p>
<ul>
<li>Genetic analysis in PLINK complete</li>
<li>Models are set up for analysis once data intake is done</li>
</ul>
</div><div class="column" style="width:40%;">
<p><strong>Status</strong></p>
<ol type="1">
<li>Have n = 238 currently</li>
<li>Reviewed 95 charts thus far (CAR team)</li>
<li>Of n = 108, 75 have had recurrence, 33 without (rate of about 70% of anyarrhythmia)</li>
<li>Of ECG based confirmation, 43 AF recurrence events (53%)</li>
</ol>
</div>
</div>
</section>
</section>
<section id="september-12-2022" class="level1">
<h1>September 12, 2022</h1>
<hr>
<section id="af-registry-1" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="af-registry-1">AF Registry</h2>
<p>REDCap data dictionary is finalized. Next steps are:</p>
<ol type="1">
<li>Data entry for outcomes</li>
<li>Finalization of new patients to add from AF ablation registry</li>
</ol>
</section>
<section id="genetics" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="genetics">Genetics</h2>
<p>Currently able to:</p>
<ul>
<li>Format large SNP data sets into appropriate tables</li>
<li>Utilize <code>PLINK</code> and <code>MERLIN</code> to process Chen’s genetics data</li>
<li>Run from R to help analyze findings</li>
</ul>
<p>Next steps:</p>
<ul>
<li>Confirm ancestry (for practice) percent likelihood</li>
<li>Then, once outcome data is complete, analyze differences by race</li>
</ul>
<p>Analytical methods:</p>
<ul>
<li>Time to binary outcome of recurrence</li>
<li>Recurrent event analysis for AF recurrence</li>
<li>Can revise with Bayesian modeling in <code>STAN</code> as well</li>
</ul>
</section>
</section>
<section id="august-22-2022" class="level1">
<h1>August 22, 2022</h1>
<hr>
<section id="af-registry-2" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="af-registry-2">AF Registry</h2>
<div class="columns">
<div class="column" style="width:40%;">
<p><strong>Status</strong></p>
<ul>
<li>REDCap is in progress</li>
<li>Have an additional list of registry patients that may/may not have had ablation</li>
</ul>
</div><div class="column" style="width:40%;">
<p><strong>Next Steps</strong></p>
<ol type="1">
<li>Complete 100 REDCap patients</li>
<li>Identify 300 patients that have potentially had ablation</li>
</ol>
</div>
</div>
<hr>
</section>
<section id="carto-maps" class="level2 smaller">
<h2 class="smaller anchored" data-anchor-id="carto-maps">CARTO Maps</h2>
<ul>
<li>Ablation quantification</li>
<li>Activation mapping</li>
<li>EGM annotation</li>
<li>Voltage mapping</li>
<li>Conduction velocity</li>
<li>Earliest/latest activation</li>
<li>Geometry data</li>
</ul>
</section>
</section>
<section id="august-15-2022" class="level1">
<h1>August 15, 2022</h1>
<hr>
<div class="columns">
<div class="column" style="width:40%;">
<p><strong>UIC-wide data summary</strong></p>
<ul>
<li>N ~ XXX had ablation</li>
<li>Date range from &gt;= 2010</li>
</ul>
<p><strong>AF registry data</strong></p>
<ul>
<li>N=~300 had ablation</li>
<li>Date range from &gt;= 2010</li>
</ul>
</div><div class="column" style="width:40%;">
<p><strong>Next Steps</strong></p>
<ol type="1">
<li>Generate list of MRN for data collection</li>
<li>Use new REDCap to obtain ECG/ablation data</li>
<li>Run preliminary analysis on first 100 patients</li>
<li>Consider expansion to JBVA/ACMC data</li>
</ol>
</div>
</div>
</section>
<section id="august-1-2022" class="level1">
<h1>August 1, 2022</h1>
<section id="af-registry-3" class="level2">
<h2 class="anchored" data-anchor-id="af-registry-3">AF Registry</h2>
<p>Consider additional variables:</p>
<ul>
<li>Recurrent events and adjudication (HF, cardiac admission, MACE)</li>
<li>Holter/ECG data and repeat collection (P-wave morphology, Wilson’s vector gradient, amplitude of AF waves “coarseness”)</li>
</ul>
<p>Will check with the MESA and ARIC data base on how repeat events were defined.</p>
</section>
<section id="murine-ep-studies" class="level2">
<h2 class="anchored" data-anchor-id="murine-ep-studies">Murine EP Studies</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">"../figures/electromap.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="../figures/electromap.png" class="img-fluid figure-img" style="width:60.0%"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="human-ep-studies" class="level2">
<h2 class="anchored" data-anchor-id="human-ep-studies">Human EP Studies</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">"../figures/openep-ablation.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="../figures/openep-ablation.png" class="img-fluid figure-img" width="711"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="july-25-2022" class="level1">
<h1>July 25, 2022</h1>
<section id="forest-plots" class="level2">
<h2 class="anchored" data-anchor-id="forest-plots">Forest Plots</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Forest Plots</span></span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(vlndr)</span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a>m <span class="ot">&lt;-</span> <span class="fu">rx</span>(</span>
<span id="cb24-4"><a href="#cb24-4" aria-hidden="true" tabindex="-1"></a>    Petal.Length <span class="sc">~</span> <span class="fu">X</span>(Sepal.Length) <span class="sc">+</span> Petal.Width <span class="sc">+</span> <span class="fu">S</span>(Species),</span>
<span id="cb24-5"><a href="#cb24-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">label =</span> <span class="fu">list</span>(</span>
<span id="cb24-6"><a href="#cb24-6" aria-hidden="true" tabindex="-1"></a>        Petal.Length <span class="sc">~</span> <span class="st">"Length of Petals"</span>, </span>
<span id="cb24-7"><a href="#cb24-7" aria-hidden="true" tabindex="-1"></a>        Species <span class="sc">~</span> <span class="st">"Genus of the Flower"</span></span>
<span id="cb24-8"><a href="#cb24-8" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb24-9"><a href="#cb24-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">pattern =</span> <span class="st">"direct"</span></span>
<span id="cb24-10"><a href="#cb24-10" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">|&gt;</span></span>
<span id="cb24-11"><a href="#cb24-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">fmls</span>(<span class="at">order =</span> <span class="dv">2</span>) <span class="sc">|&gt;</span></span>
<span id="cb24-12"><a href="#cb24-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">fit</span>(<span class="at">.fit =</span> lm,</span>
<span id="cb24-13"><a href="#cb24-13" aria-hidden="true" tabindex="-1"></a>            <span class="at">data =</span> iris,</span>
<span id="cb24-14"><a href="#cb24-14" aria-hidden="true" tabindex="-1"></a>            <span class="at">archetype =</span> <span class="cn">TRUE</span>) <span class="sc">|&gt;</span></span>
<span id="cb24-15"><a href="#cb24-15" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mdls</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</div><div class="column" style="width:50%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>tbl <span class="ot">&lt;-</span> <span class="fu">tbl_forest</span>( <span class="co">#&lt;&lt;</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">object =</span> m,</span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">formula =</span> Petal.Length <span class="sc">~</span> Sepal.Length,</span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">vars =</span> <span class="st">"Species"</span>,</span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">columns =</span> <span class="fu">list</span>(</span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>        beta <span class="sc">~</span> <span class="st">"Estimate"</span>,</span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>        conf <span class="sc">~</span> <span class="st">"95% Confidence Interval"</span>,</span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>        n <span class="sc">~</span> <span class="st">"Number of Samples"</span></span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis =</span> <span class="fu">list</span>(</span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a>        lim <span class="sc">~</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">3</span>),</span>
<span id="cb25-12"><a href="#cb25-12" aria-hidden="true" tabindex="-1"></a>        breaks <span class="sc">~</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>, <span class="dv">2</span>),</span>
<span id="cb25-13"><a href="#cb25-13" aria-hidden="true" tabindex="-1"></a>        lab <span class="sc">~</span> <span class="st">"ß (95% CI)"</span>,</span>
<span id="cb25-14"><a href="#cb25-14" aria-hidden="true" tabindex="-1"></a>        title <span class="sc">~</span> <span class="st">"Petal Length by Sepal Length"</span></span>
<span id="cb25-15"><a href="#cb25-15" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb25-16"><a href="#cb25-16" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</div>
</div>
<p>A simple way to make sub-group analysis Forest plots, working on improving customization.</p>
<hr>
</section>
</section>
<section id="july-18-2022" class="level1">
<h1>July 18, 2022</h1>
<section id="af-registry-4" class="level2">
<h2 class="anchored" data-anchor-id="af-registry-4">AF Registry</h2>
<p>The AF registry is described <a href="https://asshah4.github.io/aflubber">here</a>. Current issues:</p>
<ol type="1">
<li>Data quality - consistency between reviewers</li>
<li>Missingness - decisions on acceptable thresholds</li>
<li>Variable selection - echo findings, EP studies, medications, labs, symptoms</li>
<li>Management - REDCap, shared excel sheet</li>
<li>Adjudication/review - outcomes, clinical follow-up length</li>
</ol>



</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-Andrade2023" class="csl-entry" role="listitem">
Andrade, Jason G., Marc W. Deyell, Laurent Macle, George A. Wells, Matthew Bennett, Vidal Essebag, Jean Champagne, et al. 2023. <span>“Progression of <span>Atrial Fibrillation</span> After <span>Cryoablation</span> or <span>Drug Therapy</span>.”</span> <em>New England Journal of Medicine</em> 388 (2): 105–16. <a href="https://doi.org/10.1056/NEJMoa2212540">https://doi.org/10.1056/NEJMoa2212540</a>.
</div>
<div id="ref-Benjamin2023" class="csl-entry" role="listitem">
Benjamin, Emelia J., Kevin L. Thomas, Alan S. Go, Patrice Desvigne-Nickens, Christine M. Albert, Alvaro Alonso, Alanna M. Chamberlain, et al. 2023. <span>“Transforming <span>Atrial Fibrillation Research</span> to <span>Integrate Social Determinants</span> of <span>Health</span>: <span>A National Heart</span>, <span>Lung</span>, and <span>Blood Institute Workshop Report</span>.”</span> <em>JAMA Cardiology</em> 8 (2): 182–91. <a href="https://doi.org/10.1001/jamacardio.2022.4091">https://doi.org/10.1001/jamacardio.2022.4091</a>.
</div>
<div id="ref-Essien2021" class="csl-entry" role="listitem">
Essien, Utibe R., Jelena Kornej, Amber E. Johnson, Lucy B. Schulson, Emelia J. Benjamin, and Jared W. Magnani. 2021. <span>“Social Determinants of Atrial Fibrillation.”</span> <em>Nature Reviews. Cardiology</em> 18 (11): 763–73. <a href="https://doi.org/10.1038/s41569-021-00561-0">https://doi.org/10.1038/s41569-021-00561-0</a>.
</div>
<div id="ref-Holmqvist2015" class="csl-entry" role="listitem">
Holmqvist, Fredrik, Sunghee Kim, Benjamin A. Steinberg, James A. Reiffel, Kenneth W. Mahaffey, Bernard J. Gersh, Gregg C. Fonarow, et al. 2015. <span>“Heart Rate Is Associated with Progression of Atrial Fibrillation, Independent of Rhythm.”</span> <em>Heart (British Cardiac Society)</em> 101 (11): 894–99. <a href="https://doi.org/10.1136/heartjnl-2014-307043">https://doi.org/10.1136/heartjnl-2014-307043</a>.
</div>
<div id="ref-Kerr2005" class="csl-entry" role="listitem">
Kerr, Charles R., Karin H. Humphries, Mario Talajic, George J. Klein, Stuart J. Connolly, Martin Green, John Boone, Robert Sheldon, Paul Dorian, and David Newman. 2005. <span>“Progression to Chronic Atrial Fibrillation After the Initial Diagnosis of Paroxysmal Atrial Fibrillation: <span>Results</span> from the <span>Canadian Registry</span> of <span>Atrial Fibrillation</span>.”</span> <em>American Heart Journal</em> 149 (3): 489–96. <a href="https://doi.org/10.1016/j.ahj.2004.09.053">https://doi.org/10.1016/j.ahj.2004.09.053</a>.
</div>
<div id="ref-vandeLeur2021" class="csl-entry" role="listitem">
Leur, Rutger R. van de, Karim Taha, Max N. Bos, Jeroen F. van der Heijden, Deepak Gupta, Maarten J. Cramer, Rutger J. Hassink, et al. 2021. <span>“Discovering and <span>Visualizing Disease-Specific Electrocardiogram Features Using Deep Learning</span>.”</span> <em>Circulation: Arrhythmia and Electrophysiology</em> 14 (2): e009056. <a href="https://doi.org/10.1161/CIRCEP.120.009056">https://doi.org/10.1161/CIRCEP.120.009056</a>.
</div>
<div id="ref-Takigawa2014" class="csl-entry" role="listitem">
Takigawa, Masateru, Atsushi Takahashi, Taishi Kuwahara, Kenji Okubo, Yoshihide Takahashi, Yuji Watari, Katsumasa Takagi, et al. 2014. <span>“Long-<span>Term Follow-Up After Catheter Ablation</span> of <span>Paroxysmal Atrial Fibrillation</span>.”</span> <em>Circulation: Arrhythmia and Electrophysiology</em> 7 (2): 267–73. <a href="https://doi.org/10.1161/CIRCEP.113.000471">https://doi.org/10.1161/CIRCEP.113.000471</a>.
</div>
<div id="ref-Verweij2020" class="csl-entry" role="listitem">
Verweij, Niek, Jan-Walter Benjamins, Michael P. Morley, Yordi J. Van De Vegte, Alexander Teumer, Teresa Trenkwalder, Wibke Reinhard, Thomas P. Cappola, and Pim Van Der Harst. 2020. <span>“The <span>Genetic Makeup</span> of the <span>Electrocardiogram</span>.”</span> <em>Cell Systems</em> 11 (3): 229–238.e5. <a href="https://doi.org/10.1016/j.cels.2020.08.005">https://doi.org/10.1016/j.cels.2020.08.005</a>.
</div>
<div id="ref-Vesela2023" class="csl-entry" role="listitem">
Veselá, Jana, Pavel Osmančík, Dalibor Heřman, Sabri Hassouna, Radka Raková, Tomáš Veselý, and Petr Budera. 2023. <span>“Prediction of Post-Operative Atrial Fibrillation in Patients After Cardiac Surgery Using Heart Rate Variability.”</span> <em>BMC Cardiovascular Disorders</em> 23 (1): 290. <a href="https://doi.org/10.1186/s12872-023-03309-5">https://doi.org/10.1186/s12872-023-03309-5</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>Copyright (c) 2023, Anish. S. Shah, MD, MS</p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>