---
title: "Research-In-Progress"
author: "Anish Sanjay Shah, MD/MS<br><span style = 'font-size: 70%;'>Cardiology Fellow<br>University of Illinois Chicago</span>"
format: 
  revealjs:
    scrollable: true
execute:
  cache: true
tbl-cap-location: bottom
fig-cap-location: bottom
---

```{r}
#| label: setup
#| include: false
#| cache: false

# Libraries
library(tidyverse)
library(rmdl)
library(EGM)
library(hrd)
library(googledrive)
library(googlesheets4)
library(clock)
library(gt)
library(glue)
library(ggsci)
library(scales)
library(vroom)

# Location
find_project_folder <- function() {
	x <- sessionInfo()$running
	if (grepl("mac", x)) {
		file.path("/Users", "asshah4", "projects")
	} else if (grepl("Windows", x)) {
		file.path("C:/Users", "asshah4", "projects")
	}
}


# Used in YAML above as well - lower is darker, higher is lighter
base::readRenviron("_environment")
col_arrhythmia <- Sys.getenv("primary")
col_clinical <- Sys.getenv("danger")
col_computational <- Sys.getenv("warning")
col_epidemiology <- Sys.getenv("secondary")
col_text <- Sys.getenv("dark")


tbl_status_updates <- function(progress, effort) {
	
	data <-
		left_join(progress, effort, by = "Type") |>
		group_by(Type) |>
		mutate(Type = paste0(Type, " (", Effort, ")")) |>
		mutate(Deadline = as.Date(Deadline)) |>
		select(-Effort) |>
		filter(Priority < 3) |>
		select(Type, Project, Task, Deadline, Priority, Status) |>
		group_by(Project) |>
		arrange(Project, Priority) |>
		mutate(num = 1:n()) 
		
	
	data |>
		gt(rowname_col = "Project", groupname_col = "Type") |>
		opt_table_lines(extent = "none") |>
		opt_row_striping(row_striping = FALSE) |>
		tab_options(data_row.padding = px(2)) |>
		# Headers and Row Groups
		tab_style(
			style =  cell_text(
				size = "large",
				weight = "bold",
				transform = "capitalize"
			),
			locations = cells_column_labels()
		) |>
		tab_style(
			style = cell_text(color = "white"),
			locations = cells_stub(rows = num != 1)
		) |>
		tab_style(
			style = list(cell_fill(color = col_epidemiology),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = col_epidemiology)),
			locations = cells_row_groups(groups = contains("Epidemiology"))
		) |>
		tab_style(
			style = list(cell_fill(color = col_arrhythmia),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = col_arrhythmia)),
			locations = cells_row_groups(groups = contains("Arrhythmia"))
		) |>
		tab_style(
			style = list(cell_fill(color = col_clinical),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = col_clinical)),
			locations = cells_row_groups(groups = contains("Clinical"))
		) |>
		tab_style(
			style = list(cell_fill(color = col_computational),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = col_computational)),
			locations = cells_row_groups(groups = contains("Computational"))
		) |>
		tab_style(
			style = list(cell_fill(color = col_text),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = col_text)),
			locations = cells_row_groups(groups = contains("Academic"))
		) |>
		# General formatting
		tab_style(
			style = cell_text(color = "white"),
			locations = cells_body(columns = Task,
														 rows = is.na(Task))
		) |>
		tab_style(
			style = cell_text(color = "white"),
			locations = cells_body(columns = Deadline,
														 rows = is.na(Deadline))
		) |>
		tab_style(
			style = cell_text(align = "center"),
			locations = list(cells_stub(), cells_column_labels())
		) |>
		tab_style(
			style = cell_borders(sides = "bottom", color = col_text),
			locations = list(
				cells_body(columns = everything()),
				cells_stub(rows = everything())
			)
		) |> 
		tab_style(
			style = cell_text(color = "grey"),
			locations = cells_body(columns = Task,
														 rows = Status == "complete")
		) |>
		tab_style(
			style = cell_text(weight = "bold"),
			locations = cells_body(columns = Task,
														 rows = Status == "waiting")
		) |>
		tab_style(
			style = cell_text(style = "italic"),
			locations = cells_body(columns = Task,
														 rows = Status == "defer")
		) |>
		# Columns
		cols_hide(columns = c(Priority, num, Status)) |>
		fmt_date(columns = "Deadline", date_style = "iso") |>
		tab_style(
			style = cell_text(size = "x-small"),
			locations = cells_body(columns = "Deadline")
		) |>
		tab_style(
			style = cell_text(style = "italic"),
			locations = cells_stub()
		) |>
		cols_width(
			Project ~ pct(40),
			Task ~ pct(45),
			#Description ~ pct(35),
			Deadline ~ pct(15)
		) 
}
```


```{r}
#| label: tbl-progress
#| tbl-cap: Progress updates
#| echo: false
#| cache: false
dt <- 
	date_build(year = 2024, month = 6, day = 10) |>
	add_days(-30)

progress <-
	drive_get("research_log") |>
	read_sheet(sheet = "project_sheet") |>
	filter(Deadline >= dt)

effort <-
	drive_get("research_log") |>
	read_sheet(sheet = "calendar_sheet") |>
	mutate(
		Start = date_start(Start, "minute"),
		End = date_end(End, "minute")
	) |>
	mutate(Duration = difftime(End, Start, units = "hours")) |>
	filter(str_detect(Calendar, "Clinical|Research|Academics|Administrative")) |>
	filter(Duration < 24) |>
	mutate(Type = gsub("Research/", "", gsub("/Service", "", gsub("/Extracurricular", "", Calendar)))) |>
	select(-c(Calendar, Description)) |>
	filter(Start >= dt) |>
	filter(Start > dt & End < add_days(dt, 7)) |>
	select(Type, Title, Duration) |>
	mutate(Total = as.numeric(sum(Duration))) |>
	mutate(Effort = as.numeric(Duration)/Total) |>
	group_by(Type) |>
	summarize(Effort = paste0(signif(100 * sum(Effort), 2), "%")) 

tbl_status_updates(progress, effort)
```

# June 10, 2024

## Updates {.smallest}

1. AFEQT/SDOH draft manuscript revisions
	- Potential journals: Annals of Internal Medicine, JAMA IM, Circulation: Quality/Outcomes, JACC EP
1. Cluster access requested!
	- [ ] Email for training/intro session as account creation completed
1. AHA abstract + Early Career Award application (both extended to June 10 and June 13 respectively)
1. AF-CTRS
	- [ ] Meet with AF-CTRS team to review 
	- Aim to have generic weights in ~2 weeks
1. Future collaborations/projects
	- [x] SS and DS - will continue to provide mentoring support remotely
	- [ ] Continue to work on AF-CTRS + AF phenotying work
	- [ ] ECG/genetics related 
	- [ ] ACER-based CCTS data pulls
	
## Research products

- 4 manuscripts (1 published, 1 in preparation, 2 under review)
- 4 co-authored manuscripts (published)
- 3 software packages published, peer-reviewed
- Completion of an NIH-F32 award
- K23 aims with 50-60% preliminary data collected
- AFGen fellowship
- ECG/genetics with UIC AF Registry + UK Biobank project/proposal

# June 3, 2024

## Updates {.smallest}

1. AFEQT/SDOH draft pending feedback from Alvaro/Emelia
1. Cluster access has been set-up for the lab
1. AHA application for Early Career Award 
1. AF-CTRS weights on Crossover Study dataset + AFEQT dataset
1. UK Biobank proposal/access tentatively approved (still waiting for feedback from AFGen Consortium) 
	- Structural/sarcomeric genes
	- Ion channel genes
	- Metabolism genes
	- Immune/inflammation-related genes
1. Future meetings will likely be virtual

## LMNA/ECG {.smallest}

Premises: 

- *All* LMNA patients will develop AF, but contribution of genetics to how AF develops is unknown (prior to or after cardiomyopathy)

Cases/controls:

- sinus beat in LMNA patient prior to AF development
- sinus beat in LMNA patient after AF development
- sinus beats in AF patients *wild-type*, age-matched
- sinus beats in *wild-type* patients without AF, age-matched

Power:

- **Likely** between 30-60 patients, with another 10-15 for validation *although can likely get adequate power with lower number*

# May 20, 2024

## Updates {.smallest}

- [ ] Re-draft manuscript to send to full co-author list
- [ ] Application for AHA Early Career Award (Epi), requires *unsubmitted* manuscript + abstract for consideration
- [ ] ACER to set-up cluster space for our data-pull pipeline
- [ ] AF-CTRS software test on cross-over study dataset (Ashwini, Jerry, and Miles)
- [ ] UK Biobank proposal to AFGen group 

# May 15, 2024

## Updates {.smallest}

AFEQT/SDOH:

- [ ] Meeting with Alvaro on 5/15
- [ ] Race/ethnicity: $NHB$ and $HL$ populations had significant decrease particularly in AFEQT treatment domain over that of NDI quartile
- [ ] Interaction testing: generally, there are no binomial interactions in the dataset for SDOH factors, however there is three way interaction for $NDI_{Q4} \times NHB \times Male$
- [ ] Re-draft manuscript to send to full co-author list
- [ ] Application for AHA Early Career Award (Epi), requires *unsubmitted* manuscript + abstract for consideration

SDOH research:

- [ ] DT/JA requested retrieval of SDOH variables for the sub-population of chemotherapy patients 
- [ ] Resident x1 and Student x1 proposed SDOH research projects &rarr; present to lab for feedback

ECG/Genetics:

- [x] Annotate all arrhythmia genes (PolyPhen, SIFT, REVEL, LOF/indel) in WES cohort
- [ ] Generate PGS on all 300 WES patients $\pm$ additional 600 GWAS patients
- [ ] Train models for structural, ion channel, and PGS in interim

## Logistics {.smaller}

1. Meet with ACER initially to set-up shared workspaces for the lab, and to provide cost estimates for compute time.
1. Discuss need for PHI-safe data storage on cluster/remote server for combination of [all clinical data and ECG data]{.red}
1. UIC/UIH access...
	a. Complete ECG-based phenotype/genotype preliminary data for K23/CDA
	a. Manage ECG-dataset (`n = 800,000`)
	a. Update both CCTS/MUSE datasets
	a. Pipeline for data pull requests (MRN, code, diagnosis, etc)
1. Timeline for research updates for SS and DS

# May 6, 2024

## Updates {.smaller}

1. AFEQT/SDOH: manuscript, pending initial co-author review. Meeting with Alvaro Alonso on 5/17 to discuss informally. Then likely send to additional senior authors.
1. ECG/genetics: validation cohort proposal completed
1. AF-CTRS: Have drafted initial scoring algorithm. Need to trial basic weights on a dataset.
1. CV research data: Meeting with ACER to help host a safe cloud environment for future data pulls.

# April 29, 2024

## Updates {.smaller}

1. AFEQT paper to be disseminated to co-authors by Wednesday 5/1
1. ECG/genetics: UK Biobank Proposal to be shared to mentors by 5/3
1. K23 aims: draft to share with key mentors on 5/6
1. AF-CTRS: writing out different weighting options currently
1. LLM: upcoming meetings on 5/3
1. AF phenotyping/trajectories: pending application/integration of DS and SS coding projects into our CCTS dataset
1. Cluster access for data - need to discuss lab-wide access / data maintenance (low priority for now)

# April 26, 2024 

## Updates {.smaller}

AFEQT paper:

- [x] Introduction
- [x] Methods
- [x] Results
- [ ] Discussion

ECG/genetics:

- [ ] UK Biobank Proposal
- [ ] PGS on local GWAS
- [x] VCF annotation pipeline

# April 22, 2024

## Tasks

- [ ] Completion of AFEQT/SDOH paper week
	- [x] Dataset for JBVA cohort
- [x] Cardiovascular computational biorepository pipeline
	- [ ] Consider data-sharing for LLM project with Dr. Philip Yu (PI)
- [x] AFL/FH accepted and submitted, pending publication
- [ ] ECG and genetic variants prediction training
	- [ ] UK Biobank ECG/genetics proposal
	
## AF-CTRS

```{mermaid}
flowchart LR
	score[AF Composite Treatment Response]
	burden[Frequency/duration of AF]
	sx[Symptoms]
	ecg[ECG]
	mct[Continuous telemetry]
	afeqt[AFEQT score]
	fc[Functional class]
	
	sx --> score
	burden --> score
	
	fc --> sx
	afeqt --> sx
	
	ecg --> burden
	mct --> burden
```

## AF-CTRS draft

- $ECG = ECG_{sinus} \times n_{sequential}$
- $ECG = \frac{ECG_{sinus} - ECG_{AF}}{ECG_{total}}$
- $MCT = Burden_{sinus} \times days$
- $AFEQT = AFEQT_{total} / 10$
- $FC = 5 - NYHA$

::: aside
E.g. if x3 ECG in sinus in a row, and no MCT data, likely has __low burden__ of AF
:::

---

A "true" model would need to impute missing data, but this would require multiple datasets for training/testing/validation

- [ ] Account for missing data in a rough model (make assumptions about AF burden)
- [ ] Adjust strength and weight of *rough* parameters
- [ ] Account for intervention (pacemaker, PVI, etc)
- [ ] Help in implementing this in AF Registry
	
# April 8, 2024

## Updates {.smallest}

- ECG/Genetic Variants
	- Re-analyze/label sarcomeric/ion channel/"wild-type" AF WES dataset
	- AHA submission for abstract
	- Pending validation cohort
- AFEQT/SDOH
	- Table 1: Baseline characteristics of full cohort (longitudinal $\rightarrow$ supplement)
	- Table 2: AFEQT change by SDOH (NDI and ethnicity)
	- Table 3: AFEQT change predictors (SDOH)
	- Table 4: Rhythm/rate control strategy predictors
	- Figure 1: Study flow-chart
	- Figure 2: AFEQT predictive means over time (by cluster)
	- Pending data collection
- AF-CTRS Score
	- Meeting with MB and JS

# April 1, 2024

## Updates {.smaller}

- AFEQT/SDOH
	- Additional analysis on cross-sectional cohort
	- Conditional/clustered analyses for predictive means
- Atrial Fibrillation Composite Treatment Response Score (AF-CTRS)
	- AF-CTRS to be used to predict efficacy of treatment (regardless of burden % or AFEQT score)
	- Output is algorithm that ties "response parameters" to a dose-responsive score $\rightarrow$ validate in AF Registry
	- Algorithm: ECG findings, AFEQT score, clinical parameters, MCT/ILR (require complex MI)
	
## ECG Prediction of Structural and Ion Channel Mutants in AF {.smaller}

This is an expansion of the __ECG/TTN__ project, which found [ECG could feasibly provide above-chance accuracy in identify TTN LOF variants]{.underline}.
Now, expand to additional *wild-types* and *mutants*...

- Sarcomeric/structural variants (e.g. TTN, PITX2, LMNA, NUP155, GJA1/5)
- Ionic channel variants (e.g. KCNE1-5, KCNQ1, SCN5A)
- No known genetic contributors (excluding any patient with non-benign VUS in any above genes $\pm$ SNPs associated with polygenic risk for AF)

Models, trained on ECG in sinus rhythm in those with known AF, [will need validation]{.underline}. 
However, should publication happen [before or after]{.underline} validation (e.g. with TOPMed/UK Biobank)

# March 29, 2024

## Update {.smaller}

- AFL/FH:
	- Revisions complete, pending table placement considerations
- ECG/TTN:
	- Results presented to AFGen fellows
	- Revised project proposal in conjunction with AFGen (for UK Biobank)
	- Potential K23 aim
- AFEQT/SDOH:
	- Additional analysis on cross-sectional cohort
	- Conditional/clustered analyses for predictive means
	
## ECG/TTN {.smaller}

Can ECG discriminate between [wild-type AF and mutant-type]{.underline} with known, causal/associated genes?

- Increase scope to include additional monogenic genes
- Train to target individual cases based on genetic variant
- Train to classify those with *no* genetic mutations
- Validate in UK Biobank

# March 25, 2024

## Updates {.smaller}

- ECG/TTN
	- Draft analysis complete, presenting to AFGen fellows on 3/27
	- Brainstorm scope of paper
- AFEQT/SDOH
	- Positive feedback from co-author on results draft
	- Potential expansion of "baseline" cohort (AFEQT)
	- Paper writing (new draft in progress), expect 2 weeks for first draft $\pm$ new analyses
~ AFL/FMHX
	- Revisions in progress, responses likely Thursday/Friday

# February 26, 2024

## Updates {.smaller}

- AFEQT/SDOH
	- data cleaning started on JBVA data, largest hurdle is getting NDI data from addresses
	- have rough draft of analysis plan, with plans to get early feedback from Alvaro (standing meeting next week)
- ECG/TTN
	- Filtering pipeline effective for TTN variant, will review with Choi (have 8-10 P/LP variants that are potentially LOF)
	- Presentation on 3/27 with AFGen group
- K23
	- Feedback from Benjamin: after refining genetics aim, can consider implementation in UK Biobank 
	- Feedback from Ranjan: need to consider the sponsorship/mentorship committee at Utah (e.g. Martin Tristani-Firouzi, Rob McLeod, Jared Bunch, etc)

## AFEQT/SDOH paper proposal {.smaller}

1. Table = cohort description by baseline AFEQT scores, with sections on clinical covariates, social determinants, etc
1. Table = Cohort description with statistical testing by AFEQT score changes
1. Figure = AFEQT score breakdown in the population, with subscales (bar graph), showing score changes
1. Table = regression model for SDOH factors and AFEQT change
1. Table = mediation analysis for SDOH factors, treatment strategies, and AFEQT scores
1. Figure = forest plot of interaction of the __highest yield__ SDOH marker and AFEQT score changes

# February 12, 2024

## Whole Exome Sequencing {.smaller}

In light of project-focus on deleterious TTN variants and a/w ECG features, needed a statistically appropriate method for variant selection.

- Using *variant effect prediction* software, including loss-of-function predction, Polyphen, SIFT, and phenotypic data, __created software pipeline to analysis WES__
- Each WES VCF, roughly 50-100 Mb in size, can be annotated to 500-800 Mb of annotations based on dozens of classifier models (called __consequences__)
- These __consequence__ additions are part of Choi's recommendation for having agreement in >80% of predictions to serve as deleterious variant "ground truth"
- Need to manually adjudicate the TTN variants, thus far is 1) reproducible with different prediction cut-offs, 2) identifies some *miRNA* sequenes that may be consequential, 3) has high degree of confidence in the LOF and INDEL variants

::: {.callout-tip}
Variant pipeline requires combination of cluster + local software, and is stable for running new variants PRN. 
:::

## AFEQT/SDOH

1. VA data cleaning, expected 10-12 hours of cleaning
1. Results and tables to be created throughout this week, pending VA data
1. Need to decide on key figures and tables for the paper
1. Expect manuscript draft (dependent on results) *week of 2/23*

# January 29, 2024

## Updates

- AFEQT/SDOH: CDW data pull pending, may be able to get all ICD codes $\pm$ medicaions
- AFL/FH: Completing revisions to paper, sharing in next 2-3 days
- ECG/TTN: focusing on variant pipeline currently

# January 21, 2024

## ECG/**TTN** Project

- Variant annotation pipeline drafted, with classification based on series of annotation tools used in parallel
- AFGen fellowship: suggested minimum threshold of >80% of the ~30 annotation system to be in agreement for tentative "ground truth" for the "severe" or LOF variants
- Algorithm for mis-sense variants being developed

## **ECG**/TTN Project

- Have developed an **alpha** version of end-to-end pipeline for sinus beat analysis
- Utilizing time varying machine learning layers to add power for *repeat* samples and *time-to* samples
- UK Biobank may be a fair option for validation if race/ethnicity factors isn't a major concern

# January 19, 2024

## AFL/FH Revisions {.smaller}

- Reviewer #1: No comments as had no initial comments or suggestions on initial submission
- Reviewer #2: Complimentary of revisions and thought the concerns had been *satisfactorily addressed*
- Reviewer #3: Thankful for *thoughtful and detailed responses* to questions
- EDI Reviewer: Thought comments had been *appropriately addressed*

::: {.callout-important}
*A statistical reviewer has entered the fray*
:::

## Statistical Review {.smallest}

- "The study is described in the abstract as a prospective, observational cohort but it seems to be a cross-sectional analysis. Can the authors please clarify the design."
- "The abstract also notes that the primary independent variable is family history of atrial fibrillation in first degree relatives, and that secondary independent variables include self-reported race-ethnicity and sex. The statistical analysis section indicates that the secondary exposure includes race-ethnicity, sex, and potential modifiable risk factors. The title suggests that interest lies in the association between family history and early onset atrial flutter and that race/ethnicity might be a potential confounder as the title suggests exploring the association in racial/ethnic subgroups. Nothing is mentioned about sex. The focus of the paper needs to be made very clear throughout."
- "The statistical analysis section is confusing."
- "The section indicates that multivariate (should be multivariable) logistic regression analysis was used with sequential adjustment for sex, race, smoking, etc. Sex and race should not be listed as potential confounders (per point above – please clarify the role of these variables in the analysis)."
- "The analysis in Tables 2 are unadjusted but very problematic given the 10-year age difference in individuals with early- versus late-onset atrial flutter.  These should be, at a minimum, adjusted for age."
- "The statistical analysis section indicates the use of multivariable logistic regression, mixed effects models and interaction models – but the analysis presented are almost all unadjusted."
- "Table 3 stratifies by family history. Why stratify by the primary independent variable? The sample size is also too small in the family history group."
- "The statistical analysis approach needs to be better organized to address the key questions."
- "In the abstract, please clarify that the odds ratios that are reported are all unadjusted, if this is the case."
- "Throughout, the authors refer to effect size when they mean effect – it might even be clearer to refer to odds ratios rather than effects here."
- "The language in the clinical implications section needs to be toned down as it infers causality – e.g., individuals with a family history and individuals who self-identify as Black had higher odds."
- "Lastly, I am not sure the Figure is needed."

## Statistical Response {.smaller}

- [ ] Study design: Although patients were enrolled in a prospective, observational manner, the analysis is cross-sectional (no *semiparametric* elements)
- [ ] Variable roles: Independent exposure is family history, while race/ethnicity is a potential confounder in a highly diverse cohort. Sex + other risk factors are secondary exposures.
- [ ] Statistical analysis section: Word-choice and revision are clarified. Specific models are labeled explicitly in results and tables.
- [ ] Table 3: Concern about analyzing by family history strata. Table 1 can instead by changed to strata by exposure (instead of outcome). Can include subgroup analysis as alternative table for all variables. 
- [ ] Odds Ratios: Language changed to reflect intent (e.g. **effect** and **OR** instead of **effect size**).
- [ ] Clinical Implications: Language toned down to reflect observational nature of study.
- [ ] Figure: Justification on why family history and pedigrees are relevant even though this is observational study. Consideration for supplement.

# January 8, 2024

## AFEQT/SDOH background

- Currently there is a push for social determinant research to be integrated into atrial fibrillation research [@Benjamin2023]
- Social determinants in AF research has been predominantly oriented at diagnosis and clinical outcomes [@Essien2021]
- Neither race, ethnicity, or socioeconomic differences have significant research progress in AF diagnosis or management [Norby2021]

*We have a rare opportunity to evaluate how __symptom burden__ is affected by SDOH factors in a multi-ethnic population.*

# December 18, 2023

## AFEQT/SDOH updates 

- [x] Data collection has been completed for UIC patients
- [ ] Initial analyses to be performed this week (target: 12/22/23)
- [ ] Will need to collect JBVA data = will need Jerry's help... will send data collection template
- [ ] Continue to collect additional AFEQTs from Hispanic/Latinx individuals = need to speak with Annette about focusing on Spanish-speakers

# December 11, 2023

## ECG & TTN Variant Project {.smaller}

[Background]{.underline}: VUS/LP/P TTN variants are associated with increased incidence of EOAF. The mechanism is not fully understood. 

[Aims]{.underline}: Identify if VUS/LP/P TTN variants may cause structural atrial changes (an atrial myopathy) that is identifiable or ECG, or other related conduction changes. We [aim to identify if VUS/LP/P TTN variants can be classified on ECG using signal processing techniques]{.red}

[Approach]{.underline}: Utilize sinus rhythm ECGs from patients with WES/CGS with paroxysmal AF. Develop beat-by-beat algorithm to identify features that may be associated with VUS/LP/P TTN variants.

## ECG data {.smaller}

- `n = 298` patients were available and had ECG in sinus rhythm at some point from 2010 to 2023
- This amounted to `n = 7953` unique ECG that were thought to be in sinus rhythm
- Using a wavelet-decomposition approach, identified morphology of P wave, QRS complex, and T wave and created single-beat structures
- Using these `n = 51493` beats, there were `n = 6202` [cases]{.red} (which represents `n = 36` individuals with potential pathogenic TTN variants (based on SIFT/PolyPhen scores)

## Model {.smaller}

- Sequential convoluted neural network approach
- Masking layers to control for `0` values (representing non-depolarized intervals and padded sequences)
- Use combination of `relu` (rectified linear activation) and increasing `dense` layers (with signal drop out)
- Categorize using a `softmax` approach for classification between case/control
- Initial training/testing to __first-beat-only__ approach to avoid sample bias
- Trained for 10 `epochs` based on model accuracy and minimizing `loss` on each repetition

## Results

[Training]{.red}

- Training accuracy was `88%` with only `2%` loss after total of 10 steps

[Testing]{.red}

- Testing accuracy was `85%` with `13%` loss
- Example: single beat (control) may have a confidence of `75%` in being a control

---

![First 10 epochs of training](../figures/keras-accuracy-121023.png)

## Next steps

1. Confirm the TTN variant cases, trialing different definitions of pathological beats
1. Extend analysis to full data-set
1. Modify approaches in terms of machine learning "layers"

# December 4, 2023

## HRS abstract proposals {.smaller}

- [TTN variant prediction]{.red}: Would like to submit abstract, currently training an ECG-based model to classify variants (pathogenic vs. controls). Training data of approximately 50-60k sinus beats, with 500-700 "pathogenic TTN cases". *Data augmented by breaking ECGs into individual beats*
- [AFL & Family History]{.red}: Abstract to be submitted, with equivalent co-authors as manuscript submission
- [TTN variants and EP outcomes]{.red}: Gursuk/James and others wanted to pursue EP-based outcomes. Have ~9 cases of PVI performed on TTN variants that are non-benign (e.g. VUS or LP/P). 

# November 20, 2023

## AFL/FH updates 

- [ ] Feedback? Should we request for an extension?
- [ ] Circulate amongst co-authors

*Why do we believe that Black v. Non-Black individuals had higher rates of AFL despite balanced risk factors?*

## AF Registry and CCTS Data

1. Have all EMR data from 2010 to 2023 now, which increased file sizes by 30% (now over 70 Gb), and increased `n = 140k` to `n = 210k`
1. Have all ECG for same patient population (approximately 40 Gb in XML format)
1. Created `parquet` format to utilize Apache Arrow for I/O

- [ ] Development of filtering pipeline for MRNs, diagnoses, medications, procedures, etc.

## AFEQT/SDOH {.smaller}

Which key variables should be included for evaluation? Some of these have been studied, but not in context of AFEQT/symptom burden

- ECG factors at baseline compared to at time of repeat evaluation (heart rate, rhythm)
- Maintenance of sinus rhythm (percent burden of AF, frequency of ECG in sinus rhythm)
- Medication changes or intensification (addition of anti-arrhythmic)
- Referral to cardioversion or ablation
- Administration of anticoagulation therapy

## ECG/TTN

Timeline for abstract submission to HRS (12/15):

- Complete initial model training on pathogenic TTN cases
- Exclude other TTN variants from analysis to have *cleaner* controls
- Identify appropriate metrics (e.g. AUC, net classification, etc)
- Identify if segments of ECG are more highly weighted (variance importance)

# November 13, 2023

## Updates {.smaller}

- [ ] AFL/FH paper revisions before resubmission (**deadline 12/05**), would like feedback in next 3-4 days, then *circulate to co-authors*
- [ ] Clean CCTS/AF Registry data over next 2 weeks, with anticipation of AFEQT being complete in that time-frame
- [ ] Submit rank list for EP fellowship on Wednesday 11/15
- [ ] Meetings with Emilia Benjamin and Seung H. Choi in next 2-3 weeks to discuss AF phenotyping project
- [ ] PRS calculations with SNPs from NW

# October 26, 2023 

## AFL/FH Simple Revisions {.smaller}

- Conform with AHA guidelines on reporting health differences by races
- Including __Clinical Perspective__ section
- More detailed methods on patient selection and recruitment
- Directionality when discussing associations (e.g. weight versus *increased weight*)
- Add definitions for social drinking
- Clarify anti-hypertensive agents and selection
- Clarify results and statistical methods to match discussion
- Requesting how and why we defined EOAFL as age < 65

## AFL/FH Complex Revisions {.smaller}

1. Concern is the definition of family history to include AF, HF, and CVA, as is __inconsistent__ with the idea of EOAFL as a separate entity. *Unsure how to respond*
1. Can occult AF be analyzed as a comparative group? *Unsure about an additional dataset. However would strengthen the paper if we had a matched cohort as sensitivity analysis.*
1. In those without stated FH, the FDR were not contacted - reviewers are concerned that frequently patients are unaware of FH, and recommended contacting every study participant.  *Two paths: 1) Ignore this and state we had a rigorous approach to FH questionnaires; 2) Pursue the ~300 patients with EOAFL and contact FDR to limit confounding.*
1. Provide a hypothesis and reasoning for the choice of variables. Particularly that of race. *Would refer to genetics and poor representation in other studies.*
1. Concern for why we emphasized difference between Black race and White/Latinx race. *Would compare Black to non-Black instead*
1. Hypothesis on inverse relationsihp between HTN/BB/CCB and EOAFL risk 

# October 23, 2023 

## Updates {.smaller}

AFEQT/SDOH: Current focus is on obtaining updated AFEQT, led by *JS*, with goal to complete AFEQT in next several weeks. In parallel, using EMR-based records to evaluate clinical outcomes. It will take roughly 30 hours of statistical work to incorporate the clinical and AFEQT data (roughly 1-2 weeks), once the AFEQTs are done.

AFL/FH: Our good news was that JAHA is willing to review our paper again with revisions. A majority of these will likely be completed within the next 4-5 days (deadline = 10/31) but have to decide on if new/additional FH data collection is needed. The most challenging critique was the potential bias in FH reporting during initial screening. To overcome this, the most thorough option would be to obtain FH data from every patient to avoid systemic errors/bias. However, a limited approach of verifying FH in just the EOAFL may be reasonable. Would like everyone's thoughts here.

Phenotype AF: Two students have joined this effort, *SS* and *SD*, who will be working on initially on the DSP and ML aspects. Current priority is to finish training the models using the TTN data. Unfortunately, this has been delayed with Boards and interviews, but hope to speed this back up in the next 1-2 weeks.

AF/PRS: For our evaluation of the PRS in AF in ethnic cohorts, we are meeting with the eMERGE group (NW). We will need to understand if the PRS can be validated in our UIC cohort, and if there is an existing mechanism to integrate a GIRA with our own EHR.

# September 18, 2023

## EMR Data

This has been somewhat streamlined to [extract data]{.green} from the CCTS data pull, with full cohort results returned from CCTS.

1. Need to extract AF registry patients and add SDOH, medication intensification, and procedure reports.
1. Working with Tofovic/Nischal to extract echocardiogram data
1. Extracted roughly 8k ECG, digitized, and segmented to individual beats, which provides roughly 40k samples for testing amongst the ~300 WES patients
1. Working with Chen to complete power calculations based on our population and current literature

## AFEQT/SDOH

- Spoke with Jerry/Muriel about re-organizing the AFEQT data for manuscript
- Will need to complete roughly `n=600` follow-up AFEQT
- Power calculation suggest would need `n=200` Hispanic/Latinx
- Timeline TBD (expect ~ 4-6 weeks)
- Subsequently will need to extract patient outcomes (intensification, burden) from CCTS pull
- Adding Angela Hussain to data collection as student researcher
- Check in with Annette to help with Spanish-speaking requirements

## AF/Genetics

Met with AFGen group (Patrick Elinor, Emilia Benjamin), encouraged computational approach in genetics. 
Currently working to filter and annotate our WES dataset, but expanding to novel gene regions (e.g. neurocardiac receptors) that are not included in general cardiomyopathy/arrhythmia panels.

1. Variant evaluation/assessment of different reference genomes
1. Variant filtering based on SIFT/PolyPhen
1. Annotation based on multiple sources (USCS, NCBI, etc)
1. Inclusion of broader, related genes (e.g. **ADRB2** not included in recent evaluation)

# September 11, 2023

## AFEQT {.smaller}

Current manuscript has `n=453` individuals split amongst races, with some strength of assocation b/w SES/SDOH and ∆AFEQT. 

- Have `n=760` baseline AFEQT data points, but need roughly `n=600` additional follow-up AFEQT scores
- Have `n=1506` currently in AF Registry, thus `743` more patients that can be added
- Would allow us to be powered for mediation analysis, and stratified analyses
- Able to incorporate [intensification]{.orange} and [recurrence]{.orange} rates now, as well as census tract (vs. zip) and language barriers now

---

[Next]{.underline}: 

1. Complete the additional `600` AFEQT follow-ups
2. Extract data from CCTS to "flush out" registry data
3. More specific/appropriate analyses for SDOH including [mediation]{.blue}

## PRS for AF/eMERGE

1. We are meeting with NW and the eMERGE group to validate PRS in AA/HL population here at UIC this week.
1. Met with Hua Yua Chen to go over PRS power calculations 
1. Generating multiple sample size/power calculations based on C-statistics seen in other AF/PRS papers &rarr; however may not apply equivalently in AA/HL population as primarily derived from Caucasian populations

# September 8, 2023

## AF/Genetics {.smaller}

Completed [computational genetics coursework]{.grey} on variant analysis. Although some work has already been done on this, going through the basics will allow us to have in-house [variant annotation pipeline]{.green}

1. Uploaded our ~2G of patients to cluster 
1. Generated functions to [split VCF]{.green} by individual case 
1. Identified variant locations 

[Next]{.underline}: Evaluate variants using **PolyPhen** and **SIFT** databases

## ECG/pAF {.smaller}

Recently was asked to review/adjudicate ECG for the PhysioNet challenge on AF prediction algorithms. In line with my interest at phenotying AF based on ECG markers. 

1. Segmentation of individual beats has been automated 
1. Augmenting the data to individual beats (similar to @vandeLeur2021)

[Next]{.underline}: Training on labeled [AF patients with TTN variants]{.orange}

---

![Annotation of individual beats](../figures/shiva-ecg-annotated.png){#fig-annotated-ecg}

## AF/SDOH

- eMERGE network meeting this week
- Processing UIC/CCTS data 
- Uploaded January to June ECG data from MUSE
- Will still need updates from CCTS for 2023 remaining clinical data
- Have to solidify definitions for [therapy intensification]{.red} and [AF outcomes]{.red}

# August 21, 2023

## Phenotyping AF {.smaller}

*Identifying phenotypes of AF that are at risk of progression (e.g. developing higher burden, persistent AF, TICM, etc)*

1. Extracted clinical covariates for ~6k patients, stratified by time/onset (this is ~7 million diagnoses)
1. Extracted medications by time/duration of prescription
	a. Filtered down to all cardiovascular medications
	a. Classified medications that are relevant to AF (AC/AP, AAD)
	a. 2.5 million prescriptions were analyzed
1. Evaluated ECG related to this population and initiated segmentation/feature extraction
1. **Goals**: Pull procedure reports to assess burden, and process text-based echocardiography data

## ECG and Genetic Variants {.smaller}

1. Completed the software needed to annotate ECGs on single leads
1. Will have to evaluate efficacy amongst 12-lead systems (e.g. 5000 x 12 array)
1. Using [semantic segmentation]{.green} approach to classify waveforms
1. Have started [cluster-based]{.green} pipeline development for re-calling VCF to identify variants based on...
	a. Multiple different population panels (different allele frequencies)
	a. Filter on read depth and quality
	a. *Will need to learn about other filtering approaches*
	
## SDOH/AF/HSR {.smaller}

- Current knowledge gap is interaction between race/ethnicity and socioeconomic status in management of AF [@Essien2021]
- Similar to AF Registry + Cluster, can use the definition of **treatment intensification** as part of health care differences
- Parent project to AF/language 
- Unlike other cohorts (e.g. ARIC had `n=3200` but only `n=500` were Black), have more diverse population pool

# August 14, 2023

## AFL/FH {.smaller}

Completed revisions to paper with tentative plans for re-submission.

1. Called `n=31` families to confirm AFL/AF diagnoses and clarify CHF and CVA histories
1. Re-analyzed pedigrees with multiple definitions of +FH in FDRs as a *sensitivity analysis*
1. Major findings remain same (and **robust** to adjustment):
	- +FH doubles risk for "premature" or earlier onset of AFL
	- EOAFL as corrolary increases likelihood of AF and AFL in FDRs
	- Black race a/w EOAFL, partially mediated through OSA/ETOH (most prominent in this subgroup)
1. Resubmission to... **JAHA**, **AHJ**, **IJC**, **BMJ Heart**

## AF/SDOH {.smaller}

- `n=4537` patients (still missing updated CCTS pull of about 2k)
- 92% have NDI via census tract
- 52% have social histories completed
- 81% are English speaking, 10% Spanish
- Medications/Labs/Vitals/Visits/Comorbidities being evaluated currently

## AF Phenotyping {.smaller}

![P wave detection](../figures/saclova-2022-p-wave-detection.png){#fig-pwave-detection}

Working on ECG beat segmentation and annotation using machine learning approach across multiple leads. Will be applicable across ECG-based projects. Necessary step prior to creating median beats.

# July 31, 2023

## AFL/FH Reviewer Concerns {.smaller}

1. How many FDR were directly contacted?
1. What constitutes an "atrial arrhythmia" in family members? Often asymptomatic. Needs clear definition
1. Why LOAFL versus EOAFL, instead of EOAFL versus controls?
1. Why is FH/Race significant only in a single racial group?
1. Overstatement of importance of FH over *4q25* papers in isolated AFL
1. How are CVA events labeled as cardioembolic?
1. Why include CHF as part of arrhythmia risk factors in FDR?
1. Define excess ETOH usage
1. Pedigree needs to be labeled more specifically/informatively

# July 24, 2023

## Active projects

1. AFL/FH: Revisions to be addressed in next draft. Will identify additional journals to target.
1. HRV/CVD: EHJ 
1. ECG/PAF: Active project involving computational approach at identifying sub-phenotypes of paroxysmal AF based on ECG-driven parameters *with subsequent work on clinical patterns and trajectory*

## Phenotyping AF

Project is a "big data" project using over clinical data from 2010 to 2023, supplementing the AF Registry with *data mining* and *machine learning* approaches.

Composed of three elements:

1. ECG data in form 12-leads of 10 second signals, sampled at 500 Hz from 2010 to now
1. Clinical structured documentation including vitals, medications, labs, echo/study reports from 2010 to now
1. Clinical unstructured data from MD documentation

## Data overview 

ECG data: 

- 500k ECGs collected from 2010 to 2013
- 70 Gb of raw XML data (converts to ~50 Gb of WFDB)

Raw clinical data: 

- 43 Gb raw data (70% is from clinical notes alone)
- 145k unique patients with CVD diagnoses (cardiomyopathy, arrhythmia, vascular dz, etc)

## AF-specific data

ECG data:

- 33k ECG with a diagnosis of an atrial arrhythmia
- 6k unique patients with a diagnosis of AF confirmed by ECG
- Of those 6k patients, 95k ECG taken in total (both sinus + AF)

Clinical data:

- Of the 6k patients c- AF on ECG, 3.9k have clinical data available from CCTS
- Aggregate file size for this 3.9k is < 1 Gb in total
- 600k ICD diagnoses, 4.5 million lab results, 500k visits, 1.5 million vital sign measurements, 2.9 million medications

## Project status

Currently have working pipeline for extracting relevant diagnoses, medications, labs, etc, for additional projects. Can email ashah282@uic.edu for questions or project proposals.

Can see current proposal [here](https://asshah4.github.io/research/t32/phenotype-af.html)

# June 12, 2023

## Backbone for aims {.smaller}

1. Sympathetically-mediated AF occurs predominantly earlier in paroxysmal AF, in hearts with minimal scar burden. 
1. To cause episodes of AF would require abnormalities in conduction from both intracardiac and extracardiac causes
1. Abnormalities from both intracardiac and extracardiac causes are either acquired or genetic
1. These factors may also lead to different rates of progression of AF

[Question]{.underline}: Do certain genetic variants contribute to the progression of paroxysmal AF to persistent AF? 

## Trajectory of pAF {.smaller}

- @Kerr2005 from a Canadian Registry followed `n = 757` for ~8 years, 5 year recurrence was >60%
	- Positive predictors = age, +AS, +MR, LAE, reduced LVEF
	- Negative predictor = rapid heart rate during AF 
- Post catheter ablation, 5 years recurrence was 60% (1st ablation), and 20% after (>2 ablations) (@Takigawa2014)
	- Predictors = LAVI and patient age
- @Holmqvist2015 noted progression based on age + increased heart rate in sinus (CHA2DS2VASc had minimal discriminatory power)
- @Andrade2023 evaluated 3 year follow in ablation versus AAD, found `HR = 0.51`
- *Acutely, post-cardiac surgery AF can be predicted by decreased HRV (@Vesela2023)*
	
## Predictors of pAF progression {.smaller}

**CUrrently progression is poorly defined...**

- Clinical: need for rhythm control intervention (ECV, PVI), hospitalization for symptoms or heart failure, % burden increase by an inflection point 
- ECG: Decrease HF-HRV, increased P wave duration/dispersion, decreased P wave amplitude (during AF, decreased F-wave amplitude)

**Persistent** = `n` continous ECG in AF, diagnosis by cardiology, >7 day duration of episodes (or less)

::: aside
How should we define progression, rate of progression, and persistence?
:::

## Approach for evaluation

1. Establish clinical progression of disease with multiple *versions* of the definition
1. Obtain ECG data (segmentation to evaluate sinus beats)
1. Assess genetic variants &harr; ECG characteristics to compare with outcomes

---

![Manhattan plot @Verweij2020](../figures/verweij-2020-manhattan.png){#fig-ecg-genetic-manhattan}

---

![Cluster of genetic variants by ECG component @Verweij2020](../figures/verweij-2020-clusters.png){#fig-ecg-genetic-clusters}

---

![Individual variants with ECG components @Verweij2020](../figures/verweij-2020-ecg-and-variants.png){#fig-ecg-variants-and-components}

## Next steps

- [ ] Obtain ECG data (currently have from 2010 to 2017)
- [ ] Segmentation approach for sinus beats
- [ ] F-wave identification for AF beats
- [x] Contact Broad Institute about SNP-array data from previous GWAS
- [ ] NLP approach to assessing paroxysmal versus persistent

# June 5, 2023

## Aims Progress {.smaller}

Goal: **Identify characteristics that define sympathetically-mediated paroxysmal AF (as compared to scar-mediated pAF) in trajectory of paroxysmal &rarr; persistent**

- Pending mentorship meeting (dates requested for Alvaro, Amit, Dawood)
- Met with Lampert - be much more specific in ANS factors in pAF
	a. Evaluate ECG during AF: coarse AF v. fine AF &rarr; triggering event (ADHF, surgery, etc...)
	a. Evaluate ECG during sinus: P wave morphology/area/dispersion and AF burden
	a. *Evaluate candidate genes (ANS neurohormonal pathways) in Emory Cohort (which has high vagolysis burden) as initial study; validate these genes in AF registry with burden/rate of AF progression*

## AF Phenotyping

- Pending CCTS to upload data (hundreds of GB in size)
- Complete setting up cluster computational pipeline for NLP of unstructured text
- Specific goal is to identify timepoints: onset of AF, transition to persistent AF
- Initiate analysis of ECG data
	1. Convert to annotation format (WFDB)
	1. Identify median P waves per ECG

# May 22, 2023

## ECG analysis {.smaller}

__K23 AIM #1__ is to identify paroxysmal AF into low-scar burden (and high vagolytic triggers) in AF registry.

- [x] Have imported _superset_ of ECGs from MUSE (~500k ECGs) in XML/binary format (including raw signal, diagnoses, timing, etc), approximately ~1 TB
- [x] Convert XML format into digital signal... software developed (same as for _HRX_ abstract), approximately ~250 GB
- [ ] Analysis of ECG markers: high frequency HRV, P wave dispersion, P wave terminal forces, and GEH (Wilson's vector gradient), and if possible to identify "coarseness" of AF
- [ ] Split data into PRE/POST AF diagnosis, and identify control group from other ECGs in superset

::: aside
For EGM and ECG, working with Bhaumik to identify morphology of signals in machine learning project
:::

# May 12, 2023

## Updates {.smaller}

:::: {.columns} 
::: {.column width="50%"}
1. ECG data is being pulled from MUSE in XML format
2. CCTS data pull to be completed today
3. EGM signal processing - work in progress with goal of automated beat generation of multi-channel leads (R/C++ package)
:::
::: {.column width="50%"}
- [ ] Need storage space for ECG data (raw and processed), cluster allows only 50Gb for read/write storage (batch processing): >100k ECG
- [ ] CCTS data will need raw storage space: 10 years of patient charts + notes
:::
::::

# May 8, 2023

## Goals

:::: {.columns} 
::: {.column width="50%"}
1. Phenotyping of AF, as current grouping is inadequate. Focus on risk category changing from paroxysmal to persistent (baseline concept of multi-wavelet reentry)
1. ECG- and EGM-based analysis of AF as additive components
1. CARTO/Rhythmia/EnSite cardiac mapping software to determine LA scar burden 
:::
::: {.column width="50%"}
- [x] NLP coursework, cluster set-up preparation
- [x] HRX abstract preparation on EGM and [Physionet/WFDB](https://physionet.org/content/wfdb/10.7.0/)
- [ ] Identify next paper to start preparing
- [ ] eMERGE/AF IRB _waiting on approval from eMERGE_
- [ ] EPS study IRB _in progress_
:::
::::

## HRx abstract

![Automated EGM annotation of single lead, with high-fidelity signal resolution. 75% file size reduction and 500% read/write speed-up.](../figures/shiva-avnrt-egm.png){#fig-his-bundle-potentials}

# April 17, 2023 

## Goals {.smaller}

:::: {.columns} 
::: {.column width="50%"}
1. Phenotyping of AF, as current grouping is inadequate. Focus on risk category changing from paroxysmal to persistent
1. ECG- and EGM-based analysis of AF as additive components
1. CARTO/Rhythmia/EnSite cardiac mapping software to determine LA scar burden 
:::
::: {.column width="50%"}
- [x] AFL/FH manuscript submission
- [x] HRV/CVD manuscript revisions
- [ ] Identify next paper to start preparing
- [ ] eMERGE/AF IRB
- [ ] EPS study IRB
- [ ] NLP coursework, cluster set-up preparation
- [ ] HRX abstract preparation on EGM and [Physionet/WFDB](https://physionet.org/content/wfdb/10.7.0/)
:::
::::

# April 10, 2023

## AF Ontology {.smaller}

Generation of key features that are related to arrhythmias:

1. Clinical history and trajectory
1. Echocardiographic findings
1. ECG-based features
1. Family and social history
1. Potential genetic markers

---

![Example from HF evaluation and progression](../figures/nagamine-2022-patient-timeline.png){#fig-feature-extraction}

---

![Utilizing patient features to create a matrix-based cluster](../figures/nagamine-2022-feature-generation.png){#fig-feature-gen}

---

![Support-vector based clustering, multidimensional K-clustering in feature-space](../figures/nagamine-2022-heart-failure-clusters.png){#fig-adhf-cluster}

## Phenotype-based WES cohort {.smaller}

:::: {.columns}
::: {.column width="60%"}
| Age | Comorbidities | Proportion |
| - | --- | - |
| - | none | 10-15% |
| &le; 55 | * | 20% |
| &le; 65 | &le; 1 | 20% |
| &le; 65 | &le; 2 | 10% |
| &le; 65 | &le; 3 | 10% |
| &ge; 65 | &le; 1 | 10-15% |
:::
::: {.column width="40%"}
Alternatively can aim for two major groups within paroxysmal cohort definition:

1. &le; 65y + structural heart disease (70%)
1. &le; 65y - structural heart disease (30%)

_Aim for 20-30% to have had PVI to be able to integrate intracardiac findings_.
:::
::::

---

![Autonomic inflexibility and CV mortality in HF and LF bands](../figures/tbl-hrv-groups-cvd.png){#tbl-hrv-grps-cvd}

---

![Improved model concordance with addition of autonomic inflexiblity](../figures/tbl-hrv-concordance.png){#tbl-hrv-concordance}

# March 31, 2023

## AF Phenotyping

NLP combined with DSP can help in identifying sub-phenotypes of AF...

1. Polygenic risk score assessment based on phenotypes and sub-phenotypes
1. Current phenotype approaches limited to structured text 
1. Unstructured data can be extracted from clinical notes using NLP tools: BERT, Sci/RoBERTa, meta mapper, etc

[AWS HPC + CCTS data + AF registry + EP lab + biomarkers]{.green}

---

![Complexity of phenotyping is based on how structured or unstructured the data may be](../figures/emerge-example-creating-phenotypes.png){#fig-emerge-example}

---

![Identifying a true phenotype is complex and most often incorrect unless corroborated through other mechanisms](../figures/emerge-venn-diagram-true-cases.png){#fig-emerge-true-cases}

---

![Phenotypke KB current standard for diagnoses of AF](../figures/phenotype-kb-afib.png){#fig-phenotype-kb-afib}

## EPS signal processing

Utilize combination of surface ECG and intracardiac EGM to...

1. Identify intracardiac features, e.g. multichannel morphologies
1. Evaluate atrial abnormalities, such as conduction left &rarr; right activation 
1. Feed features into LSTM or convolutional neural network to understand different AF phenotypes

---

![Annotated ECG to identify P wave, QRS, and T wave fiducial points](../figures/shiva-annotated-ecg-green.png){#fig-shiva-ecg}

- Summation of P wave force vectors across surface ECG (signal-averaged)
- Identify terminal P wave force abnormalities
- Evaluate changes in P wave morphology over time

# March 20, 2023

## Updates {.smaller}

- AFL/FH:
	- manuscript was sent for initial revisions (03/03/23)
	- feedback received on 03/17/23 from _Jordan_
	- second draft to be returned on 3/20/23
- [Aim 1]{.green}: "Big data" approach using claims-based data, IRB/CCTS approval obtained
	1. NLP/ML area of focus will be initially on AF (only performed x 1)
	1. Approach for recurrent events, Weibull distribution expected
- [HRV/CVD]{.blue} : 
	- priority for this week
	- HF/LF HRV values showing >10 fold HR for bottom versus top quartile (with almost >95% sensitivity)
- Genetic analysis:
	- ACER will start AWS workspace this week
	
## Help Wanted {.smaller}

- Ablation database: working with Wissner to revise ablation database
	- manual data reconciliation
	- OCR/OMR-based PDF conversion approach
	- ablation outcomes x genetics
- AF phenotyping: key part of [Aim 1]{.green}
	- Computational Biorepository for Cardiovascular Disease: CCTS data pull active, all clinical notes, data points, etc since 2010 on CVD
	- AF ontology needed using a NLP/LLM (e.g. BERT)
- NLP/ML for EHR data:
	- CBCD: overlap of DCM/AF registries, CCTS data pull of all clinical notes since 2010 on CVD
	- Key part of __Aim 1__ is to identify AF phenotypes in those with paroxysmal AF
	- AF Ontology: working with Andrew Boyd on AF-NLP framework (SciB)

# February 27, 2023

## Updates

- AFL/FH: will need age of diagnosis of event of family history prior to final analyses
- CAR: analysis pending by Konda, however unclear if we can strongly accept null-hypothesis

## Non-linearity in ANS dysfunction {.smaller}

1. Proportional hazard assumptions
	a. Defined using time-variance, time-interaction, Martingale residuals
	a. Satisfied PH assumptions
1. Non-linearity of HRV and CV mortality
	a. Spline analysis with up to 5 knots
	a. Parametric threshold analysis (survival modeling)
	a. Non-parametric (binary outcome) threshold analysis
	a. Identified __non-linearity__ of response
1. Re-analyzed data using cut-point (as shown in @fig-hf-hrv-reactivity-quartiles)
	a. Almost 100% accuracy in classifier of low-risk cohort
	a. Almost 10-fold hazard in identify high-risk cohort

---

![Increase of ~10 fold in CV mortality in 1/4 patients identified by abnormal resting and reactive vagal tone. Robust classifier of resilience.](../figures/hf-hrv-reactivity-km-quartile-curves.png){#fig-hf-hrv-reactivity-quartiles}

## Genetic analysis {.smaller}

1. WES-generated VCF files on n=28 patients from EO-AFL + FH subgroup
1. Due to size limitations (**no access to HPC**) converted VCF data ...
	a. VCF to Apache Arrow
	a. Arrow conversion to `feather` and to `parquet` for header, annotation, and columnar genotype data
	a. Analyze as in-memory array
	a. Re-vert to ASCII-based format for GZip formated VCF
1. Filtering of data
	a. Limited to coding regions
	a. Filtered for read depth > 20
	a. Compared to dbSNP builds using `SIFT` and `PolyPhen` 
	a. __Pending further annotations based on arrhythmia-panels__

## Specific aims

Have revised [aims](http://asshah4.github.io/research/k23/overview.html), and have drafted the x1 page __specific aims__ to be shared with mentors.

Mentors: D Darbar, AJ Shah, A Alonso, M McCauley, _A Boyd_

# February 20, 2023

## AFL/FH

- Genetic basis for FH may be more promising than association with FH broadly, but would require additional WES to be sent
- Would need to repeat VCF analysis for an arrhythmia panel (instead of cardiomyopathy panel)

## CBCD {.smaller}

Computational biorepository for cardiovascular disease

- Will need a large computational biorepository
	- Claims data from both CPT/ICD codes
	- Clinical documentation (raw text from clinical notes)
	- Medication history
	- Study data e.g. XML of ECG, echo reports, coronary angiograms, device interrogations, etc
- CCTS to pull data, DRA to be submitted today
	
::: aside
Also working on VA research database access, CART-CL reporting data, MVP, ARIC, and UK Biobank as additional large-scale data
:::

# February 6, 2023

## Updates

- AFL/FH: __WES__ needed prior to completing revisions
- ARIC: potential option to pursue genetic profiling in ARIC per Alvaro, need formal proposal
- K23: drafting specific aims in 1-pager format
- VCF: data able to be analyzed/cleaned, however requires more CPU to perform

::: aside
Currently on inpatient service with limited time
:::

---

![Chromosome 1 summary](../figures/vcf-sample-chr-1-from-mike.png){#fig-vcfr-summary}

# January 9, 2023

## Updates

:::: {.columns}
::: {.column width="50%"}
AFL/FH:

- Pedigrees/genetics completed (thanks to [Ana]{.blue}, [Shashank]{.blue}, [Mike]{.blue})
- Outline/draft to be re-written
:::
::: {.column width="50%"}
1. [HRV/CV Mortality]{.blue} paper rejected, need to re-think strategy with senior authors
1. K23 aims to include 1) EP lab as [translational]{.orange} component, 2) [arrhythmia risk prediction]{.blue} as [computational]{.green} component
:::
::::

## [AFL/genetics]{.blue}

- ~87 individuals with +FH
- ~81 individuals with WES
- ~18 individuals with VUS in total
- **1 individual** with VUS + FH

## Multiple Pedigrees

![AFL/surrogate disease burden](../figures/multiple_pedigree_plot.png)
## TTN Mutant

![AFL/mutant](../figures/pedigree_uic0530.png)
# December 19, 2022

## Updates

:::: {.columns}
::: {.column width="50%"}
AFL/FH:

- Pedigrees TBD
- Draft completed, pending feedback
- Supplemental tables needed?
:::
::: {.column width="50%"}
AF/Recurrence:

- Scott added additional patients from UIC (September 2020 to now)
- Data collection pending
:::
::::

## K23 Aims {.smaller}

Objective: _Clinical EP researcher using computational neurocardiology techniques to study arrhythmia mechanisms_

1. Evaluating vagolysis and its effect on triggered arrhythmia mechanisms, both ventricular (e.g. SCD) and atrial (triggered AF)
1. Using computational approach to phenotype triggered onset arrhythmias, e.g. atrial fibrillation

Mentorship Team: __Amit J. Shah__, __Dawood Darbar__, Rachel Lampert, Viola Vaccarino, Mark McCauley, Andrew Boyd, Alvaro Alonso

## K23 Approach

__Vagal tone and triggered arrhythmias__

- Manuscript on stress-induced vagolysis and CV mortality __under review__
- ANS dysfunction during/with ischemia __work-in-progress__
- [Create murine/translational model for vagolysis and triggered arrhythmias]{.arrhythmia[2]}

__Arrhythmia phenotyping__

- AFL/FH manuscript __work-in-progress__
- AF/race project __work-in-progress__
- [Classification of arrhythmia phenotypes using large data set (MVP/VA research data, UIC, ARIC)]{.computational[3]}

## Murine Model of Vagolysis {.smaller}

- Atria are heavily innervated by ANS ganglia
	- Sympathovagal balance locally mediated through adrenergic lysis of cholinergic activity 
	- Neuropeptide Y (NPY) causes cholinergic inhibition through Y2R 
- Pro-arrhythmic murine model (compared to healthy controls)
	- Ex-vivo vagal-sparing Langendorf preparation
	- Catecholamine infusion and NPY Y2R antagonists to modulate arrhythmic state
- Measure atrial conductive properties as outcome
	- Baseline, with catecholamine infusion, VNS stimulation, and Y2R antagonism

# December 12, 2022

## AF Catheter Ablation and Recurrence

Data/power increase:

- Inclusion of VA data
- Additional data pull from EPIC by Scott Uphouse (pending)
- Inclusion of UIC billing code data?

## Atrial Flutter and Family History

:::: {.columns}

::: {.column width="50%"}
Pedigrees:

- How should these be incorporated into the manuscript? 
:::

::: {.column width="50%"}
Genetics:

- 305 patients that have undergone whole exome sequencing, but unclear how to match these to the AFL data
- Should we genotype the rest of the EO-AFL patients?
:::

::::

## ECG/EGM analysis

```{r}
library(shiva)
x <- read_lspro(file = file.path(find_project_folder(), "research", "t32", "sample-egm.txt"))

ggm(x, channels = c("I", "CS", "HIS M", "HIS D", "RV")) 
```


# November 21, 2022

## Updates

- AHA IPA: Completed draft, pending revisions
- AFL Paper: Draft to be done by Wednesday
- AF Ablation + Biorepository: Pending IRB
- Stress and CVD Mortality: Submitted to _Circulation_, revisions Pending

# November 14, 2022  

## Early Onset Atrial Flutter  

1. Draft of paper expanded to include results 
1. Prominent findings of family history and association with arrhythmia 
1. Key tables and figures established  

Next steps:

- Find agreement in key results and figures
- Identify main finding
- Complete discussion section

# October 10, 2022

---

__AF Registry__

Status:

- Current [overview](https://github.com/asshah4/aflubber/R/af_ablation_recurrence.html)
- Not yet powered
- Will need to revise out approach

# October 2, 2022

---

__AF Registry__

- Current number: 219
- Recurrence rate: 46%
- Death = 12 
- MACE = 70 (including repeats)

---

```{r}
#| eval: false
#| echo: false
#| cache: true
store <- file.path(find_project_folder(), "mims", "_targets")
curves <- targets::tar_read(curves, store = store)
curves$lf_stress
```

--- 

```{r}
x <- cardiac_atlas$right_atrium_rao

ggplot() +
	theme_void() +
	geom_sf(data = x$chambers, aes(fill = structure)) +
	geom_sf(data = x$projections, aes(fill = structure)) + 
	geom_sf(data = x$valves, aes(fill = structure)) + 
	geom_sf(data = x$septum, aes(fill = structure)) + 
	geom_sf(data = x$vessels, fill = "white") +
	scale_fill_brewer(palette = "YlGnBu") +
	theme(legend.position = "none") 
```

GIS-based MAP of right atrium for mapping and localization purposes for publications.


# September 26, 2022

---


## AF Registry {.smaller}

:::: {.columns}

::: {.column width="40%"}
__Analysis__

- Genetic analysis in PLINK complete
- Models are set up for analysis once data intake is done
:::

::: {.column width="40%"}
__Status__

1. Have n = 238 currently
1. Reviewed 95 charts thus far (CAR team)
1. Of n = 108, 75 have had recurrence, 33 without (rate of about 70% of anyarrhythmia)
1. Of ECG based confirmation, 43 AF recurrence events (53%)

:::

::::

# September 12, 2022

---

## AF Registry {.smaller}

REDCap data dictionary is finalized. Next steps are:

1. Data entry for outcomes
2. Finalization of new patients to add from AF ablation registry

## Genetics {.smaller}

Currently able to:

- Format large SNP data sets into appropriate tables
- Utilize `PLINK` and `MERLIN` to process Chen's genetics data
- Run from R to help analyze findings

Next steps:

- Confirm ancestry (for practice) percent likelihood 
- Then, once outcome data is complete, analyze differences by race

Analytical methods:

- Time to binary outcome of recurrence
- Recurrent event analysis for AF recurrence
- Can revise with Bayesian modeling in `STAN` as well


# August 22, 2022

---

## AF Registry {.smaller}

:::: {.columns}

::: {.column width="40%"}
__Status__

- REDCap is in progress
- Have an additional list of registry patients that may/may not have had ablation

:::

::: {.column width="40%"}
__Next Steps__

1. Complete 100 REDCap patients
1. Identify 300 patients that have potentially had ablation

:::

::::

---

## CARTO Maps {.smaller}

- Ablation quantification
- Activation mapping
- EGM annotation
- Voltage mapping
- Conduction velocity
- Earliest/latest activation
- Geometry data



# August 15, 2022

--- 

:::: {.columns}
::: {.column width="40%"}
__UIC-wide data summary__

- N ~ XXX had ablation
- Date range from >= 2010

__AF registry data__

- N=~300 had ablation
- Date range from >= 2010

:::
::: {.column width="40%"}
__Next Steps__

1. Generate list of MRN for data collection
1. Use new REDCap to obtain ECG/ablation data
1. Run preliminary analysis on first 100 patients
1. Consider expansion to JBVA/ACMC data
:::
::::

# August 1, 2022

## AF Registry

Consider additional variables:

- Recurrent events and adjudication (HF, cardiac admission, MACE)
- Holter/ECG data and repeat collection (P-wave morphology, Wilson's vector gradient, amplitude of AF waves "coarseness")

Will check with the MESA and ARIC data base on how repeat events were defined.

## Murine EP Studies

```{r, out.width = "60%"}
knitr::include_graphics("../figures/electromap.png")
```

## Human EP Studies

```{r}
knitr::include_graphics("../figures/openep-ablation.png")
```

# July 25, 2022

## Forest Plots

::::{.columns}

:::{.column width="50%"}
```{r}
#| echo: true
#| eval: false
# Forest Plots
library(vlndr)
m <- rx(
	Petal.Length ~ X(Sepal.Length) + Petal.Width + S(Species),
	label = list(
		Petal.Length ~ "Length of Petals", 
		Species ~ "Genus of the Flower"
	),
	pattern = "direct"
	) |>
	fmls(order = 2) |>
	fit(.fit = lm,
			data = iris,
			archetype = TRUE) |>
	mdls()
```
:::

:::{.column width="50%"}
```{r}
#| echo: true
#| eval: false
tbl <- tbl_forest( #<<
	object = m,
	formula = Petal.Length ~ Sepal.Length,
	vars = "Species",
	columns = list(
		beta ~ "Estimate",
		conf ~ "95% Confidence Interval",
		n ~ "Number of Samples"
	),
	axis = list(
		lim ~ c(0, 3),
		breaks ~ c(0, 1, 2),
		lab ~ "ß (95% CI)",
		title ~ "Petal Length by Sepal Length"
	)
)
```
:::

::::

A simple way to make sub-group analysis Forest plots, working on improving customization.

---

```{r}
#| label: Example forest plot
#| echo: false
#| eval: false
tbl |>
	tab_style(style = cell_fill(color = "white"),
						locations = cells_body(columns = everything()))
```

# July 18, 2022

## AF Registry

The AF registry is described [here](https://asshah4.github.io/aflubber). Current issues:

1. Data quality - consistency between reviewers
1. Missingness - decisions on acceptable thresholds
1. Variable selection - echo findings, EP studies, medications, labs, symptoms
1. Management - REDCap, shared excel sheet
1. Adjudication/review - outcomes, clinical follow-up length
