---
title: "Research-In-Progress"
author: "Anish Sanjay Shah, MD/MS<br><span style = 'font-size: 70%;'>Cardiology Fellow<br>University of Illinois at Chicago</span>"
format: 
  revealjs:
    footer: "[Stress Epidemiology]{style='color:#1E87E5FF'}, [Arrhythmia Mechanisms]{style='color:#F8A725FF'}, [Clinical]{style='color:#C1185AFF'}, [Computational Neurocardiology]{style='color:#00887AFF'}"
    scrollable: true
execute:
  cache: true
tbl-cap-location: bottom
fig-cap-location: bottom
---

```{r}
#| label: setup
#| include: false
#| cache: false

# Libraries
library(tidyverse)
library(arcana)
library(durandal)
library(volundr)
library(coraline)
library(aion)
library(card)
library(googledrive)
library(googlesheets4)
library(clock)
library(gt)
library(glue)
library(ggsci)
library(scales)

# Location
find_project_folder <- function() {
	x <- sessionInfo()$running
	if (grepl("mac", x)) {
		file.path("/Users", "asshah4", "projects")
	} else if (grepl("Windows", x)) {
		file.path("C:/Users", "asshah4", "projects")
	}
}

# Used in YAML above as well - lower is darker, higher is lighter
stress <- pal_material("blue", reverse = TRUE)(9)
arrhythmia <- pal_material("yellow", reverse = TRUE)(9)
clinical <- pal_material("pink", reverse = TRUE)(9)
computational <- pal_material("teal", reverse = TRUE)(9)
text <- pal_material("blue-grey", reverse = TRUE)(9)

# Google data
log <- 
	drive_get("research_log") |>
	read_sheet(sheet = "calendar_sheet") |>
	mutate(
		Start = date_start(Start, "minute"),
		End = date_end(End, "minute")
	) |>
	mutate(Duration = difftime(End, Start, units = "hours")) |>
	filter(str_detect(Calendar, "Clinical|Research|Academics|Administrative")) |>
	filter(Duration < 24) |>
	mutate(Type = gsub("Research/", "", gsub("/Service", "", gsub("/Extracurricular", "", Calendar)))) |>
	select(-c(Calendar, Description))

progress <-
	drive_get("research_log") |>
	read_sheet(sheet = "progress_sheet")


tbl_status_updates <- function(progress, effort) {
	
	data <-
		left_join(progress, effort, by = "Type") |>
		group_by(Type) |>
		mutate(Type = paste0(Type, " (", Effort, ")")) |>
		mutate(Deadline = as.Date(Deadline)) |>
		select(-Effort) |>
		filter(Priority < 3) |>
		select(Type, Project, Task, Deadline, Priority, Status) |>
		group_by(Project) |>
		arrange(Project, Priority) |>
		mutate(num = 1:n()) 
		
	
	data |>
		gt(rowname_col = "Project", groupname_col = "Type") |>
		opt_table_lines(extent = "none") |>
		opt_row_striping(row_striping = FALSE) |>
		tab_options(data_row.padding = px(2)) |>
		# Headers and Row Groups
		tab_style(
			style =  cell_text(
				size = "large",
				weight = "bold",
				transform = "capitalize"
			),
			locations = cells_column_labels()
		) |>
		tab_style(
			style = cell_text(color = "white"),
			locations = cells_stub(rows = num != 1)
		) |>
		tab_style(
			style = list(cell_fill(color = stress[5]),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = stress[5])),
			locations = cells_row_groups(groups = contains("Epidemiology"))
		) |>
		tab_style(
			style = list(cell_fill(color = arrhythmia[2]),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = arrhythmia[2])),
			locations = cells_row_groups(groups = contains("Arrhythmia"))
		) |>
		tab_style(
			style = list(cell_fill(color = clinical[5]),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = clinical[5])),
			locations = cells_row_groups(groups = contains("Clinical"))
		) |>
		tab_style(
			style = list(cell_fill(color = computational[3]),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = computational[3])),
			locations = cells_row_groups(groups = contains("Computational"))
		) |>
		tab_style(
			style = list(cell_fill(color = text[3]),
									 cell_text(color = "white"),
									 cell_borders(sides = "all", color = text[3])),
			locations = cells_row_groups(groups = contains("Academic"))
		) |>
		# General formatting
		tab_style(
			style = cell_text(color = "white"),
			locations = cells_body(columns = Task,
														 rows = is.na(Task))
		) |>
		tab_style(
			style = cell_text(color = "white"),
			locations = cells_body(columns = Deadline,
														 rows = is.na(Deadline))
		) |>
		tab_style(
			style = cell_text(align = "center"),
			locations = list(cells_stub(), cells_column_labels())
		) |>
		tab_style(
			style = cell_borders(sides = "bottom", color = text[5]),
			locations = list(cells_body(columns = everything()), cells_stub(rows = everything()))
		)  |>
		tab_style(
			style = cell_text(color = "grey"),
			locations = cells_body(columns = Task,
														 rows = Status == "complete")
		) |>
		# Columns
		cols_hide(columns = c(Priority, num, Status)) |>
		fmt_date(columns = "Deadline", date_style = "iso") |>
		tab_style(
			style = cell_text(size = "x-small"),
			locations = cells_body(columns = "Deadline")
		) |>
		tab_style(
			style = cell_text(style = "italic"),
			locations = cells_stub()
		) |>
		cols_width(
			Project ~ pct(40),
			Task ~ pct(45),
			#Description ~ pct(35),
			Deadline ~ pct(15)
		) 
		
}
```


```{r}
#| label: tbl-progress
#| tbl-cap: Progress updates
#| echo: false
dt <- 
	date_build(year = 2023, month = 2, day = 20) |>
	add_days(-14)

effort <-
	log |>
	filter(Start >= dt) |>
	filter(Start > dt & End < add_days(dt, 7)) |>
	select(Type, Title, Duration) |>
	mutate(Total = as.numeric(sum(Duration))) |>
	mutate(Effort = as.numeric(Duration)/Total) |>
	group_by(Type) |>
	summarize(Effort = paste0(signif(100 * sum(Effort), 2), "%")) 

tbl_status_updates(progress, effort)
```

# February 20, 2023

## AFL/FH

- Genetic basis for FH may be more promising than association with FH broadly, but would require additional WES to be sent
- Would need to repeat VCF analysis for an arrhythmia panel (instead of cardiomyopathy panel)

## CBCD {.smaller}

Computational biorepository for cardiovascular disease

- Will need a large computational biorepository
	- Claims data from both CPT/ICD codes
	- Clinical documentation (raw text from clinical notes)
	- Medication history
	- Study data e.g. XML of ECG, echo reports, coronary angiograms, device interrogations, etc
- CCTS to pull data, DRA to be submitted today
	
::: aside
Also working on VA research database access, CART-CL reporting data, MVP, ARIC, and UK Biobank as additional large-scale data
:::

## HRV/CVD

![Increase of ~10 fold in CV mortality in 1/4 patients identified by abnormal resting and reactive vagal tone. Robust classifier of resilience.](../figures/hf-hrv-reactivity-km-quartile-curves.png){#fig-hf-hrv-reactivity-quartiles}

::: aside
Non-linear relationship established by spline analysis against CV death.
:::

## Specific aims

## {.smaller}

1. __Identify clinical phenotypes of vagally-triggered AF.__: 
	a. We will use a computational approach to [develop a pragmatic, population-level datasets]{.green} to identify phenotypes of pAF using clinical covariates, including ECG, TTE, cardiac monitoring devices, and markers of psychosocial stress. Hypothesis: _Low-burden groups will exist that have differences in clinical characteristics that can be used to reliably classify individuals._
	a. We will assess [recurrent clinical outcomes]{.blue}, including arrhythmia burden, conduction disease, atrial and ventricular cardiomyopathy, along with types of therapies (pharmacological/interventional). Hypothesis: _The hazard distribution will vary amongst the identified clinical phenotypes. The rate of escalation of intervention will also vary based on phenotypes._

## {.smaller}

2. __Determine the intracardiac characteristics of vagally-triggered AF.__:
	a. [Structural and electrical features]{.green} will be obtained through [EPS]{.red}. The EAM and EGM data collected will be used to identify intra-procedural AF phenotypes. Hypothesis: _Electrical and anatomical phenotypes will allow differentiation of re-entrant and triggered pAF._
	a. Electrical phenotypes of pAF will be compared with phenotypes identified in __1a__. Hypothesis: _The clinical phenotypes will correlate with the unique electrical phenotypes obtained during EPS._
	a. [Physiological stress, through catecholamine infusion, and neuropsychological stress, through mental stress challenge]{.red} will be applied during EPS to measure changes in electrical properties. Simultaneous measurement of critical [vagolytic neurohormones (NPY, Gal)]{.orange} will be measured at time of stressor. Hypothesis: _Catecholamine-infusion will lead to &darr;NPY and &darr;Gal, while mental stress will lead to &uarr;NPY and &uarr;Gal._
	a. [Vagally-triggered AF]{.orange} will be identified by interview prior to EPS. Hypothesis: _In those with suspected VT-pAF, triggered firing of pulmonary veins will be seen, and AF will be reliably induced by vagolysis._
	
## {.smaller}

3. __Evaluate role of common genetic variants in VTA.__:
	a. We will use a candidate-gene approach to assess association of common genetic variants with the clinical and electrical phenotypes identified in __1a__ and __2a__ in [individuals undergoing EPS]{.red}. Hypothesis: _Variants in genes associated with the cardiovagal outflow (e.g. Y2R and Gal1R), will be more common in those with VT-pAF_
	a. We will identify individuals that are classified as having [resting and stress-inducded vagal dysfunction]{.blue} (as standardized against CV death). Genetic data through [next-generation sequencing]{.green} will be obtained. Hypothesis: _We will identify novel variants that are associated with dysfunction in cardiovagal activity._

# February 6, 2023

## Updates

- AFL/FH: __WES__ needed prior to completing revisions
- ARIC: potential option to pursue genetic profiling in ARIC per Alvaro, need formal proposal
- K23: drafting specific aims in 1-pager format
- VCF: data able to be analyzed/cleaned, however requires more CPU to perform

::: aside
Currently on inpatient service with limited time
:::

---

![Chromosome 1 summary](../figures/vcf-sample-chr-1-from-mike.png){#fig-vcfr-summary}

# January 9, 2023

## Updates

:::: {.columns}
::: {.column width="50%"}
AFL/FH:

- Pedigrees/genetics completed (thanks to [Ana]{.blue}, [Shashank]{.blue}, [Mike]{.blue})
- Outline/draft to be re-written
:::
::: {.column width="50%"}
1. [HRV/CV Mortality]{.blue} paper rejected, need to re-think strategy with senior authors
1. K23 aims to include 1) EP lab as [translational]{.orange} component, 2) [arrhythmia risk prediction]{.blue} as [computational]{.green} component
:::
::::


```{r} 
#| cache: true 
store <- file.path(find_project_folder(), "aflubber", "_targets") 
afl_tbls <- targets::tar_read(flutter_tables, store = store)  
```

---

::: {.panel-tabset}

## Onset

```{r}
afl_tbls$early
```

## Adjusted Models

```{r}
afl_tbls$adjusted
```

## Individual Risk

```{r}
afl_tbls$individual
```

## Risk by +/- FH

```{r}
afl_tbls$individual_by_family
```

:::

## [AFL/genetics]{.blue}

- ~87 individuals with +FH
- ~81 individuals with WES
- ~18 individuals with VUS in total
- **1 individual** with VUS + FH

---

::: {.panel-tabset}

## Multiple Pedigrees

![AFL/surrogate disease burden](../figures/multiple_pedigree_plot.png)

## Cohort with Variants

```{r}
afl_tbls$variant_cohort
```

## TTN Mutant

![AFL/mutant](../figures/pedigree_uic0530.png)

## Variant Overview

```{r}
afl_tbls$variant_table
```

:::

# December 19, 2022

## Updates

:::: {.columns}
::: {.column width="50%"}
AFL/FH:

- Pedigrees TBD
- Draft completed, pending feedback
- Supplemental tables needed?
:::
::: {.column width="50%"}
AF/Recurrence:

- Scott added additional patients from UIC (September 2020 to now)
- Data collection pending
:::
::::

## K23 Aims {.smaller}

Objective: _Clinical EP researcher using computational neurocardiology techniques to study arrhythmia mechanisms_

1. Evaluating vagolysis and its effect on triggered arrhythmia mechanisms, both ventricular (e.g. SCD) and atrial (triggered AF)
1. Using computational approach to phenotype triggered onset arrhythmias, e.g. atrial fibrillation

Mentorship Team: __Amit J. Shah__, __Dawood Darbar__, Rachel Lampert, Viola Vaccarino, Mark McCauley, Andrew Boyd, Alvaro Alonso

## K23 Approach

__Vagal tone and triggered arrhythmias__

- Manuscript on stress-induced vagolysis and CV mortality __under review__
- ANS dysfunction during/with ischemia __work-in-progress__
- [Create murine/translational model for vagolysis and triggered arrhythmias]{.arrhythmia[2]}

__Arrhythmia phenotyping__

- AFL/FH manuscript __work-in-progress__
- AF/race project __work-in-progress__
- [Classification of arrhythmia phenotypes using large data set (MVP/VA research data, UIC, ARIC)]{.computational[3]}

## Murine Model of Vagolysis {.smaller}

- Atria are heavily innervated by ANS ganglia
	- Sympathovagal balance locally mediated through adrenergic lysis of cholinergic activity 
	- Neuropeptide Y (NPY) causes cholinergic inhibition through Y2R 
- Pro-arrhythmic murine model (compared to healthy controls)
	- Ex-vivo vagal-sparing Langendorf preparation
	- Catecholamine infusion and NPY Y2R antagonists to modulate arrhythmic state
- Measure atrial conductive properties as outcome
	- Baseline, with catecholamine infusion, VNS stimulation, and Y2R antagonism

# December 12, 2022

## AF Catheter Ablation and Recurrence

Data/power increase:

- Inclusion of VA data
- Additional data pull from EPIC by Scott Uphouse (pending)
- Inclusion of UIC billing code data?

## Atrial Flutter and Family History

:::: {.columns}

::: {.column width="50%"}
Pedigrees:

- How should these be incorporated into the manuscript? 
:::

::: {.column width="50%"}
Genetics:

- 305 patients that have undergone whole exome sequencing, but unclear how to match these to the AFL data
- Should we genotype the rest of the EO-AFL patients?
:::

::::

## ECG/EGM analysis

```{r}
library(rhythm)
x <- read_lspro(file = file.path(find_project_folder(), "research", "t32", "sample-egm.txt"))

ggm(x, channels = c("I", "CS", "HIS M", "HIS D", "RV")) |>
	add_intervals(channel = "I")
```


# November 21, 2022

## Updates

- AHA IPA: Completed draft, pending revisions
- AFL Paper: Draft to be done by Wednesday
- AF Ablation + Biorepository: Pending IRB
- Stress and CVD Mortality: Submitted to _Circulation_, revisions Pending

# November 14, 2022  

## Early Onset Atrial Flutter  

1. Draft of paper expanded to include results 
1. Prominent findings of family history and association with arrhythmia 
1. Key tables and figures established  

```{r} 
#| cache: true 
store <- file.path(find_project_folder(), "aflubber", "_targets") 
afl_tbls <- targets::tar_read(flutter_tables, store = store)  
```

---  

### Table: Cohort Descriptions  

```{r}
#| echo: false
afl_tbls$early |>
	as_raw_html()
```

---

### Table: Unadjusted Risk

```{r}
#| eval: false
#| echo: false
afl_tbls$individual
	as_raw_html()
```

---

### Table: Risk by Family History

```{r}
#| echo: false
afl_tbls$individual_by_family |>
	as_raw_html()
```

---

### Table: Fully Adjusted Model

```{r}
#| echo: false
afl_tbls$adjusted |>
	as_raw_html()
```

---

### Table: Grouped Forest Plots

```{r}
#| echo: false
afl_tbls$forest 
```

---

Next steps:

- Find agreement in key results and figures
- Identify main finding
- Complete discussion section

# October 10, 2022

---

__AF Registry__

Status:

- Current [overview](https://github.com/asshah4/aflubber/R/af_ablation_recurrence.html)
- Not yet powered
- Will need to revise out approach

# October 2, 2022

---

__AF Registry__

- Current number: 219
- Recurrence rate: 46%
- Death = 12 
- MACE = 70 (including repeats)

---

```{r}
#| eval: false
#| echo: false
#| cache: true
store <- file.path(find_project_folder(), "mims", "_targets")
curves <- targets::tar_read(curves, store = store)
curves$lf_stress
```

--- 

```{r}
x <- cardiac_atlas$right_atrium_rao

ggplot() +
	theme_void() +
	geom_sf(data = x$chambers, aes(fill = structure)) +
	geom_sf(data = x$projections, aes(fill = structure)) + 
	geom_sf(data = x$valves, aes(fill = structure)) + 
	geom_sf(data = x$septum, aes(fill = structure)) + 
	geom_sf(data = x$vessels, fill = "white") +
	scale_fill_brewer(palette = "YlGnBu") +
	theme(legend.position = "none") 
```

GIS-based MAP of right atrium for mapping and localization purposes for publications.


# September 26, 2022

---


## AF Registry {.smaller}

:::: {.columns}

::: {.column width="40%"}
__Analysis__

- Genetic analysis in PLINK complete
- Models are set up for analysis once data intake is done
:::

::: {.column width="40%"}
__Status__

1. Have n = 238 currently
1. Reviewed 95 charts thus far (CAR team)
1. Of n = 108, 75 have had recurrence, 33 without (rate of about 70% of anyarrhythmia)
1. Of ECG based confirmation, 43 AF recurrence events (53%)

:::

::::

# September 12, 2022

---

## AF Registry {.smaller}

REDCap data dictionary is finalized. Next steps are:

1. Data entry for outcomes
2. Finalization of new patients to add from AF ablation registry

## Genetics {.smaller}

Currently able to:

- Format large SNP data sets into appropriate tables
- Utilize `PLINK` and `MERLIN` to process Chen's genetics data
- Run from R to help analyze findings

Next steps:

- Confirm ancestry (for practice) percent likelihood 
- Then, once outcome data is complete, analyze differences by race

Analytical methods:

- Time to binary outcome of recurrence
- Recurrent event analysis for AF recurrence
- Can revise with Bayesian modeling in `STAN` as well


# August 22, 2022

---

## AF Registry {.smaller}

:::: {.columns}

::: {.column width="40%"}
__Status__

- REDCap is in progress
- Have an additional list of registry patients that may/may not have had ablation

:::

::: {.column width="40%"}
__Next Steps__

1. Complete 100 REDCap patients
1. Identify 300 patients that have potentially had ablation

:::

::::

---

## CARTO Maps {.smaller}

- Ablation quantification
- Activation mapping
- EGM annotation
- Voltage mapping
- Conduction velocity
- Earliest/latest activation
- Geometry data



# August 15, 2022

--- 

:::: {.columns}

::: {.column width="40%"}
__UIC-wide data summary__

- N ~ XXX had ablation
- Date range from >= 2010

__AF registry data__

- N=~300 had ablation
- Date range from >= 2010

:::

::: {.column width="40%"}
__Next Steps__

1. Generate list of MRN for data collection
1. Use new REDCap to obtain ECG/ablation data
1. Run preliminary analysis on first 100 patients
1. Consider expansion to JBVA/ACMC data
:::

::::


# August 1, 2022

## AF Registry

Consider additional variables:

- Recurrent events and adjudication (HF, cardiac admission, MACE)
- Holter/ECG data and repeat collection (P-wave morphology, Wilson's vector gradient, amplitude of AF waves "coarseness")

Will check with the MESA and ARIC data base on how repeat events were defined.

## Murine EP Studies

```{r, out.width = "60%"}
knitr::include_graphics("../figures/electromap.png")
```

## Human EP Studies

```{r}
knitr::include_graphics("../figures/openep-ablation.png")
```

# July 25, 2022

## Forest Plots

::::{.columns}

:::{.column width="50%"}
```{r}
#| echo: true
#| eval: false
# Forest Plots
library(volundr)
m <- rx(
	Petal.Length ~ X(Sepal.Length) + Petal.Width + S(Species),
	label = list(
		Petal.Length ~ "Length of Petals", 
		Species ~ "Genus of the Flower"
	),
	pattern = "direct"
	) |>
	fmls(order = 2) |>
	fit(.fit = lm,
			data = iris,
			archetype = TRUE) |>
	mdls()
```
:::

:::{.column width="50%"}
```{r}
#| echo: true
#| eval: false
tbl <- tbl_forest( #<<
	object = m,
	formula = Petal.Length ~ Sepal.Length,
	vars = "Species",
	columns = list(
		beta ~ "Estimate",
		conf ~ "95% Confidence Interval",
		n ~ "Number of Samples"
	),
	axis = list(
		lim ~ c(0, 3),
		breaks ~ c(0, 1, 2),
		lab ~ "ß (95% CI)",
		title ~ "Petal Length by Sepal Length"
	)
)
```
:::

::::

A simple way to make sub-group analysis Forest plots, working on improving customization.

---

```{r}
#| label: Example forest plot
#| echo: false
#| eval: false
tbl |>
	tab_style(style = cell_fill(color = "white"),
						locations = cells_body(columns = everything()))
```

# July 18, 2022


## AF Registry

The AF registry is described [here](https://asshah4.github.io/aflubber). Current issues:

1. Data quality - consistency between reviewers
1. Missingness - decisions on acceptable thresholds
1. Variable selection - echo findings, EP studies, medications, labs, symptoms
1. Management - REDCap, shared excel sheet
1. Adjudication/review - outcomes, clinical follow-up length
